<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>Mutation Comparison</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<!--Output Html_Mutation_Table_String-->
<table border="0" cellspacing="1" cellpadding="3">
<tr><th colspan="7" align="left" class="mutation_header_row">Predicted mutations</th></tr><tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>

<!-- Item Lines -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_135</td><!-- Seq_Id -->
<td align="right">3,883</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">C55F</font>&nbsp;(T<font class="mutation_in_codon">G</font>C&rarr;T<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=tgallagh@patricbrc.o rg</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_145</td><!-- Seq_Id -->
<td align="right">1,246</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">21.2%</td>
<td align="center"><font class="snp_type_synonymous">Y1074Y</font>&nbsp;(TA<font class="mutation_in_codon">T</font>&rarr;TA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Pyoverdine sidechain non&#8209;ribosomal peptide synthetase PvdJ @ Siderophore biosynthesis non&#8209;ribosomal peptide synthetase modules</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_145</td><!-- Seq_Id -->
<td align="right">1,254</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">28.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">P1072A</font>&nbsp;(<font class="mutation_in_codon">C</font>CG&rarr;<font class="mutation_in_codon">G</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Pyoverdine sidechain non&#8209;ribosomal peptide synthetase PvdJ @ Siderophore biosynthesis non&#8209;ribosomal peptide synthetase modules</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_145</td><!-- Seq_Id -->
<td align="right">1,255</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">28.8%</td>
<td align="center"><font class="snp_type_synonymous">Y1071Y</font>&nbsp;(TA<font class="mutation_in_codon">T</font>&rarr;TA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Pyoverdine sidechain non&#8209;ribosomal peptide synthetase PvdJ @ Siderophore biosynthesis non&#8209;ribosomal peptide synthetase modules</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_290</td><!-- Seq_Id -->
<td align="right">6,600</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">D17V</font>&nbsp;(G<font class="mutation_in_codon">A</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=tgallagh@patricbrc.o rg</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_337</td><!-- Seq_Id -->
<td align="right">2,524</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_synonymous">F512F</font>&nbsp;(TT<font class="mutation_in_codon">C</font>&rarr;TT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">2&#8209;Amino&#8209;2&#8209;deoxy&#8209;isochorismate synthase (EC 2.6.1.86) PhzE</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_400</td><!-- Seq_Id -->
<td align="right">4,889</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_synonymous">S387S</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_423</td><!-- Seq_Id -->
<td align="right">4,184</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">A340T</font>&nbsp;(<font class="mutation_in_codon">G</font>CA&rarr;<font class="mutation_in_codon">A</font>CA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Copper resistance protein B</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_445</td><!-- Seq_Id -->
<td align="right">1,708</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center"><font class="snp_type_synonymous">V1019V</font>&nbsp;(GT<font class="mutation_in_codon">G</font>&rarr;GT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FIG007317: Chromosome segregation protein SMC&#8209;like</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_445</td><!-- Seq_Id -->
<td align="right">5,917</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">V386L</font>&nbsp;(<font class="mutation_in_codon">G</font>TC&rarr;<font class="mutation_in_codon">C</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FIG149030: hypothetical protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_445</td><!-- Seq_Id -->
<td align="right">7,086</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center">intergenic&nbsp;(&#8209;14/&#8209;567)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=tgallagh@patricbrc.o rg</i></td>
<td align="left">FIG149030: hypothetical protein/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_447</td><!-- Seq_Id -->
<td align="right">5,122</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_synonymous">R73R</font>&nbsp;(CG<font class="mutation_in_codon">G</font>&rarr;CG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_448</td><!-- Seq_Id -->
<td align="right">11,031</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_synonymous">R9R</font>&nbsp;(CG<font class="mutation_in_codon">T</font>&rarr;CG<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">FIG005107: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_490</td><!-- Seq_Id -->
<td align="right">4,977</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">T40I</font>&nbsp;(A<font class="mutation_in_codon">C</font>C&rarr;A<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_490</td><!-- Seq_Id -->
<td align="right">16,372</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">D91N</font>&nbsp;(<font class="mutation_in_codon">G</font>AC&rarr;<font class="mutation_in_codon">A</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">putative membrane protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_531</td><!-- Seq_Id -->
<td align="right">9,992</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">T250I</font>&nbsp;(A<font class="mutation_in_codon">C</font>C&rarr;A<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative two&#8209;component sensor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_531</td><!-- Seq_Id -->
<td align="right">12,490</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">R1083C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative two&#8209;component sensor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_532</td><!-- Seq_Id -->
<td align="right">15,035</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">F115I</font>&nbsp;(<font class="mutation_in_codon">T</font>TC&rarr;<font class="mutation_in_codon">A</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Large exoproteins involved in heme utilization or adhesion</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_532</td><!-- Seq_Id -->
<td align="right">15,042</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_synonymous">L112L</font>&nbsp;(CT<font class="mutation_in_codon">C</font>&rarr;CT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Large exoproteins involved in heme utilization or adhesion</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_532</td><!-- Seq_Id -->
<td align="right">15,315</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_synonymous">Q21Q</font>&nbsp;(CA<font class="mutation_in_codon">G</font>&rarr;CA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Large exoproteins involved in heme utilization or adhesion</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_532</td><!-- Seq_Id -->
<td align="right">15,320</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">21.6%</td>
<td align="center">coding&nbsp;(58/921&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Large exoproteins involved in heme utilization or adhesion</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_532</td><!-- Seq_Id -->
<td align="right">15,321</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_synonymous">A19A</font>&nbsp;(GC<font class="mutation_in_codon">T</font>&rarr;GC<font class="mutation_in_codon">C</font>)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Large exoproteins involved in heme utilization or adhesion</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_532</td><!-- Seq_Id -->
<td align="right">15,322</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">A19G</font>&nbsp;(G<font class="mutation_in_codon">C</font>T&rarr;G<font class="mutation_in_codon">G</font>T)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Large exoproteins involved in heme utilization or adhesion</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_532</td><!-- Seq_Id -->
<td align="right">15,323</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">A19T</font>&nbsp;(<font class="mutation_in_codon">G</font>CT&rarr;<font class="mutation_in_codon">A</font>CT)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Large exoproteins involved in heme utilization or adhesion</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_532</td><!-- Seq_Id -->
<td align="right">15,323:1</td><!-- Position -->
<td align="center">+T</td><!-- Cell Mutation -->
<td align="right">26.3%</td>
<td align="center">coding&nbsp;(55/921&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Large exoproteins involved in heme utilization or adhesion</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_532</td><!-- Seq_Id -->
<td align="right">15,330</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">32.9%</td>
<td align="center"><font class="snp_type_synonymous">Y16Y</font>&nbsp;(TA<font class="mutation_in_codon">T</font>&rarr;TA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Large exoproteins involved in heme utilization or adhesion</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_532</td><!-- Seq_Id -->
<td align="right">15,336</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">31.6%</td>
<td align="center"><font class="snp_type_synonymous">G14G</font>&nbsp;(GG<font class="mutation_in_codon">T</font>&rarr;GG<font class="mutation_in_codon">G</font>)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Large exoproteins involved in heme utilization or adhesion</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_532</td><!-- Seq_Id -->
<td align="right">15,337</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">31.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">G14A</font>&nbsp;(G<font class="mutation_in_codon">G</font>T&rarr;G<font class="mutation_in_codon">C</font>T)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Large exoproteins involved in heme utilization or adhesion</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_532</td><!-- Seq_Id -->
<td align="right">15,348</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">9.2%</td>
<td align="center"><font class="snp_type_synonymous">G10G</font>&nbsp;(GG<font class="mutation_in_codon">T</font>&rarr;GG<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Large exoproteins involved in heme utilization or adhesion</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_532</td><!-- Seq_Id -->
<td align="right">15,357</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">9.6%</td>
<td align="center"><font class="snp_type_synonymous">G7G</font>&nbsp;(GG<font class="mutation_in_codon">T</font>&rarr;GG<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Large exoproteins involved in heme utilization or adhesion</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_549</td><!-- Seq_Id -->
<td align="right">903</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">R295H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Uncharacterized protein YjgR</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_549</td><!-- Seq_Id -->
<td align="right">2,899</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">14.0%</td>
<td align="center"><font class="snp_type_synonymous">G238G</font>&nbsp;(GG<font class="mutation_in_codon">G</font>&rarr;GG<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Methyl&#8209;accepting chemotaxis sensor/transducer protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_549</td><!-- Seq_Id -->
<td align="right">2,900</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">H239N</font>&nbsp;(<font class="mutation_in_codon">C</font>AT&rarr;<font class="mutation_in_codon">A</font>AT)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Methyl&#8209;accepting chemotaxis sensor/transducer protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_549</td><!-- Seq_Id -->
<td align="right">2,902</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center"><font class="snp_type_synonymous">H239H</font>&nbsp;(CA<font class="mutation_in_codon">T</font>&rarr;CA<font class="mutation_in_codon">C</font>)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Methyl&#8209;accepting chemotaxis sensor/transducer protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_565</td><!-- Seq_Id -->
<td align="right">8,908</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q1027H</font>&nbsp;(CA<font class="mutation_in_codon">G</font>&rarr;CA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">DNA&#8209;directed RNA polymerase beta subunit (EC 2.7.7.6)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_565</td><!-- Seq_Id -->
<td align="right">13,932</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_synonymous">E40E</font>&nbsp;(GA<font class="mutation_in_codon">G</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">LSU ribosomal protein L1p (L10Ae)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_581</td><!-- Seq_Id -->
<td align="right">3,609</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">14.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">E65D</font>&nbsp;(GA<font class="mutation_in_codon">G</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Copper tolerance protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_581</td><!-- Seq_Id -->
<td align="right">6,835</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">L7Q</font>&nbsp;(C<font class="mutation_in_codon">T</font>G&rarr;C<font class="mutation_in_codon">A</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_581</td><!-- Seq_Id -->
<td align="right">12,205</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">L22P</font>&nbsp;(C<font class="mutation_in_codon">T</font>T&rarr;C<font class="mutation_in_codon">C</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Uncharacterized protein conserved in bacteria, NMA0228&#8209;like</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_584</td><!-- Seq_Id -->
<td align="right">4,706</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_synonymous">S339S</font>&nbsp;(TC<font class="mutation_in_codon">G</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phage tail tape measure</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_584</td><!-- Seq_Id -->
<td align="right">15,474</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">14.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">G117V</font>&nbsp;(G<font class="mutation_in_codon">G</font>A&rarr;G<font class="mutation_in_codon">T</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phage endolysin</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_587</td><!-- Seq_Id -->
<td align="right">19,430</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">E169D</font>&nbsp;(GA<font class="mutation_in_codon">G</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Hydrolase, alpha/beta fold family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_588</td><!-- Seq_Id -->
<td align="right">4,137</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">15.4%</td>
<td align="center"><font class="snp_type_synonymous">A1379A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative large exoprotein involved in heme utilization or adhesion of ShlA/HecA/FhaA family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_588</td><!-- Seq_Id -->
<td align="right">24,137</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">R2C</font>&nbsp;(<font class="mutation_in_codon">C</font>GT&rarr;<font class="mutation_in_codon">T</font>GT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">putative lipoprotein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_588</td><!-- Seq_Id -->
<td align="right">27,951</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_synonymous">A97A</font>&nbsp;(GC<font class="mutation_in_codon">C</font>&rarr;GC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Oligopeptidase A (EC 3.4.24.70)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_588</td><!-- Seq_Id -->
<td align="right">50,011</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.5%</td>
<td align="center"><font class="snp_type_nonsense">E186*</font>&nbsp;(<font class="mutation_in_codon">G</font>AG&rarr;<font class="mutation_in_codon">T</font>AG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&rarr;</td>
<td align="left">Response regulator</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_589</td><!-- Seq_Id -->
<td align="right">17,577</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">P171S</font>&nbsp;(<font class="mutation_in_codon">C</font>CG&rarr;<font class="mutation_in_codon">T</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_595</td><!-- Seq_Id -->
<td align="right">2,476</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">17.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">L127F</font>&nbsp;(<font class="mutation_in_codon">C</font>TC&rarr;<font class="mutation_in_codon">T</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Arginine/ornithine ABC transporter, substrate&#8209;binding protein AotJ</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_595</td><!-- Seq_Id -->
<td align="right">4,370</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center"><font class="snp_type_synonymous">C122C</font>&nbsp;(TG<font class="mutation_in_codon">C</font>&rarr;TG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glycine/D&#8209;amino acid oxidases (deaminating)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_595</td><!-- Seq_Id -->
<td align="right">17,156</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">14.8%</td>
<td align="center"><font class="snp_type_synonymous">D180D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_596</td><!-- Seq_Id -->
<td align="right">124</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">12.9%</td>
<td align="center"><font class="snp_type_synonymous">E41E</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Translation elongation factor Tu</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_596</td><!-- Seq_Id -->
<td align="right">21,221</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_synonymous">R126R</font>&nbsp;(CG<font class="mutation_in_codon">C</font>&rarr;CG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Single&#8209;stranded DNA&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_596</td><!-- Seq_Id -->
<td align="right">22,604</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">F86Y</font>&nbsp;(T<font class="mutation_in_codon">T</font>C&rarr;T<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Isochorismate synthase (EC 5.4.4.2) @ Isochorismate synthase (EC 5.4.4.2) [pyochelin] siderophore @ Isochorismate synthase (EC 5.4.4.2) of siderophore biosynthesis</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_596</td><!-- Seq_Id -->
<td align="right">29,692</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">G941D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Dihydroaeruginoate synthetase PchE, non&#8209;ribosomal peptide synthetase modules @ Siderophore biosynthesis non&#8209;ribosomal peptide synthetase modules</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_596</td><!-- Seq_Id -->
<td align="right">37,199:1</td><!-- Position -->
<td align="center">+G</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center">coding&nbsp;(590/1050&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Pyochelin biosynthetic protein PchG, oxidoreductase (NAD&#8209;binding) @ Thiazolinyl imide reductase in siderophore biosynthesis gene cluster</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_596</td><!-- Seq_Id -->
<td align="right">37,199:2</td><!-- Position -->
<td align="center">+C</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center">coding&nbsp;(590/1050&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Pyochelin biosynthetic protein PchG, oxidoreductase (NAD&#8209;binding) @ Thiazolinyl imide reductase in siderophore biosynthesis gene cluster</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_596</td><!-- Seq_Id -->
<td align="right">37,199:3</td><!-- Position -->
<td align="center">+A</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center">coding&nbsp;(590/1050&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Pyochelin biosynthetic protein PchG, oxidoreductase (NAD&#8209;binding) @ Thiazolinyl imide reductase in siderophore biosynthesis gene cluster</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_598</td><!-- Seq_Id -->
<td align="right">22,995</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_synonymous">S369S</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">4&#8209;hydroxybenzoate transporter</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_598</td><!-- Seq_Id -->
<td align="right">42,173</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_synonymous">S19S</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_598</td><!-- Seq_Id -->
<td align="right">45,420</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">G15R</font>&nbsp;(<font class="mutation_in_codon">G</font>GG&rarr;<font class="mutation_in_codon">C</font>GG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Glutathione reductase (EC 1.8.1.7)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_599</td><!-- Seq_Id -->
<td align="right">13,447</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">R38S</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">A</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=tgallagh@patricbrc.o rg</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_602</td><!-- Seq_Id -->
<td align="right">18,032</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A115T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">3&#8209;oxoacyl&#8209;[acyl&#8209;carrier&#8209;protein] synthase (EC 2.3.1.41)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_602</td><!-- Seq_Id -->
<td align="right">19,895</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">T374M</font>&nbsp;(A<font class="mutation_in_codon">C</font>G&rarr;A<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Multidrug efflux system, outer membrane factor lipoprotein => OpmE of MexPQ&#8209;OpmE system</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_602</td><!-- Seq_Id -->
<td align="right">26,762</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">E213V</font>&nbsp;(G<font class="mutation_in_codon">A</font>A&rarr;G<font class="mutation_in_codon">T</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Argininosuccinate synthase (EC 6.3.4.5)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_602</td><!-- Seq_Id -->
<td align="right">37,936</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">17.4%</td>
<td align="center"><font class="snp_type_synonymous">G503G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative serine protease</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_602</td><!-- Seq_Id -->
<td align="right">40,987</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">P38S</font>&nbsp;(<font class="mutation_in_codon">C</font>CG&rarr;<font class="mutation_in_codon">T</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">Phage protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_618</td><!-- Seq_Id -->
<td align="right">2,320</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">A337V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Low&#8209;specificity L&#8209;threonine aldolase (EC 4.1.2.48)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_618</td><!-- Seq_Id -->
<td align="right">6,008</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_synonymous">C751C</font>&nbsp;(TG<font class="mutation_in_codon">C</font>&rarr;TG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_618</td><!-- Seq_Id -->
<td align="right">7,970</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_synonymous">I84I</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">GbcA Glycine betaine demethylase subunit A</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_618</td><!-- Seq_Id -->
<td align="right">16,565</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_nonsense">Q653*</font>&nbsp;(<font class="mutation_in_codon">C</font>AG&rarr;<font class="mutation_in_codon">T</font>AG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">DgcA Dimethylglycine demethylase subunit A</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_618</td><!-- Seq_Id -->
<td align="right">16,569</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_synonymous">L651L</font>&nbsp;(CT<font class="mutation_in_codon">C</font>&rarr;CT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">DgcA Dimethylglycine demethylase subunit A</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_618</td><!-- Seq_Id -->
<td align="right">18,608</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">V154L</font>&nbsp;(<font class="mutation_in_codon">G</font>TG&rarr;<font class="mutation_in_codon">T</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Predicted hydrocarbon binding protein (contains V4R domain)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_618</td><!-- Seq_Id -->
<td align="right">36,039</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center">intergenic&nbsp;(&#8209;313/+108)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Transcriptional regulator containing an amidase domain and an AraC&#8209;type DNA&#8209;binding HTH domain/L&#8209;serine dehydratase, beta subunit (EC 4.3.1.17) / L&#8209;serine dehydratase, alpha subunit (EC 4.3.1.17)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_618</td><!-- Seq_Id -->
<td align="right">43,857</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q39R</font>&nbsp;(C<font class="mutation_in_codon">A</font>G&rarr;C<font class="mutation_in_codon">G</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Betaine aldehyde dehydrogenase (EC 1.2.1.8)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_618</td><!-- Seq_Id -->
<td align="right">44,529</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">P263L</font>&nbsp;(C<font class="mutation_in_codon">C</font>C&rarr;C<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Betaine aldehyde dehydrogenase (EC 1.2.1.8)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_619</td><!-- Seq_Id -->
<td align="right">26,737</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">P78L</font>&nbsp;(C<font class="mutation_in_codon">C</font>G&rarr;C<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ABC&#8209;type antimicrobial peptide transport system, permease component</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_619</td><!-- Seq_Id -->
<td align="right">27,865</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">L59M</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Efflux transport system, outer membrane factor (OMF) lipoprotein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_619</td><!-- Seq_Id -->
<td align="right">27,923</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">D78G</font>&nbsp;(G<font class="mutation_in_codon">A</font>C&rarr;G<font class="mutation_in_codon">G</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Efflux transport system, outer membrane factor (OMF) lipoprotein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_619</td><!-- Seq_Id -->
<td align="right">43,707</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">G603D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">putative outermembrane protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_619</td><!-- Seq_Id -->
<td align="right">46,924</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_synonymous">P68P</font>&nbsp;(CC<font class="mutation_in_codon">G</font>&rarr;CC<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ATP&#8209;dependent protease La (EC 3.4.21.53) Type II</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_619</td><!-- Seq_Id -->
<td align="right">72,294</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">R226C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Type IV fimbrial biogenesis protein PilW</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_621</td><!-- Seq_Id -->
<td align="right">3,511</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">G16D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">FOG: TPR repeat, SEL1 subfamily</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_621</td><!-- Seq_Id -->
<td align="right">4,046</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">A554V</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_621</td><!-- Seq_Id -->
<td align="right">10,233</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center">coding&nbsp;(470/996&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ABC transporter, ATP&#8209;binding protein (cluster 1, maltose/g3p/polyamine/iron); ABC transporter, ATP&#8209;binding protein (cluster 10, nitrate/sulfonate/bicarbonate)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_621</td><!-- Seq_Id -->
<td align="right">10,234</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center">coding&nbsp;(469/996&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ABC transporter, ATP&#8209;binding protein (cluster 1, maltose/g3p/polyamine/iron); ABC transporter, ATP&#8209;binding protein (cluster 10, nitrate/sulfonate/bicarbonate)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_621</td><!-- Seq_Id -->
<td align="right">10,235</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center">coding&nbsp;(468/996&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ABC transporter, ATP&#8209;binding protein (cluster 1, maltose/g3p/polyamine/iron); ABC transporter, ATP&#8209;binding protein (cluster 10, nitrate/sulfonate/bicarbonate)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_621</td><!-- Seq_Id -->
<td align="right">17,419</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">12.9%</td>
<td align="center">intergenic&nbsp;(&#8209;143/+26)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Aspartyl aminopeptidase (EC 3.4.11.21)/LSU rRNA pseudouridine(746) synthase (EC 5.4.99.29) @ tRNA pseudouridine(32) synthase (EC 5.4.99.28)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_621</td><!-- Seq_Id -->
<td align="right">33,118</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">N630D</font>&nbsp;(<font class="mutation_in_codon">A</font>AT&rarr;<font class="mutation_in_codon">G</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_624</td><!-- Seq_Id -->
<td align="right">732</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">G244D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Serine hydroxymethyltransferase (EC 2.1.2.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_624</td><!-- Seq_Id -->
<td align="right">5,862</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">14.8%</td>
<td align="center">intergenic&nbsp;(&#8209;564/&#8209;125)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/Membrane protease subunits, stomatin/prohibitin homologs</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_624</td><!-- Seq_Id -->
<td align="right">8,657</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">R220H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">HflC protein paralog among fluorescent psuedomonads</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_624</td><!-- Seq_Id -->
<td align="right">22,773</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">A100V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Pyoverdine chromophore precursor synthetase PvdL @ Siderophore biosynthesis non&#8209;ribosomal peptide synthetase modules</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_624</td><!-- Seq_Id -->
<td align="right">44,510</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">18.8%</td>
<td align="center"><font class="snp_type_synonymous">G164G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">CRISPR&#8209;associated helicase Cas3</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_624</td><!-- Seq_Id -->
<td align="right">49,368</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_synonymous">S61S</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Hypothertical protein, coexpressed with pyoverdine biosynthesis regulon</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_624</td><!-- Seq_Id -->
<td align="right">52,533</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">R64C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Pirin</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_624</td><!-- Seq_Id -->
<td align="right">63,419</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">15.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">G117V</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FIG014801: Cation ABC transporter, periplasmic cation&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_628</td><!-- Seq_Id -->
<td align="right">13,605</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center">coding&nbsp;(1141/1800&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Predicted signal&#8209;transduction protein containing cAMP&#8209;binding and CBS domains</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_628</td><!-- Seq_Id -->
<td align="right">20,821</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_synonymous">H173H</font>&nbsp;(CA<font class="mutation_in_codon">C</font>&rarr;CA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transcriptional regulator, LysR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_628</td><!-- Seq_Id -->
<td align="right">21,193</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_synonymous">G49G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transcriptional regulator, LysR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_628</td><!-- Seq_Id -->
<td align="right">36,587</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">A184T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ABC transporter, ATP&#8209;binding protein (cluster 9, phospholipid)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_628</td><!-- Seq_Id -->
<td align="right">38,818</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">G328R</font>&nbsp;(<font class="mutation_in_codon">G</font>GG&rarr;<font class="mutation_in_codon">A</font>GG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Trk potassium uptake system protein TrkH</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_628</td><!-- Seq_Id -->
<td align="right">49,109</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">S239R</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Segregation and condensation protein B</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_628</td><!-- Seq_Id -->
<td align="right">49,173</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">L261M</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Segregation and condensation protein B</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_637</td><!-- Seq_Id -->
<td align="right">35,736</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.9%</td>
<td align="center"><font class="snp_type_synonymous">P134P</font>&nbsp;(CC<font class="mutation_in_codon">G</font>&rarr;CC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Fatty acid desaturase (EC 1.14.19.1); Delta&#8209;9 fatty acid desaturase (EC 1.14.19.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_637</td><!-- Seq_Id -->
<td align="right">37,363</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">V208M</font>&nbsp;(<font class="mutation_in_codon">G</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">sodium&#8209;solute symporter, putative</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_637</td><!-- Seq_Id -->
<td align="right">40,310</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">14.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">A15G</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">G</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Sensory box/GGDEF domain protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_641</td><!-- Seq_Id -->
<td align="right">10,086</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_synonymous">T319T</font>&nbsp;(AC<font class="mutation_in_codon">C</font>&rarr;AC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">probable secretion protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_641</td><!-- Seq_Id -->
<td align="right">27,592</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_synonymous">A276A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Sensory box/GGDEF family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_641</td><!-- Seq_Id -->
<td align="right">28,436</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">L120M</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_641</td><!-- Seq_Id -->
<td align="right">31,243</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_synonymous">A440A</font>&nbsp;(GC<font class="mutation_in_codon">C</font>&rarr;GC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Tfp pilus assembly protein FimV</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_641</td><!-- Seq_Id -->
<td align="right">37,917</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_synonymous">I24I</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Acyl carrier protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_641</td><!-- Seq_Id -->
<td align="right">39,727</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">H317R</font>&nbsp;(C<font class="mutation_in_codon">A</font>C&rarr;C<font class="mutation_in_codon">G</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Putative cytochrome P450 hydroxylase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_641</td><!-- Seq_Id -->
<td align="right">50,288</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A282T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Siderophore biosynthesis non&#8209;ribosomal peptide synthetase modules</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_641</td><!-- Seq_Id -->
<td align="right">53,336</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">H39N</font>&nbsp;(<font class="mutation_in_codon">C</font>AC&rarr;<font class="mutation_in_codon">A</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_641</td><!-- Seq_Id -->
<td align="right">76,891</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">15.0%</td>
<td align="center"><font class="snp_type_synonymous">A125A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Tetrapartite efflux system, inner membrane component FusBC&#8209;like</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_641</td><!-- Seq_Id -->
<td align="right">88,629</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">L652M</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ATP&#8209;dependent helicase HrpA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_641</td><!-- Seq_Id -->
<td align="right">90,492</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">18.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">R1273C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ATP&#8209;dependent helicase HrpA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_646</td><!-- Seq_Id -->
<td align="right">5,700</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">16.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">A204T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Exoenzymes regulatory protein AepA precursor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_646</td><!-- Seq_Id -->
<td align="right">6,646</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">22.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">A519V</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Exoenzymes regulatory protein AepA precursor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_646</td><!-- Seq_Id -->
<td align="right">11,855</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">R89H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">SAM&#8209;dependent methyltransferase</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_647</td><!-- Seq_Id -->
<td align="right">656</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">9.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q131H</font>&nbsp;(CA<font class="mutation_in_codon">G</font>&rarr;CA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">LysR&#8209;family transcriptional regulator PtxE, associated with phosphonate utilization</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_649</td><!-- Seq_Id -->
<td align="right">18,590</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_synonymous">R76R</font>&nbsp;(AG<font class="mutation_in_codon">G</font>&rarr;AG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">FIG00957911: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_649</td><!-- Seq_Id -->
<td align="right">23,333</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">18.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">P97L</font>&nbsp;(C<font class="mutation_in_codon">C</font>G&rarr;C<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_649</td><!-- Seq_Id -->
<td align="right">24,867</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_synonymous">T36T</font>&nbsp;(AC<font class="mutation_in_codon">C</font>&rarr;AC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">Conjugative transfer protein PilM in PFGI&#8209;1&#8209;like cluster</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_649</td><!-- Seq_Id -->
<td align="right">51,389</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">G278D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Protein with ParB&#8209;like nuclease domain in PFGI&#8209;1&#8209;like cluster</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_649</td><!-- Seq_Id -->
<td align="right">52,442</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">V11I</font>&nbsp;(<font class="mutation_in_codon">G</font>TC&rarr;<font class="mutation_in_codon">A</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">FIG00960798: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_652</td><!-- Seq_Id -->
<td align="right">1,998</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">G514D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Error&#8209;prone repair homolog of DNA polymerase III alpha subunit (EC 2.7.7.7)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_652</td><!-- Seq_Id -->
<td align="right">3,564</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">P463S</font>&nbsp;(<font class="mutation_in_codon">C</font>CG&rarr;<font class="mutation_in_codon">T</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">DNA polymerase IV&#8209;like protein ImuB</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_652</td><!-- Seq_Id -->
<td align="right">25,546</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">G1281D</font>&nbsp;(G<font class="mutation_in_codon">G</font>T&rarr;G<font class="mutation_in_codon">A</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Filamentous haemagglutinin family outer membrane protein associated with VreARI signalling system</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_652</td><!-- Seq_Id -->
<td align="right">27,034</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">10.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">M1777T</font>&nbsp;(A<font class="mutation_in_codon">T</font>G&rarr;A<font class="mutation_in_codon">C</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Filamentous haemagglutinin family outer membrane protein associated with VreARI signalling system</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_652</td><!-- Seq_Id -->
<td align="right">27,138</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">R1812W</font>&nbsp;(<font class="mutation_in_codon">C</font>GG&rarr;<font class="mutation_in_codon">T</font>GG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Filamentous haemagglutinin family outer membrane protein associated with VreARI signalling system</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_652</td><!-- Seq_Id -->
<td align="right">51,873</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.9%</td>
<td align="center">intergenic&nbsp;(&#8209;32/+61)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Chloramphenicol O&#8209;acetyltransferase (EC 2.3.1.28) => CatB family/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_652</td><!-- Seq_Id -->
<td align="right">52,240</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center"><font class="snp_type_synonymous">L158L</font>&nbsp;(CT<font class="mutation_in_codon">G</font>&rarr;CT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_654</td><!-- Seq_Id -->
<td align="right">23,084</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">L59H</font>&nbsp;(C<font class="mutation_in_codon">T</font>T&rarr;C<font class="mutation_in_codon">A</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phage lysozyme R (EC 3.2.1.17)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_654</td><!-- Seq_Id -->
<td align="right">38,744</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_synonymous">A47A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">Phage protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_654</td><!-- Seq_Id -->
<td align="right">70,224</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">S293Y</font>&nbsp;(T<font class="mutation_in_codon">C</font>C&rarr;T<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Integrase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_654</td><!-- Seq_Id -->
<td align="right">83,606</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">G26S</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">A</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Ribokinase (EC 2.7.1.15)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_654</td><!-- Seq_Id -->
<td align="right">86,469</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">R280S</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">A</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Ribose ABC transporter, ATP&#8209;binding protein RbsA (TC 3.A.1.2.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_654</td><!-- Seq_Id -->
<td align="right">104,318</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">V270I</font>&nbsp;(<font class="mutation_in_codon">G</font>TC&rarr;<font class="mutation_in_codon">A</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Periplasmic aromatic aldehyde oxidoreductase, FAD binding subunit YagS</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_654</td><!-- Seq_Id -->
<td align="right">116,010</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">R601C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">CobN&#8209;like chelatase BtuS for metalloporphyrine salvage</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_654</td><!-- Seq_Id -->
<td align="right">128,782</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">12.9%</td>
<td align="center">intergenic&nbsp;(+122/+191)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i></td>
<td align="left">Regulator of polyketide synthase expression/FIG00958240: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_654</td><!-- Seq_Id -->
<td align="right">128,784</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center">intergenic&nbsp;(+124/+189)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i></td>
<td align="left">Regulator of polyketide synthase expression/FIG00958240: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_655</td><!-- Seq_Id -->
<td align="right">5,862</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">D59E</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Ribose 1,5&#8209;bisphosphate phosphokinase PhnN (EC 2.7.4.23)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_655</td><!-- Seq_Id -->
<td align="right">9,541</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_synonymous">I38I</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Alpha&#8209;D&#8209;ribose 1&#8209;methylphosphonate 5&#8209;phosphate C&#8209;P lyase (EC 4.7.1.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_655</td><!-- Seq_Id -->
<td align="right">48,935</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">R12C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Branched&#8209;chain alpha&#8209;keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_655</td><!-- Seq_Id -->
<td align="right">54,044</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">G387D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_655</td><!-- Seq_Id -->
<td align="right">55,228</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center">intergenic&nbsp;(&#8209;25/+32)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_655</td><!-- Seq_Id -->
<td align="right">69,352</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">15.8%</td>
<td align="center">intergenic&nbsp;(+59/&#8209;70)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">D&#8209;erythro&#8209;7,8&#8209;dihydroneopterin triphosphate epimerase/COGs COG3492</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">4,309</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center">intergenic&nbsp;(+17/&#8209;117)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Protein of unknown function DUF1275/Virulence associated gene hudA (PA0254), similar to UbiD</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">7,645</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">P189Q</font>&nbsp;(C<font class="mutation_in_codon">C</font>G&rarr;C<font class="mutation_in_codon">A</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">10,440</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">L186F</font>&nbsp;(<font class="mutation_in_codon">C</font>TC&rarr;<font class="mutation_in_codon">T</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">P&#8209;hydroxybenzoate hydroxylase (EC 1.14.13.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">36,317</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.3%</td>
<td align="center"><font class="snp_type_synonymous">K220K</font>&nbsp;(AA<font class="mutation_in_codon">G</font>&rarr;AA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Ribulose&#8209;5&#8209;phosphate 4&#8209;epimerase and related epimerases and aldolases</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">86,437</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">11.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">D534G</font>&nbsp;(G<font class="mutation_in_codon">A</font>T&rarr;G<font class="mutation_in_codon">G</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">FIG017431: Sigma factor&#8209;like phosphatase with CBS pair domains</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">86,873</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">Y13H</font>&nbsp;(<font class="mutation_in_codon">T</font>AC&rarr;<font class="mutation_in_codon">C</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">FIG062788: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">100,121</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">17.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">S648N</font>&nbsp;(A<font class="mutation_in_codon">G</font>C&rarr;A<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Multidrug efflux system, inner membrane proton/drug antiporter (RND type) => TriC of TriABC&#8209;OpmH system</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">115,210</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.3%</td>
<td align="center"><font class="snp_type_nonsense">Y127*</font>&nbsp;(TA<font class="mutation_in_codon">C</font>&rarr;TA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Nucleoside ABC transporter, substrate&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">116,308</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.8%</td>
<td align="center"><font class="snp_type_synonymous">L152L</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">T</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Nucleoside 2&#8209;deoxyribosyltransferase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">128,358</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">A217V</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=tgallagh@patricbrc.o rg</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">130,370</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">G360C</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Guanine deaminase (EC 3.5.4.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">142,464</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">A412T</font>&nbsp;(<font class="mutation_in_codon">G</font>CG&rarr;<font class="mutation_in_codon">A</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">C4&#8209;dicarboxylate transport protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">145,992</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">S7N</font>&nbsp;(A<font class="mutation_in_codon">G</font>C&rarr;A<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transcriptional regulator, HxlR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">163,116</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">E74K</font>&nbsp;(<font class="mutation_in_codon">G</font>AG&rarr;<font class="mutation_in_codon">A</font>AG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">3&#8209;oxoacyl&#8209;[acyl&#8209;carrier&#8209;protein] synthase (EC 2.3.1.41)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">167,285</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">16.0%</td>
<td align="center">intergenic&nbsp;(&#8209;122/&#8209;132)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">VgrG protein/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">180,265</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">R250H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">T6SS component TssC (ImpC/VipB)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">188,884</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">T382I</font>&nbsp;(A<font class="mutation_in_codon">C</font>C&rarr;A<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">T6SS component TssM (IcmF/VasK)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">196,845</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">M112I</font>&nbsp;(AT<font class="mutation_in_codon">G</font>&rarr;AT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ABC&#8209;type antimicrobial peptide transport system, permease component</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_663</td><!-- Seq_Id -->
<td align="right">201,344</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.6%</td>
<td align="center"><font class="snp_type_synonymous">E12E</font>&nbsp;(GA<font class="mutation_in_codon">G</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Radical SAM domain protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_664</td><!-- Seq_Id -->
<td align="right">14,203</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">L521P</font>&nbsp;(C<font class="mutation_in_codon">T</font>G&rarr;C<font class="mutation_in_codon">C</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Macrolide export ATP&#8209;binding/permease protein MacB</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_664</td><!-- Seq_Id -->
<td align="right">37,809</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">G453V</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">T6SS AAA+ chaperone ClpV (TssH)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_664</td><!-- Seq_Id -->
<td align="right">66,340</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">15.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">Y70C</font>&nbsp;(T<font class="mutation_in_codon">A</font>C&rarr;T<font class="mutation_in_codon">G</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Acyl&#8209;CoA dehydrogenase; probable dibenzothiophene desulfurization enzyme</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_664</td><!-- Seq_Id -->
<td align="right">76,721</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_synonymous">S350S</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Various polyols ABC transporter, substrate&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_664</td><!-- Seq_Id -->
<td align="right">90,713</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">15.0%</td>
<td align="center">intergenic&nbsp;(&#8209;148/+17)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i></td>
<td align="left">Glyceraldehyde&#8209;3&#8209;phosphate dehydrogenase, putative/FIG00956741: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_664</td><!-- Seq_Id -->
<td align="right">120,053</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">15.0%</td>
<td align="center">coding&nbsp;(916/1452&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Chitinase (EC 3.2.1.14)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_664</td><!-- Seq_Id -->
<td align="right">122,992</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">13.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">D494G</font>&nbsp;(G<font class="mutation_in_codon">A</font>C&rarr;G<font class="mutation_in_codon">G</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adenylylsulfate reductase alpha&#8209;subunit (EC 1.8.99.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">2,564</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.8%</td>
<td align="center"><font class="snp_type_synonymous">V4V</font>&nbsp;(GT<font class="mutation_in_codon">A</font>&rarr;GT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">putative plasmid stabilization protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">2,572</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">T7I</font>&nbsp;(A<font class="mutation_in_codon">C</font>C&rarr;A<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">putative plasmid stabilization protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">2,577</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">A9T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">putative plasmid stabilization protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">2,582</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center"><font class="snp_type_synonymous">R10R</font>&nbsp;(CG<font class="mutation_in_codon">C</font>&rarr;CG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">putative plasmid stabilization protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">4,714</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">Y15C</font>&nbsp;(T<font class="mutation_in_codon">A</font>C&rarr;T<font class="mutation_in_codon">G</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">10,923</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center"><font class="snp_type_synonymous">P35P</font>&nbsp;(CC<font class="mutation_in_codon">G</font>&rarr;CC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Glycosidases</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">15,532</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">15.4%</td>
<td align="center"><font class="snp_type_synonymous">S214S</font>&nbsp;(TC<font class="mutation_in_codon">C</font>&rarr;TC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">23,934</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">A128D</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcriptional regulator, LysR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">47,112</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_synonymous">P188P</font>&nbsp;(CC<font class="mutation_in_codon">G</font>&rarr;CC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Tryptophan 2,3&#8209;dioxygenase (EC 1.13.11.11)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">72,972</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">L70F</font>&nbsp;(<font class="mutation_in_codon">C</font>TC&rarr;<font class="mutation_in_codon">T</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Alkanesulfonates&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">79,934</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">A143V</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Rhodanese&#8209;related sulfurtransferase, 4 domains</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">87,627</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center"><font class="snp_type_synonymous">S237S</font>&nbsp;(TC<font class="mutation_in_codon">G</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Seryl&#8209;tRNA synthetase (EC 6.1.1.11)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">109,101</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_synonymous">A56A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">GNAT family acetyltransferase YjcF</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">125,945</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_synonymous">A100A</font>&nbsp;(GC<font class="mutation_in_codon">C</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">NADH&#8209;ubiquinone oxidoreductase chain J (EC 1.6.5.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">167,448</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">Y634H</font>&nbsp;(<font class="mutation_in_codon">T</font>AT&rarr;<font class="mutation_in_codon">C</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Rhs&#8209;family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">175,255</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_synonymous">I459I</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">TonB&#8209;dependent receptor; Outer membrane receptor for ferric enterobactin and colicins B, D</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">184,613</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A110T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FIG002283: Isochorismatase family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">187,455</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_synonymous">V7V</font>&nbsp;(GT<font class="mutation_in_codon">C</font>&rarr;GT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Outer membrane low permeability porin, OprD family => OccD7/OpdB proline</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">214,359</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">18.8%</td>
<td align="center"><font class="snp_type_synonymous">I510I</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Chaperone protein ClpB (ATP&#8209;dependent unfoldase)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">217,288</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">L659V</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">G</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Superfamily I DNA and RNA helicases and helicase subunits</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_665</td><!-- Seq_Id -->
<td align="right">221,120</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">R1936H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Superfamily I DNA and RNA helicases and helicase subunits</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">11,714</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.0%</td>
<td align="center"><font class="snp_type_nonsense">W431*</font>&nbsp;(T<font class="mutation_in_codon">G</font>G&rarr;T<font class="mutation_in_codon">A</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Aspartyl&#8209;tRNA synthetase (EC 6.1.1.12) @ Aspartyl&#8209;tRNA(Asn) synthetase (EC 6.1.1.23)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">26,682</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">G256D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">46,064</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">14.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">A111T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Signal transduction histidine&#8209;protein kinase BarA (EC 2.7.13.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">52,223</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">V558M</font>&nbsp;(<font class="mutation_in_codon">G</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Predicted hydrolase of alkaline phosphatase superfamily</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">70,560</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center">intergenic&nbsp;(&#8209;54/+131)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Carbon storage regulator/Aspartokinase (EC 2.7.2.4)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">72,391</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">K771E</font>&nbsp;(<font class="mutation_in_codon">A</font>AG&rarr;<font class="mutation_in_codon">G</font>AG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Alanyl&#8209;tRNA synthetase (EC 6.1.1.7)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">89,211</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">G48D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Arginine/ornithine ABC transporter, substrate&#8209;binding protein AotJ</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">111,430</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">A183T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Murein&#8209;DD&#8209;endopeptidase (EC 3.4.99.&#8209;)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">123,810</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">S107N</font>&nbsp;(A<font class="mutation_in_codon">G</font>C&rarr;A<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Thioredoxin</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">130,595</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_nonsense">S296*</font>&nbsp;(T<font class="mutation_in_codon">C</font>G&rarr;T<font class="mutation_in_codon">A</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">135,683</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">R382C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Sensory box/GGDEF family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">136,183</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center"><font class="snp_type_synonymous">H548H</font>&nbsp;(CA<font class="mutation_in_codon">C</font>&rarr;CA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Sensory box/GGDEF family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">148,526</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center">intergenic&nbsp;(&#8209;82/&#8209;61)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">FKBP&#8209;type peptidyl&#8209;prolyl cis&#8209;trans isomerase SlyD (EC 5.2.1.8)/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">149,161</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center"><font class="snp_type_synonymous">L137L</font>&nbsp;(CT<font class="mutation_in_codon">G</font>&rarr;CT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Acetate kinase (EC 2.7.2.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">153,343</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.2%</td>
<td align="center"><font class="snp_type_synonymous">P198P</font>&nbsp;(CC<font class="mutation_in_codon">G</font>&rarr;CC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Outer membrane porin, OmpA family</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_668</td><!-- Seq_Id -->
<td align="right">159,834</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center">intergenic&nbsp;(+238/&#8209;104)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">hypothetical protein/tRNA&#8209;Pseudo&#8209;GAG</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_669</td><!-- Seq_Id -->
<td align="right">1,679</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">D38E</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Uncharacterized protein YebT</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">11,638</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_synonymous">G384G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Respiratory nitrate reductase alpha chain (EC 1.7.99.4)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">16,818</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.3%</td>
<td align="center"><font class="snp_type_synonymous">P355P</font>&nbsp;(CC<font class="mutation_in_codon">C</font>&rarr;CC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Nitrate/nitrite sensor protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">22,574</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A81S</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">T</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">27,091</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">A51S</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">T</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Osmoprotectant ABC transporter, permease protein OsmW</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">93,052</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">V47I</font>&nbsp;(<font class="mutation_in_codon">G</font>TC&rarr;<font class="mutation_in_codon">A</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Nitroreductase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">93,122</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center"><font class="snp_type_synonymous">A23A</font>&nbsp;(GC<font class="mutation_in_codon">T</font>&rarr;GC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Nitroreductase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">97,986</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">A86T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Hybrid sensory histidine kinase in two&#8209;component regulatory system with EvgA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">98,995</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">V176A</font>&nbsp;(G<font class="mutation_in_codon">T</font>C&rarr;G<font class="mutation_in_codon">C</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Two&#8209;component system response regulator protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">109,470</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.9%</td>
<td align="center"><font class="snp_type_synonymous">L231L</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">T</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Endonuclease/exonuclease/phosphatase family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">112,327</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">G459S</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">A</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ATP&#8209;dependent helicase HrpB</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">112,612</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">A364T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ATP&#8209;dependent helicase HrpB</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">114,857</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_synonymous">L170L</font>&nbsp;(CT<font class="mutation_in_codon">G</font>&rarr;CT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Ferrous&#8209;iron efflux pump FieF</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">121,420</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_synonymous">V113V</font>&nbsp;(GT<font class="mutation_in_codon">G</font>&rarr;GT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">AMP nucleosidase (EC 3.2.2.4)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">126,586</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">G161R</font>&nbsp;(<font class="mutation_in_codon">G</font>GA&rarr;<font class="mutation_in_codon">A</font>GA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Two&#8209;component system sensor histidine kinase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">126,656</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.4%</td>
<td align="center"><font class="snp_type_synonymous">N137N</font>&nbsp;(AA<font class="mutation_in_codon">C</font>&rarr;AA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Two&#8209;component system sensor histidine kinase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_670</td><!-- Seq_Id -->
<td align="right">146,741</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.3%</td>
<td align="center"><font class="snp_type_synonymous">A229A</font>&nbsp;(GC<font class="mutation_in_codon">C</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcriptional regulator, LysR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_672</td><!-- Seq_Id -->
<td align="right">16,626</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.8%</td>
<td align="center"><font class="snp_type_synonymous">I4I</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Gliding motility&#8209;associated ABC transporter permease protein GldF</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_672</td><!-- Seq_Id -->
<td align="right">30,103</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">G69D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transcription termination protein NusB</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_672</td><!-- Seq_Id -->
<td align="right">31,159</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">V280F</font>&nbsp;(<font class="mutation_in_codon">G</font>TC&rarr;<font class="mutation_in_codon">T</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">3,4&#8209;dihydroxy&#8209;2&#8209;butanone 4&#8209;phosphate synthase (EC 4.1.99.12) / GTP cyclohydrolase II (EC 3.5.4.25)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_672</td><!-- Seq_Id -->
<td align="right">48,403</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_synonymous">P147P</font>&nbsp;(CC<font class="mutation_in_codon">A</font>&rarr;CC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phenylacetaldehyde dehydrogenase (EC 1.2.1.39)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_672</td><!-- Seq_Id -->
<td align="right">50,876</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">A66V</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_672</td><!-- Seq_Id -->
<td align="right">53,351</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.3%</td>
<td align="center"><font class="snp_type_synonymous">G628G</font>&nbsp;(GG<font class="mutation_in_codon">G</font>&rarr;GG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Polyketide synthase modules and related proteins</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_672</td><!-- Seq_Id -->
<td align="right">56,511</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center">intergenic&nbsp;(+400/+15)</td>
<td align="center"><i>rasttk_feature_annotation_tool=tgallagh@patricbrc.o rg</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/probable fimbrial protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_672</td><!-- Seq_Id -->
<td align="right">75,406</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.3%</td>
<td align="center"><font class="snp_type_synonymous">L100L</font>&nbsp;(CT<font class="mutation_in_codon">G</font>&rarr;CT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_672</td><!-- Seq_Id -->
<td align="right">91,711</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">P1068H</font>&nbsp;(C<font class="mutation_in_codon">C</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Probable sensor/response regulator hybrid (EC 2.7.3.&#8209;)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_672</td><!-- Seq_Id -->
<td align="right">91,775</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">R1047W</font>&nbsp;(<font class="mutation_in_codon">C</font>GG&rarr;<font class="mutation_in_codon">T</font>GG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Probable sensor/response regulator hybrid (EC 2.7.3.&#8209;)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">7,658</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">V96I</font>&nbsp;(<font class="mutation_in_codon">G</font>TC&rarr;<font class="mutation_in_codon">A</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Glutathione&#8209;independent formaldehyde dehydrogenase (EC 1.2.1.46)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">22,717</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">R363H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Pyruvate carboxylase subunit B (biotin&#8209;containing) (EC 6.4.1.1)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">27,609</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center">intergenic&nbsp;(&#8209;68/&#8209;18)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Transcriptional regulator, RpiR family/Glucose&#8209;6&#8209;phosphate 1&#8209;dehydrogenase (EC 1.1.1.49)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">29,258</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">D29N</font>&nbsp;(<font class="mutation_in_codon">G</font>AT&rarr;<font class="mutation_in_codon">A</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Uncharacterized protease YegQ</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">42,696</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">C344S</font>&nbsp;(T<font class="mutation_in_codon">G</font>C&rarr;T<font class="mutation_in_codon">C</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glycosyltransferase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">45,337</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">R340C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">O&#8209;antigen export system, ATP&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">48,482</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">L37M</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Mannose&#8209;1&#8209;phosphate guanylyltransferase (EC 2.7.7.13) / Mannose&#8209;6&#8209;phosphate isomerase (EC 5.3.1.8)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">67,528</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">A50V</font>&nbsp;(G<font class="mutation_in_codon">C</font>A&rarr;G<font class="mutation_in_codon">T</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">FIG00955342: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">73,072</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.8%</td>
<td align="center">intergenic&nbsp;(+21/&#8209;195)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">FIG00965800: hypothetical protein/Uncharacterized MFS&#8209;type transporter</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">74,781</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">P259S</font>&nbsp;(<font class="mutation_in_codon">C</font>CG&rarr;<font class="mutation_in_codon">T</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Gate supefamily SpmA/SpmB domains protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">77,783</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">R131H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Inhibitor of vertebrate c&#8209;type lysozyme, periplasmic => Ivy</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">88,564</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">F72S</font>&nbsp;(T<font class="mutation_in_codon">T</font>C&rarr;T<font class="mutation_in_codon">C</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">GTP&#8209;binding protein EngB</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">90,003</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">R667H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">DNA polymerase I (EC 2.7.7.7)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">95,199</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">G452S</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">A</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Ribonucleotide reductase of class II (coenzyme B12&#8209;dependent) (EC 1.17.4.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">96,855</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">A244D</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Zinc ABC transporter, substrate&#8209;binding protein ZnuA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">109,206</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_synonymous">L240L</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">T</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">C4&#8209;dicarboxylate transport transcriptional regulatory protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">118,864</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">L200M</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Excinuclease ATPase subunit</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">121,264</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A4T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">glutamine synthetase family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">123,346</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">G188S</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">A</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Oxidoreductase, short&#8209;chain dehydrogenase/reductase family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">129,086</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">T146I</font>&nbsp;(A<font class="mutation_in_codon">C</font>C&rarr;A<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Dicarboxylate MFS transporter</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">135,472</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_synonymous">L5L</font>&nbsp;(CT<font class="mutation_in_codon">C</font>&rarr;CT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Glutamine synthetase adenylyltransferase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">146,828</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">R159H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Phosphoserine phosphatase</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">165,747</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center">intergenic&nbsp;(&#8209;358/+125)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/tRNA&#8209;5&#8209;carboxymethylaminomethyl&#8209;2&#8209; thiouridine(34) synthesis protein MnmE</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">193,690</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_synonymous">P304P</font>&nbsp;(CC<font class="mutation_in_codon">G</font>&rarr;CC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Rossmann fold nucleotide&#8209;binding protein Smf possibly involved in DNA uptake</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_673</td><!-- Seq_Id -->
<td align="right">195,865</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">15.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">A75T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Coproporphyrinogen III oxidase, aerobic (EC 1.3.3.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_693</td><!-- Seq_Id -->
<td align="right">8,813</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_synonymous">G30G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_693</td><!-- Seq_Id -->
<td align="right">21,962</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center">intergenic&nbsp;(+68/&#8209;49)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">PqsE, quinolone signal response protein/Anthranilate synthase, aminase component (EC 4.1.3.27) [PQS biosynthesis]</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_693</td><!-- Seq_Id -->
<td align="right">32,772</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">A196S</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">T</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Outer membrane beta&#8209;barrel assembly protein BamC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_693</td><!-- Seq_Id -->
<td align="right">67,089</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">L92F</font>&nbsp;(<font class="mutation_in_codon">C</font>TC&rarr;<font class="mutation_in_codon">T</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phosphoribosylcarboxyaminoimidazole (NCAIR) mutase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_693</td><!-- Seq_Id -->
<td align="right">78,822</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">16.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">G356V</font>&nbsp;(G<font class="mutation_in_codon">G</font>A&rarr;G<font class="mutation_in_codon">T</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Glycerate kinase (EC 2.7.1.31)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_693</td><!-- Seq_Id -->
<td align="right">82,733</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_synonymous">S877S</font>&nbsp;(TC<font class="mutation_in_codon">T</font>&rarr;TC<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Na(+) H(+) antiporter subunit A / Na(+) H(+) antiporter subunit B</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_693</td><!-- Seq_Id -->
<td align="right">136,374</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">T73I</font>&nbsp;(A<font class="mutation_in_codon">C</font>C&rarr;A<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_693</td><!-- Seq_Id -->
<td align="right">144,476</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">13.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q223R</font>&nbsp;(C<font class="mutation_in_codon">A</font>G&rarr;C<font class="mutation_in_codon">G</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Efflux ABC transporter, permease/ATP&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_693</td><!-- Seq_Id -->
<td align="right">155,414</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.6%</td>
<td align="center"><font class="snp_type_synonymous">L235L</font>&nbsp;(CT<font class="mutation_in_codon">G</font>&rarr;CT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Deoxyguanosinetriphosphate triphosphohydrolase (EC 3.1.5.1), subgroup 1</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_693</td><!-- Seq_Id -->
<td align="right">158,996</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center">coding&nbsp;(212/912&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcriptional regulator, LysR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_693</td><!-- Seq_Id -->
<td align="right">160,970</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">R48L</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">RhlC, TDP&#8209;rhamnosyltransferase 2 (EC 2.4.1.&#8209;)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_693</td><!-- Seq_Id -->
<td align="right">164,950</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">A131V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;lactate dehydratase (EC 4.2.1.130)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_693</td><!-- Seq_Id -->
<td align="right">182,851</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">14.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">V114I</font>&nbsp;(<font class="mutation_in_codon">G</font>TT&rarr;<font class="mutation_in_codon">A</font>TT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">colicin/pyosin nuclease family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_698</td><!-- Seq_Id -->
<td align="right">2,148</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_synonymous">D213D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Exonuclease SbcC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_698</td><!-- Seq_Id -->
<td align="right">15,198</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_synonymous">P35P</font>&nbsp;(CC<font class="mutation_in_codon">G</font>&rarr;CC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Lipoate&#8209;protein ligase A</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_703</td><!-- Seq_Id -->
<td align="right">380</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center">intergenic&nbsp;(&#8211;/+628)</td>
<td align="center"><i>&#8211;</i>&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">&#8211;/Putative large exoprotein involved in heme utilization or adhesion of ShlA/HecA/FhaA family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_704</td><!-- Seq_Id -->
<td align="right">55</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">S1055R</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Large exoproteins involved in heme utilization or adhesion</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_704</td><!-- Seq_Id -->
<td align="right">7,221</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center">intergenic&nbsp;(+44/&#8209;309)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Lysine&#8209;specific permease/Putative transmembrane protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_704</td><!-- Seq_Id -->
<td align="right">21,373</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center">intergenic&nbsp;(&#8209;52/+31)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Aminopeptidase N/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_704</td><!-- Seq_Id -->
<td align="right">44,320</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">R281H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glutamyl&#8209;tRNA reductase (EC 1.2.1.70)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_704</td><!-- Seq_Id -->
<td align="right">63,565</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">A171T</font>&nbsp;(<font class="mutation_in_codon">G</font>CG&rarr;<font class="mutation_in_codon">A</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_71</td><!-- Seq_Id -->
<td align="right">5,022</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">11.7%</td>
<td align="center">intergenic&nbsp;(&#8209;183/+6)</td>
<td align="center"><i>rasttk_feature_annotation_tool=tgallagh@patricbrc.o rg</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/Mobile element protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_714</td><!-- Seq_Id -->
<td align="right">35,493</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center"><font class="snp_type_synonymous">S106S</font>&nbsp;(TC<font class="mutation_in_codon">C</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_714</td><!-- Seq_Id -->
<td align="right">43,622</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">15.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A292V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Dihydrolipoamide dehydrogenase of branched&#8209;chain alpha&#8209;keto acid dehydrogenase (EC 1.8.1.4)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_714</td><!-- Seq_Id -->
<td align="right">53,315</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">20.0%</td>
<td align="center">intergenic&nbsp;(&#8209;65/+18)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Pellicle/biofilm biosynthesis inner membrane protein PslL, acetyltransferase/Pellicle/biofilm biosynthesis inner membrane protein PslK, MATE transporter family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_714</td><!-- Seq_Id -->
<td align="right">67,089</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">R176H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Pellicle/biofilm biosynthesis protein PslA, polyprenyl glycosylphosphotransferase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_714</td><!-- Seq_Id -->
<td align="right">77,242</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A118T</font>&nbsp;(<font class="mutation_in_codon">G</font>CG&rarr;<font class="mutation_in_codon">A</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transposase</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_714</td><!-- Seq_Id -->
<td align="right">87,047</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center">intergenic&nbsp;(+788/+223)</td>
<td align="center"><i>rasttk_feature_annotation_tool=tgallagh@patricbrc.o rg</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/Tn4652, transposase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_715</td><!-- Seq_Id -->
<td align="right">2,627</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">R100W</font>&nbsp;(<font class="mutation_in_codon">C</font>GG&rarr;<font class="mutation_in_codon">T</font>GG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">General secretion pathway protein K</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_715</td><!-- Seq_Id -->
<td align="right">4,140</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_synonymous">I132I</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">General secretion pathway protein H</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_715</td><!-- Seq_Id -->
<td align="right">6,000</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">14.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">V71M</font>&nbsp;(<font class="mutation_in_codon">G</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">General secretion pathway protein F</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_715</td><!-- Seq_Id -->
<td align="right">13,714</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">16.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">G166D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Amidophosphoribosyltransferase (EC 2.4.2.14)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_715</td><!-- Seq_Id -->
<td align="right">30,918</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_nonsense">W205*</font>&nbsp;(T<font class="mutation_in_codon">G</font>G&rarr;T<font class="mutation_in_codon">A</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transcriptional regulator PA3124, LysR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_715</td><!-- Seq_Id -->
<td align="right">32,441</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">G267S</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">A</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_715</td><!-- Seq_Id -->
<td align="right">33,134</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">11.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">K111E</font>&nbsp;(<font class="mutation_in_codon">A</font>AG&rarr;<font class="mutation_in_codon">G</font>AG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">16 kDa heat shock protein A</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_715</td><!-- Seq_Id -->
<td align="right">34,524</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">A28V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Oxidoreductase, short&#8209;chain dehydrogenase/reductase family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_715</td><!-- Seq_Id -->
<td align="right">44,339</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">A353S</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">T</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Inner&#8209;membrane proton/drug antiporter (MSF type) of tripartite multidrug efflux system</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_715</td><!-- Seq_Id -->
<td align="right">48,267</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center">intergenic&nbsp;(+25/&#8209;44)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">Biosynthetic Aromatic amino acid aminotransferase alpha (EC 2.6.1.57) @ Aromatic&#8209;amino&#8209;acid aminotransferase (EC 2.6.1.57)/tRNA&#8209;Asn&#8209;GTT</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_715</td><!-- Seq_Id -->
<td align="right">53,178</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">L190F</font>&nbsp;(TT<font class="mutation_in_codon">G</font>&rarr;TT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Undecaprenyl&#8209;phosphate alpha&#8209;N&#8209;acetylglucosaminyl 1&#8209;phosphate transferase (EC 2.7.8.33)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_715</td><!-- Seq_Id -->
<td align="right">66,867</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">V36M</font>&nbsp;(<font class="mutation_in_codon">G</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">O&#8209;antigen acetylase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_715</td><!-- Seq_Id -->
<td align="right">67,956</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.8%</td>
<td align="center"><font class="snp_type_synonymous">Y12Y</font>&nbsp;(TA<font class="mutation_in_codon">C</font>&rarr;TA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">UDP&#8209;2&#8209;acetamido&#8209;2&#8209;deoxy&#8209;D&#8209;glucuronic acid dehydrogenase (NAD+) (EC 1.1.1.335)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_715</td><!-- Seq_Id -->
<td align="right">95,155</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center">intergenic&nbsp;(+2/+88)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">MutT/Nudix family protein/4&#8209;hydroxy&#8209;2&#8209;oxoglutarate aldolase (EC 4.1.3.16) @ 2&#8209;dehydro&#8209;3&#8209;deoxyphosphogluconate aldolase (EC 4.1.2.14)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_715</td><!-- Seq_Id -->
<td align="right">95,787</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">A40D</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">4&#8209;hydroxy&#8209;2&#8209;oxoglutarate aldolase (EC 4.1.3.16) @ 2&#8209;dehydro&#8209;3&#8209;deoxyphosphogluconate aldolase (EC 4.1.2.14)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">24,820</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">A44V</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Two&#8209;component transcriptional response regulator, OmpR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">27,307</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_synonymous">S678S</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">41,914</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">R1458H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">NAD&#8209;specific glutamate dehydrogenase (EC 1.4.1.2), large form</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">46,571</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">R261H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Pellicle/biofilm biosynthesis protein PelB</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">48,660</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_synonymous">G957G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Pellicle/biofilm biosynthesis protein PelB</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">59,857</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">R279W</font>&nbsp;(<font class="mutation_in_codon">C</font>GG&rarr;<font class="mutation_in_codon">T</font>GG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">69,300</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">R332C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative two&#8209;component sensor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">78,957</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">D15Y</font>&nbsp;(<font class="mutation_in_codon">G</font>AC&rarr;<font class="mutation_in_codon">T</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcriptional regulator, AcrR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">89,324</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">9.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">A101G</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">G</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Glycerol kinase (EC 2.7.1.30)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">109,932</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.0%</td>
<td align="center">intergenic&nbsp;(+49/&#8209;153)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Predicted transcriptional regulator for fatty acid degradation FadQ, TetR family/beta&#8209;N&#8209;acetylglucosaminidase (EC 3.2.1.52)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">115,612</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_synonymous">A120A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transcription&#8209;repair coupling factor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">118,756</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">V362I</font>&nbsp;(<font class="mutation_in_codon">G</font>TA&rarr;<font class="mutation_in_codon">A</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Aromatic amino acid transport protein AroP</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">124,170</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">C70Y</font>&nbsp;(T<font class="mutation_in_codon">G</font>C&rarr;T<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Na(+)&#8209;translocating NADH&#8209;quinone reductase subunit F (EC 1.6.5.8)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">136,184</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">R37H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">MotA/TolQ/ExbB proton channel family protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">145,038</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">19.3%</td>
<td align="center"><font class="snp_type_synonymous">K173K</font>&nbsp;(AA<font class="mutation_in_codon">G</font>&rarr;AA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">LSU rRNA pseudouridine(955/2504/2580) synthase (EC 5.4.99.24)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">147,741</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">R19C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Maf&#8209;like protein YceF</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">150,123</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q123K</font>&nbsp;(<font class="mutation_in_codon">C</font>AG&rarr;<font class="mutation_in_codon">A</font>AG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Malonyl CoA&#8209;acyl carrier protein transacylase (EC 2.3.1.39)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">160,508</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">P101L</font>&nbsp;(C<font class="mutation_in_codon">C</font>G&rarr;C<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Fe&#8209;S oxidoreductase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">167,029</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">13.1%</td>
<td align="center"><font class="snp_type_synonymous">P260P</font>&nbsp;(CC<font class="mutation_in_codon">T</font>&rarr;CC<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Enoyl&#8209;[acyl&#8209;carrier&#8209;protein] reductase [NADH] (EC 1.3.1.9), FabV => refractory to triclosan</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">177,549</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center">intergenic&nbsp;(&#8209;183/+116)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=tgallagh@patricbrc.o rg</i></td>
<td align="left">2&#8209;keto&#8209;3&#8209;deoxy&#8209;D&#8209;arabino&#8209;heptulosonate&#8209;7&#8209; phosphate synthase I alpha (EC 2.5.1.54)/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">189,863</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.3%</td>
<td align="center"><font class="snp_type_synonymous">R74R</font>&nbsp;(CG<font class="mutation_in_codon">G</font>&rarr;CG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcriptional regulator, AcrR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">194,137</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_synonymous">E242E</font>&nbsp;(GA<font class="mutation_in_codon">G</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">219,193</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">28.6%</td>
<td align="center"><font class="snp_type_synonymous">L234L</font>&nbsp;(CT<font class="mutation_in_codon">G</font>&rarr;CT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Proposed precorrin&#8209;4 hydrolase (analogous to cobalt&#8209;precorrin&#8209;5A hydrolase) / Precorrin&#8209;3B C(17)&#8209;methyltransferase (EC 2.1.1.131)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">227,340</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">R40G</font>&nbsp;(<font class="mutation_in_codon">A</font>GG&rarr;<font class="mutation_in_codon">G</font>GG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Translation initiation factor 2 (IF&#8209;2; GTPase)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">232,028</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">A161V</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Geranyl&#8209;CoA carboxylase biotin&#8209;containing subunit (EC 6.4.1.5)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">242,971</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.1%</td>
<td align="center"><font class="snp_type_synonymous">A124A</font>&nbsp;(GC<font class="mutation_in_codon">C</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Two&#8209;component system response regulator protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">259,165</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">R238C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transglycosylase, Slt family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">265,305</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">P533L</font>&nbsp;(C<font class="mutation_in_codon">C</font>C&rarr;C<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ABC transporter, fused permease protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">267,771</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">16.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">R59Q</font>&nbsp;(C<font class="mutation_in_codon">G</font>G&rarr;C<font class="mutation_in_codon">A</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Arylesterase precursor (EC 3.1.1.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">279,031</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">D68N</font>&nbsp;(<font class="mutation_in_codon">G</font>AT&rarr;<font class="mutation_in_codon">A</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">2&#8209;keto&#8209;3&#8209;deoxy&#8209;D&#8209;arabino&#8209;heptulosonate&#8209;7&#8209; phosphate synthase II (EC 2.5.1.54)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">279,052</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">A61T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">2&#8209;keto&#8209;3&#8209;deoxy&#8209;D&#8209;arabino&#8209;heptulosonate&#8209;7&#8209; phosphate synthase II (EC 2.5.1.54)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">284,827</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">18.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">V121A</font>&nbsp;(G<font class="mutation_in_codon">T</font>C&rarr;G<font class="mutation_in_codon">C</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transcriptional regulator, LysR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">346,540</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">27.6%</td>
<td align="center"><font class="snp_type_synonymous">G260G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Inner membrane protein YfdC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">347,012</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">16.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">R347H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Gamma&#8209;glutamyl&#8209;putrescine oxidase (EC1.4.3.&#8209;)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">350,553</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center">intergenic&nbsp;(&#8209;111/&#8209;50)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">FIG00955364: hypothetical protein/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">354,793</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.9%</td>
<td align="center"><font class="snp_type_synonymous">I140I</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Enoyl&#8209;CoA hydratase (EC 4.2.1.17)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">363,361</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_synonymous">A30A</font>&nbsp;(GC<font class="mutation_in_codon">C</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">364,676</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_synonymous">Q45Q</font>&nbsp;(CA<font class="mutation_in_codon">G</font>&rarr;CA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Proteinase inhibitor I11, ecotin precursor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">384,480</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">G107D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">385,899</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">17.4%</td>
<td align="center">intergenic&nbsp;(+95/&#8209;566)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_717</td><!-- Seq_Id -->
<td align="right">387,154</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">G129D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Fic domain protein, KPN_03553 type</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">888</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center">coding&nbsp;(492/1710&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">4,201</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center">intergenic&nbsp;(&#8209;21/&#8209;1)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">FIG00954510: hypothetical protein/Xanthine/uracil permease family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">13,455</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">L107P</font>&nbsp;(C<font class="mutation_in_codon">T</font>G&rarr;C<font class="mutation_in_codon">C</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">D&#8209;glycerate 2&#8209;kinase (EC 2.7.1.165)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">30,558</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">A199T</font>&nbsp;(<font class="mutation_in_codon">G</font>CA&rarr;<font class="mutation_in_codon">A</font>CA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Uncharacterized amino acid permease, GabP family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">34,710</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">A42T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Cytochrome c&#8209;type biogenesis protein CcmG/DsbE, thiol:disulfide oxidoreductase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">37,900</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">L109M</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Cytochrome c&#8209;type biogenesis protein CcmC, putative heme lyase for CcmE</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">105,593</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.9%</td>
<td align="center"><font class="snp_type_synonymous">S44S</font>&nbsp;(TC<font class="mutation_in_codon">C</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">107,993</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.8%</td>
<td align="center"><font class="snp_type_nonsense">Q24*</font>&nbsp;(<font class="mutation_in_codon">C</font>AA&rarr;<font class="mutation_in_codon">T</font>AA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Acetylpolyamine aminohydrolase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">135,057</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_synonymous">G288G</font>&nbsp;(GG<font class="mutation_in_codon">T</font>&rarr;GG<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">3&#8209;oxoacyl&#8209;[acyl&#8209;carrier&#8209;protein] synthase, KASII (EC 2.3.1.179)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">135,062</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">R290H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">3&#8209;oxoacyl&#8209;[acyl&#8209;carrier&#8209;protein] synthase, KASII (EC 2.3.1.179)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">135,235</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">G348S</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">A</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">3&#8209;oxoacyl&#8209;[acyl&#8209;carrier&#8209;protein] synthase, KASII (EC 2.3.1.179)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">150,370</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.8%</td>
<td align="center"><font class="snp_type_nonsense">Q151*</font>&nbsp;(<font class="mutation_in_codon">C</font>AA&rarr;<font class="mutation_in_codon">T</font>AA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Thioredoxin reductase (EC 1.8.1.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">160,220</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">L37P</font>&nbsp;(C<font class="mutation_in_codon">T</font>G&rarr;C<font class="mutation_in_codon">C</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcriptional regulator, Xre family</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">204,240</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.8%</td>
<td align="center">intergenic&nbsp;(+191/+22)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">FIG00954552: hypothetical protein/Uncharacterized MFS&#8209;type transporter</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">204,372</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.9%</td>
<td align="center"><font class="snp_type_synonymous">L478L</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">T</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Uncharacterized MFS&#8209;type transporter</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">214,039</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">A60T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Bis(5'&#8209;nucleosyl)&#8209;tetraphosphatase (asymmetrical) (EC 3.6.1.17)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">217,002</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center">intergenic&nbsp;(+49/&#8209;28)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Protease II (EC 3.4.21.83)/Signal peptidase I (EC 3.4.21.89)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">241,128</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">T115N</font>&nbsp;(A<font class="mutation_in_codon">C</font>C&rarr;A<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Nicotinate&#8209;nucleotide&#8209;&#8209;dimethylbenzimidazole phosphoribosyltransferase (EC 2.4.2.21)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">255,613</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.0%</td>
<td align="center"><font class="snp_type_synonymous">Y5Y</font>&nbsp;(TA<font class="mutation_in_codon">C</font>&rarr;TA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative oxidoreductase in 4&#8209;hydroxyproline catabolic gene cluster</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">267,572</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.3%</td>
<td align="center"><font class="snp_type_synonymous">L117L</font>&nbsp;(CT<font class="mutation_in_codon">G</font>&rarr;CT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">trans&#8209;3&#8209;hydroxy&#8209;L&#8209;proline dehydratase (EC 4.2.1.77)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">275,637</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">A143V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Secreted alkaline metalloproteinase (EC 3.4.24.&#8209;), PrtA/B/C/G homolog</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">281,264</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">21.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">G339D</font>&nbsp;(G<font class="mutation_in_codon">G</font>T&rarr;G<font class="mutation_in_codon">A</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">285,986</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_synonymous">H705H</font>&nbsp;(CA<font class="mutation_in_codon">T</font>&rarr;CA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">probable sensor/response regulator hybrid</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">286,115</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">15.4%</td>
<td align="center"><font class="snp_type_synonymous">D748D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">probable sensor/response regulator hybrid</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">288,190</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_synonymous">P18P</font>&nbsp;(CC<font class="mutation_in_codon">T</font>&rarr;CC<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Subtilisin&#8209;like serine proteases</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">295,614</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_synonymous">Q146Q</font>&nbsp;(CA<font class="mutation_in_codon">G</font>&rarr;CA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transcriptional regulator, AraC family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">312,786</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">V475F</font>&nbsp;(<font class="mutation_in_codon">G</font>TC&rarr;<font class="mutation_in_codon">T</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">333,442</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">D130E</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcriptional regulator, LysR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">333,488</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">23.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">I146V</font>&nbsp;(<font class="mutation_in_codon">A</font>TC&rarr;<font class="mutation_in_codon">G</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcriptional regulator, LysR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">336,809</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">I444T</font>&nbsp;(A<font class="mutation_in_codon">T</font>T&rarr;A<font class="mutation_in_codon">C</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transcriptional regulator</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">352,383</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">17.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">R1086C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Sensory box/GGDEF family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">364,696</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center">intergenic&nbsp;(&#8209;221/&#8209;236)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Membrane&#8209;bound lytic murein transglycosylase B/Membrane protein TerC, possibly involved in tellurium resistance</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">369,604</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">17.1%</td>
<td align="center"><font class="snp_type_synonymous">D31D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Dienelactone hydrolase family protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_718</td><!-- Seq_Id -->
<td align="right">378,716</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">A299V</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Two&#8209;component system sensor histidine kinase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_725</td><!-- Seq_Id -->
<td align="right">5,518</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">L131S</font>&nbsp;(T<font class="mutation_in_codon">T</font>G&rarr;T<font class="mutation_in_codon">C</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_725</td><!-- Seq_Id -->
<td align="right">7,639</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">17.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">R10C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&rarr;</td>
<td align="left">FIG00954084: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_725</td><!-- Seq_Id -->
<td align="right">14,211</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">L116P</font>&nbsp;(C<font class="mutation_in_codon">T</font>G&rarr;C<font class="mutation_in_codon">C</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Methyl&#8209;accepting chemotaxis sensor/transducer protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_725</td><!-- Seq_Id -->
<td align="right">37,186</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">A488T</font>&nbsp;(<font class="mutation_in_codon">G</font>CG&rarr;<font class="mutation_in_codon">A</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Inner membrane component of TAM transport system</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_725</td><!-- Seq_Id -->
<td align="right">42,794</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">G262D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Ser/Thr and Tyr protein phosphatase (dual specificity)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_725</td><!-- Seq_Id -->
<td align="right">44,173</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">G120S</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">A</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Acyl&#8209;CoA:1&#8209;acyl&#8209;sn&#8209;glycerol&#8209;3&#8209;phosphate acyltransferase (EC 2.3.1.51)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_725</td><!-- Seq_Id -->
<td align="right">46,534</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">16.2%</td>
<td align="center"><font class="snp_type_synonymous">P11P</font>&nbsp;(CC<font class="mutation_in_codon">T</font>&rarr;CC<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Oxidoreductase, aldo/keto reductase family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_725</td><!-- Seq_Id -->
<td align="right">74,116</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">V221M</font>&nbsp;(<font class="mutation_in_codon">G</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Benzoate 1,2&#8209;dioxygenase alpha subunit (EC 1.14.12.10)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_725</td><!-- Seq_Id -->
<td align="right">79,275</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">G266V</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Benzoate 1,2&#8209;dioxygenase alpha subunit (EC 1.14.12.10)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_725</td><!-- Seq_Id -->
<td align="right">80,158</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">A82V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">transcriptional activator</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center">k141_725</td><!-- Seq_Id -->
<td align="right">102,239</td><!-- Position -->
<td align="center">&Delta;19&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">100%</td>
<td align="center">coding&nbsp;(102&#8209;120/1020&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative oxidoreductase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_725</td><!-- Seq_Id -->
<td align="right">106,395</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">S68G</font>&nbsp;(<font class="mutation_in_codon">A</font>GC&rarr;<font class="mutation_in_codon">G</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcriptional regulator, AcrR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_725</td><!-- Seq_Id -->
<td align="right">108,773</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">26.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">A17T</font>&nbsp;(<font class="mutation_in_codon">G</font>CT&rarr;<font class="mutation_in_codon">A</font>CT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">FIG002261: Cytochrome c family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_725</td><!-- Seq_Id -->
<td align="right">128,228</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_synonymous">G48G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_729</td><!-- Seq_Id -->
<td align="right">13,515</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">A177V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">MBL&#8209;fold metallo&#8209;hydrolase superfamily</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_729</td><!-- Seq_Id -->
<td align="right">58,759</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_synonymous">P276P</font>&nbsp;(CC<font class="mutation_in_codon">G</font>&rarr;CC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Iron compound ABC transporter, permease protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_729</td><!-- Seq_Id -->
<td align="right">59,605</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.2%</td>
<td align="center">intergenic&nbsp;(&#8209;19/+27)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Iron compound ABC transporter, permease protein/Ferrienterobactin&#8209;binding periplasmic protein FepB (TC 3.A.1.14.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_729</td><!-- Seq_Id -->
<td align="right">64,807</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">A49T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Coenzyme F420&#8209;dependent N5,N10&#8209;methylene tetrahydromethanopterin reductase and related flavin&#8209;dependent oxidoreductases</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">4,727</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_synonymous">T187T</font>&nbsp;(AC<font class="mutation_in_codon">C</font>&rarr;AC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Fatty acid desaturase</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">36,719</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">G330D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Magnesium and cobalt transport protein CorA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">37,522</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">H237N</font>&nbsp;(<font class="mutation_in_codon">C</font>AC&rarr;<font class="mutation_in_codon">A</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">STAPHYLOLYTIC protease PREPROENZYME LASA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">41,540</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center"><font class="snp_type_synonymous">S393S</font>&nbsp;(TC<font class="mutation_in_codon">C</font>&rarr;TC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">General secretion pathway protein D / Type II secretion outermembrane pore forming protein (PulD)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">44,552</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_synonymous">P466P</font>&nbsp;(CC<font class="mutation_in_codon">G</font>&rarr;CC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">DinG family ATP&#8209;dependent helicase CPE1197</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">52,811</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">Y273C</font>&nbsp;(T<font class="mutation_in_codon">A</font>C&rarr;T<font class="mutation_in_codon">G</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcriptional regulator PA1859, LysR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">76,364</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.8%</td>
<td align="center"><font class="snp_type_synonymous">M1M</font>&nbsp;(AT<font class="mutation_in_codon">G</font>&rarr;AT<font class="mutation_in_codon">A</font>)&nbsp;&dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">FIG00956360: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">84,152</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center">intergenic&nbsp;(+67/&#8209;191)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Putative sterol carrier protein/acyl&#8209;CoA dehydrogenase, putative phosphotransferase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">90,188</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">A101V</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">NADH pyrophosphatase (EC 3.6.1.22), decaps 5'&#8209;NAD modified RNA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">99,754</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center">intergenic&nbsp;(&#8209;30/+35)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">DNA polymerase III epsilon subunit (EC 2.7.7.7)/Ribonuclease HI (EC 3.1.26.4)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">149,154</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">R579H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Assimilatory nitrate reductase large subunit (EC 1.7.99.4)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">151,064</td><!-- Position -->
<td align="center">19&nbsp;bp&rarr;19&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">8.6%</td>
<td align="center">coding&nbsp;(934&#8209;952/1053&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Nonspecific porin and structural outer membrane protein OprF</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">152,194</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">19.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">A143V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">RNA polymerase ECF&#8209;type sigma factor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">167,264</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_synonymous">S43S</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FIG00973752: TolA&#8209;like membrane protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">182,619</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">14.3%</td>
<td align="center"><font class="snp_type_synonymous">T58T</font>&nbsp;(AC<font class="mutation_in_codon">C</font>&rarr;AC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">203,121</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">M188I</font>&nbsp;(AT<font class="mutation_in_codon">G</font>&rarr;AT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Sensory box/GGDEF family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">208,416</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">T4M</font>&nbsp;(A<font class="mutation_in_codon">C</font>G&rarr;A<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Type III secretion cytoplasmic protein (YscK)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">209,547</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">R134H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Type III secretion effector protein (YopR, encoded by YscH)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">214,017</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_synonymous">R76R</font>&nbsp;(CG<font class="mutation_in_codon">G</font>&rarr;CG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Type III secretion chaperone protein for YopN (SycN,YscB)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">251,084</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_synonymous">G1060G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">T6SS component TssM (IcmF/VasK)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_730</td><!-- Seq_Id -->
<td align="right">255,464</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">G340D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">T6SS component TssK (ImpJ/VasE)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_732</td><!-- Seq_Id -->
<td align="right">1,560</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center">intergenic&nbsp;(+103/&#8209;68)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Phosphate ABC transporter, substrate&#8209;binding protein PstS (TC 3.A.1.7.1)/Phosphate ABC transporter, permease protein PstC (TC 3.A.1.7.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_732</td><!-- Seq_Id -->
<td align="right">25,959</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_synonymous">S245S</font>&nbsp;(TC<font class="mutation_in_codon">G</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ATP&#8209;dependent DNA helicase RecG (EC 3.6.4.12)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_732</td><!-- Seq_Id -->
<td align="right">37,213</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_synonymous">G112G</font>&nbsp;(GG<font class="mutation_in_codon">G</font>&rarr;GG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Exodeoxyribonuclease III (EC 3.1.11.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_732</td><!-- Seq_Id -->
<td align="right">37,287</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">A88T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Exodeoxyribonuclease III (EC 3.1.11.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_732</td><!-- Seq_Id -->
<td align="right">48,143</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_synonymous">Q119Q</font>&nbsp;(CA<font class="mutation_in_codon">G</font>&rarr;CA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phosphoglucomutase (EC 5.4.2.2) @ Phosphomannomutase (EC 5.4.2.8)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_732</td><!-- Seq_Id -->
<td align="right">61,005</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_synonymous">G183G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FAD&#8209;dependent oxidoreductase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_732</td><!-- Seq_Id -->
<td align="right">64,053</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A561T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative uncharacterized protein ydbH</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_732</td><!-- Seq_Id -->
<td align="right">74,405</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_synonymous">I37I</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Pyruvate dehydrogenase (quinone) (EC 1.2.5.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_732</td><!-- Seq_Id -->
<td align="right">98,815</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">14.6%</td>
<td align="center"><font class="snp_type_synonymous">G53G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&rarr;</td>
<td align="left">FIG00953342: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">1,233</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.1%</td>
<td align="center"><font class="snp_type_synonymous">P41P</font>&nbsp;(CC<font class="mutation_in_codon">C</font>&rarr;CC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Glycosyltransferase</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">9,900</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">16.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">H294D</font>&nbsp;(<font class="mutation_in_codon">C</font>AT&rarr;<font class="mutation_in_codon">G</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">18,573</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.1%</td>
<td align="center"><font class="snp_type_synonymous">V32V</font>&nbsp;(GT<font class="mutation_in_codon">C</font>&rarr;GT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Pyruvate kinase (EC 2.7.1.40)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">31,872</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">A28V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transcriptional regulator, IclR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">50,763</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center">coding&nbsp;(967/990&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Putative oxidoreductase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">60,705</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">14.3%</td>
<td align="center"><font class="snp_type_synonymous">S295S</font>&nbsp;(TC<font class="mutation_in_codon">C</font>&rarr;TC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Iron&#8209;regulated protein A precursor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">62,642</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A440V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Probable thiol oxidoreductase with 2 cytochrome c heme&#8209;binding sites</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">62,912</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">S53N</font>&nbsp;(A<font class="mutation_in_codon">G</font>C&rarr;A<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Iron&#8209;regulated protein A precursor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">73,336</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_synonymous">G148G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Putative two&#8209;component sensor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">78,328</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">A143T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Heat shock protein 60 kDa family chaperone GroEL</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">79,221</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.8%</td>
<td align="center">intergenic&nbsp;(&#8209;123/+73)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Heat shock protein 10 kDa family chaperone GroES/Protein affecting phage T7 exclusion by the F plasmid</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">94,439</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.2%</td>
<td align="center"><font class="snp_type_synonymous">S305S</font>&nbsp;(TC<font class="mutation_in_codon">G</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glutamate N&#8209;acetyltransferase (EC 2.3.1.35) @ N&#8209;acetylglutamate synthase (EC 2.3.1.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">109,549</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">W254C</font>&nbsp;(TG<font class="mutation_in_codon">G</font>&rarr;TG<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">UDP&#8209;N&#8209;acetylmuramoyl&#8209;L&#8209;alanine&#8209;&#8209;D&#8209;glutamate ligase (EC 6.3.2.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">115,673</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">15.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">A102V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Cell division protein FtsI [Peptidoglycan synthetase] (EC 2.4.1.129)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">118,241</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center">intergenic&nbsp;(&#8209;629/+127)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Transcriptional regulator MraZ/16S rRNA (cytidine(1402)&#8209;2'&#8209;O)&#8209;methyltransferase (EC 2.1.1.198)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">135,086</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">H193Y</font>&nbsp;(<font class="mutation_in_codon">C</font>AT&rarr;<font class="mutation_in_codon">T</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Hydrolase PA4440, alpha/beta fold family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">139,434</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center">intergenic&nbsp;(&#8209;66/&#8209;68)</td>
<td align="center"><i>rasttk_feature_annotation_tool=tgallagh@patricbrc.o rg</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/Membrane&#8209;bound lytic murein transglycosylase B</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">178,607</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_synonymous">E480E</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Aspartyl&#8209;tRNA(Asn) amidotransferase subunit B (EC 6.3.5.6) @ Glutamyl&#8209;tRNA(Gln) amidotransferase subunit B (EC 6.3.5.7)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">179,321</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center"><font class="snp_type_synonymous">R80R</font>&nbsp;(CG<font class="mutation_in_codon">C</font>&rarr;CG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">4&#8209;carboxymuconolactone decarboxylase (EC 4.1.1.44)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">195,396</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.4%</td>
<td align="center"><font class="snp_type_synonymous">I193I</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Dipeptide ABC transporter, substrate&#8209;binding protein DppA (TC 3.A.1.5.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">211,745</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">E116G</font>&nbsp;(G<font class="mutation_in_codon">A</font>G&rarr;G<font class="mutation_in_codon">G</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Fe(2+)/alpha&#8209;ketoglutarate&#8209;dependent dioxygenase LpxO</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">217,302</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_synonymous">R108R</font>&nbsp;(CG<font class="mutation_in_codon">C</font>&rarr;CG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Ferrichrome&#8209;iron receptor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">221,318</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center">intergenic&nbsp;(+52/+4)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Phosphoethanolamine transferase EptC [E.coli], specific for LPS heptose I residue/Lactoylglutathione lyase and related lyases</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">233,302</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">P328S</font>&nbsp;(<font class="mutation_in_codon">C</font>CC&rarr;<font class="mutation_in_codon">T</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type IV fimbrial assembly, ATPase PilB</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">242,628</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">16.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">L34P</font>&nbsp;(C<font class="mutation_in_codon">T</font>T&rarr;C<font class="mutation_in_codon">C</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ATPase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">244,472</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">12.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">Y384C</font>&nbsp;(T<font class="mutation_in_codon">A</font>C&rarr;T<font class="mutation_in_codon">G</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Hemolysin activation/secretion protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_735</td><!-- Seq_Id -->
<td align="right">248,685</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.9%</td>
<td align="center"><font class="snp_type_synonymous">S1230S</font>&nbsp;(TC<font class="mutation_in_codon">G</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Large exoproteins involved in heme utilization or adhesion</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">12,537</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">V253I</font>&nbsp;(<font class="mutation_in_codon">G</font>TC&rarr;<font class="mutation_in_codon">A</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">N&#8209;acyl&#8209;homoserine lactone acylase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">20,909</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">G53D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ABC transporter, substrate&#8209;binding protein (cluster 3, basic aa/glutamine/opines)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">25,780</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">D63N</font>&nbsp;(<font class="mutation_in_codon">G</font>AC&rarr;<font class="mutation_in_codon">A</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Outer membrane protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">37,610</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">L19P</font>&nbsp;(C<font class="mutation_in_codon">T</font>G&rarr;C<font class="mutation_in_codon">C</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">UPF0339 protein YegP.N / UPF0339 protein YegP.C</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">42,245</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_synonymous">G36G</font>&nbsp;(GG<font class="mutation_in_codon">G</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Predicted integral membrane protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">46,796</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">E523D</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">FIG001592: Phosphocarrier protein kinase/phosphorylase, nitrogen regulation associated</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">60,513</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">15.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">P137L</font>&nbsp;(C<font class="mutation_in_codon">C</font>G&rarr;C<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Putative phosphoesterase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">64,538</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center">intergenic&nbsp;(&#8209;24/+304)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Dihydroxy&#8209;acid dehydratase (EC 4.2.1.9)/23S rRNA (cytosine(1962)&#8209;C(5))&#8209;methyltransferase (EC 2.1.1.191)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">66,357</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.0%</td>
<td align="center"><font class="snp_type_synonymous">A98A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Intracellular protease</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">70,144</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_synonymous">Y77Y</font>&nbsp;(TA<font class="mutation_in_codon">C</font>&rarr;TA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">FOG: TPR repeat, SEL1 subfamily</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">74,700</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center"><font class="snp_type_synonymous">G451G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FIG022869: Oxidoreductase, GMC family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">76,242</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">R139C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FIG028932: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">91,893</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.7%</td>
<td align="center"><font class="snp_type_synonymous">R31R</font>&nbsp;(CG<font class="mutation_in_codon">C</font>&rarr;CG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">tRNA (guanine(46)&#8209;N(7))&#8209;methyltransferase (EC 2.1.1.33)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">95,728</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_synonymous">F69F</font>&nbsp;(TT<font class="mutation_in_codon">C</font>&rarr;TT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Oxygen&#8209;independent coproporphyrinogen&#8209;III oxidase&#8209;like protein YggW</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">113,181</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_synonymous">L81L</font>&nbsp;(CT<font class="mutation_in_codon">G</font>&rarr;CT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Pyrimidine operon regulatory protein PyrR</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">131,809</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">R448C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">137,438</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">A50V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Cytochrome b561 homolog 2</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">151,378</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_synonymous">R85R</font>&nbsp;(CG<font class="mutation_in_codon">G</font>&rarr;CG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">ChaX protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">175,973</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">R126H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ABC&#8209;type amino acid transport/signal transduction system, periplasmic component/domain</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">185,184</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.6%</td>
<td align="center"><font class="snp_type_synonymous">L626L</font>&nbsp;(CT<font class="mutation_in_codon">G</font>&rarr;CT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ATP&#8209;dependent Clp protease ATP&#8209;binding subunit PA0459, ClpC family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">200,749</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center">intergenic&nbsp;(+3/&#8209;47)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Uncharacterized protein, possibly involved in aromatic compounds catabolism/Transcriptional regulator, AcrR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">207,810</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">E595G</font>&nbsp;(G<font class="mutation_in_codon">A</font>G&rarr;G<font class="mutation_in_codon">G</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Malate synthase G (EC 2.3.3.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">230,376</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">P338T</font>&nbsp;(<font class="mutation_in_codon">C</font>CG&rarr;<font class="mutation_in_codon">A</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Long chain acyl&#8209;CoA dehydrogenase [fadN&#8209;fadA&#8209;fadE operon] (EC 1.3.8.8)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">236,174</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">A287S</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">T</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Heme d1 biosynthesis protein NirJ</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">240,814</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.9%</td>
<td align="center">intergenic&nbsp;(&#8209;35/+16)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Cytochrome c551 NirM/Nitrite reductase (EC 1.7.2.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">247,953</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">R555H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Nitric oxide reductase activation protein NorD</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">257,019</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">E3K</font>&nbsp;(<font class="mutation_in_codon">G</font>AA&rarr;<font class="mutation_in_codon">A</font>AA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcriptional regulator, Xre family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">260,608</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.5%</td>
<td align="center"><font class="snp_type_synonymous">T156T</font>&nbsp;(AC<font class="mutation_in_codon">C</font>&rarr;AC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Integral membrane protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">269,109</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center"><font class="snp_type_synonymous">T172T</font>&nbsp;(AC<font class="mutation_in_codon">C</font>&rarr;AC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transketolase (EC 2.2.1.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">274,731</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_synonymous">G339G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Phosphoglycerate kinase (EC 2.7.2.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">291,270</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">D350N</font>&nbsp;(<font class="mutation_in_codon">G</font>AC&rarr;<font class="mutation_in_codon">A</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Fic domain protein, PA0574 type</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">321,726</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A28T</font>&nbsp;(<font class="mutation_in_codon">G</font>CA&rarr;<font class="mutation_in_codon">A</font>CA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">LPS&#8209;assembly protein LptD @ Organic solvent tolerance protein precursor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">327,548</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">R652C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Probable two&#8209;component sensor, near polyamine transporter</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">339,843</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center">coding&nbsp;(28/360&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">pyocin R2_PP, conserved hypothetical protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">342,827</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_synonymous">V91V</font>&nbsp;(GT<font class="mutation_in_codon">G</font>&rarr;GT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">pyocin R2_PP, baseplate assembly protein J</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">343,583</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">R50H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">pyocin R2_PP, tail formation protein I</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">347,039</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">A166V</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">pyocin R2_PP, tail sheath protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">365,084</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center"><font class="snp_type_synonymous">S55S</font>&nbsp;(TC<font class="mutation_in_codon">G</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">365,117</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_synonymous">S66S</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">371,136</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">S3N</font>&nbsp;(A<font class="mutation_in_codon">G</font>C&rarr;A<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">S&#8209;adenosylmethionine decarboxylase proenzyme (EC 4.1.1.50), prokaryotic class 1A</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_736</td><!-- Seq_Id -->
<td align="right">380,165</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">W236R</font>&nbsp;(<font class="mutation_in_codon">T</font>GG&rarr;<font class="mutation_in_codon">C</font>GG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Chromosome segregation ATPases</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">2,633</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">14.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">A559T</font>&nbsp;(<font class="mutation_in_codon">G</font>CT&rarr;<font class="mutation_in_codon">A</font>CT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">11,298</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">15.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">G238V</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Sodium/bile acid symporter family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">23,915</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">L17V</font>&nbsp;(<font class="mutation_in_codon">C</font>TC&rarr;<font class="mutation_in_codon">G</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">29,191</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">A8T</font>&nbsp;(<font class="mutation_in_codon">G</font>CA&rarr;<font class="mutation_in_codon">A</font>CA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Potassium&#8209;transporting ATPase C chain (EC 3.6.3.12) (TC 3.A.3.7.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">33,008</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">F13L</font>&nbsp;(<font class="mutation_in_codon">T</font>TC&rarr;<font class="mutation_in_codon">C</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Potassium&#8209;transporting ATPase A chain (EC 3.6.3.12) (TC 3.A.3.7.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">37,188</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q251K</font>&nbsp;(<font class="mutation_in_codon">C</font>AG&rarr;<font class="mutation_in_codon">A</font>AG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">3&#8209;hydroxybutyryl&#8209;CoA dehydrogenase (EC 1.1.1.157); 3&#8209;hydroxyacyl&#8209;CoA dehydrogenase (EC 1.1.1.35)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">60,350</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">A17T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Helicase subunit of the DNA excision repair complex</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">70,870</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center"><font class="snp_type_synonymous">E231E</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Chaperone protein HtpG</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">77,390</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">15.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">G121V</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Succinyl&#8209;CoA ligase [ADP&#8209;forming] alpha chain (EC 6.2.1.5)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">90,297</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">K139R</font>&nbsp;(A<font class="mutation_in_codon">A</font>G&rarr;A<font class="mutation_in_codon">G</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Collagenase and related proteases</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">104,209</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">D401Y</font>&nbsp;(<font class="mutation_in_codon">G</font>AC&rarr;<font class="mutation_in_codon">T</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">O&#8209;acetylhomoserine sulfhydrylase (EC 2.5.1.49)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">105,279</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">R152Q</font>&nbsp;(C<font class="mutation_in_codon">G</font>G&rarr;C<font class="mutation_in_codon">A</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">23S rRNA (cytidine(2498)&#8209;2'&#8209;O)&#8209;methyltransferase (EC 2.1.1.186)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">108,113</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">17.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">A685V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Aconitate hydratase (EC 4.2.1.3) @ 2&#8209;methylisocitrate dehydratase (EC 4.2.1.99)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">120,871</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center"><font class="snp_type_synonymous">V366V</font>&nbsp;(GT<font class="mutation_in_codon">C</font>&rarr;GT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type cbb3 cytochrome oxidase biogenesis protein CcoG, involved in Cu oxidation</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">122,766</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">A344T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type cbb3 cytochrome oxidase biogenesis protein CcoI; Copper&#8209;translocating P&#8209;type ATPase (EC 3.6.3.4)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">127,247</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">14.6%</td>
<td align="center">intergenic&nbsp;(&#8209;185/&#8209;61)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/Fumarate and nitrate reduction regulatory protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">134,980</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">A310V</font>&nbsp;(G<font class="mutation_in_codon">C</font>T&rarr;G<font class="mutation_in_codon">T</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Acyl&#8209;CoA dehydrogenase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">138,815</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">17.4%</td>
<td align="center"><font class="snp_type_synonymous">R119R</font>&nbsp;(CG<font class="mutation_in_codon">C</font>&rarr;CG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">DNA polymerase III subunits gamma and tau (EC 2.7.7.7)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">157,077</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">14.8%</td>
<td align="center"><font class="snp_type_synonymous">A410A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Guanine deaminase (EC 3.5.4.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">162,219</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">20.7%</td>
<td align="center"><font class="snp_type_synonymous">G20G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Allantoicase (EC 3.5.3.4)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_741</td><!-- Seq_Id -->
<td align="right">163,467</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.9%</td>
<td align="center"><font class="snp_type_synonymous">P85P</font>&nbsp;(CC<font class="mutation_in_codon">G</font>&rarr;CC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Ureidoglycolate lyase (EC 4.3.2.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">43,769</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">E64G</font>&nbsp;(G<font class="mutation_in_codon">A</font>G&rarr;G<font class="mutation_in_codon">G</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Protein IscX, believed to be involved in assembly of Fe&#8209;S clusters</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">50,155</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">A92V</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">UPF0070 protein YfgM</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">68,333</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">V55A</font>&nbsp;(G<font class="mutation_in_codon">T</font>G&rarr;G<font class="mutation_in_codon">C</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Amidases related to nicotinamidase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">78,706</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_synonymous">L378L</font>&nbsp;(CT<font class="mutation_in_codon">G</font>&rarr;CT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Uncharacterized MFS&#8209;type transporter</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">88,917</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_synonymous">H145H</font>&nbsp;(CA<font class="mutation_in_codon">C</font>&rarr;CA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Replication protein RepA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">99,960</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_synonymous">G388G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Conjugative transfer protein TrbL</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">119,646</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">I130V</font>&nbsp;(<font class="mutation_in_codon">A</font>TC&rarr;<font class="mutation_in_codon">G</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Uncharacterized amino acid permease, GabP family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">122,820</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center">intergenic&nbsp;(&#8209;226/&#8209;175)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Membrane&#8209;bound lytic murein transglycosylase F (EC 4.2.2.n1)/Phosphoribosylformylglycinamidine synthase, synthetase subunit (EC 6.3.5.3) / Phosphoribosylformylglycinamidine synthase, glutamine amidotransferase subunit (EC 6.3.5.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">130,782</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_synonymous">G453G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">PTS system, N&#8209;acetylglucosamine&#8209;specific IIA component (EC 2.7.1.193) / Phosphotransferase system, phosphocarrier protein HPr / Phosphoenolpyruvate&#8209;protein phosphotransferase of PTS system (EC 2.7.3.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">137,886</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_synonymous">S63S</font>&nbsp;(TC<font class="mutation_in_codon">C</font>&rarr;TC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">161,462</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">D124N</font>&nbsp;(<font class="mutation_in_codon">G</font>AC&rarr;<font class="mutation_in_codon">A</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Inner membrane protein YqiK</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">174,222</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">C202F</font>&nbsp;(T<font class="mutation_in_codon">G</font>C&rarr;T<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">probable FMN oxidoreductase</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">175,519</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">100%</td>
<td align="center"><font class="snp_type_nonsynonymous">P133L</font>&nbsp;(C<font class="mutation_in_codon">C</font>G&rarr;C<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transcriptional regulator, AcrR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">178,582</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center">intergenic&nbsp;(&#8209;52/&#8209;159)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Peptidylprolyl isomerase, FKBP&#8209;type (EC 5.2.1.8)/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">180,309</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_synonymous">L523L</font>&nbsp;(TT<font class="mutation_in_codon">G</font>&rarr;TT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">181,016</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">G105D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">182,012</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">12.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">D27G</font>&nbsp;(G<font class="mutation_in_codon">A</font>C&rarr;G<font class="mutation_in_codon">G</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Two&#8209;component transcriptional response regulator, LuxR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">191,356</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_synonymous">D427D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">probable chemotaxis transducer</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">196,534</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A21T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Chemotaxis response regulator protein&#8209;glutamate methylesterase CheB (EC 3.1.1.61)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">198,276</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">R230C</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Two&#8209;component system response regulator protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">205,183</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_synonymous">D234D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">DUF124 domain&#8209;containing protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">205,828</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">L202M</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Hydrolase of the alpha/beta superfamily in cluster with COG2110</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">213,793</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_synonymous">G579G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Phosphoenolpyruvate carboxylase (EC 4.1.1.31)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">218,406</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">A248E</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">A</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">PEP phosphonomutase and related enzymes</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">219,096</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">A202T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">RNA binding S1 domain protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">221,226</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">A390G</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">G</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Cytochrome P450</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">267,064</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_synonymous">L1102L</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">T</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">DNA polymerase III alpha subunit (EC 2.7.7.7)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">297,565</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">18.2%</td>
<td align="center"><font class="snp_type_synonymous">A317A</font>&nbsp;(GC<font class="mutation_in_codon">C</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Spermidine/putrescine import ABC transporter substrate&#8209;binding protein PotD (TC 3.A.1.11.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">305,806</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">G123V</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Ferredoxin&#8209;dependent glutamate synthase (EC 1.4.7.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">316,630</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_synonymous">V322V</font>&nbsp;(GT<font class="mutation_in_codon">G</font>&rarr;GT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">L&#8209;carnitine dehydratase/bile acid&#8209;inducible protein F</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">322,425</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">17.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">A91T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transcriptional activator MetR</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">329,471</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">V55M</font>&nbsp;(<font class="mutation_in_codon">G</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glycerol uptake facilitator protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">334,920</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.4%</td>
<td align="center"><font class="snp_type_synonymous">V44V</font>&nbsp;(GT<font class="mutation_in_codon">C</font>&rarr;GT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Cytochrome b561</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">349,967</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">G607D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">PTS system, fructose&#8209;specific IIA component (EC 2.7.1.202) / Phosphotransferase system, phosphocarrier protein HPr / Phosphoenolpyruvate&#8209;protein phosphotransferase of PTS system (EC 2.7.3.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">357,484</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">P62L</font>&nbsp;(C<font class="mutation_in_codon">C</font>G&rarr;C<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Undecaprenyl phosphate&#8209;alpha&#8209;4&#8209;amino&#8209;4&#8209;deoxy&#8209;L&#8209;arabinose arabinosyl transferase (EC 2.4.2.43) @ Polymyxin resistance protein PmrK</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">364,988</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q131K</font>&nbsp;(<font class="mutation_in_codon">C</font>AG&rarr;<font class="mutation_in_codon">A</font>AG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Alginate O&#8209;acetyltransferase AlgF, periplasmic</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">376,394</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_synonymous">A109A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Alginate polymerisation protein Alg44, membrane fusion protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_745</td><!-- Seq_Id -->
<td align="right">378,064</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">R74H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Alginate polymerase/glycosyltransferase Alg8</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">9,992</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">L80M</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Outer membrane low permeability porin, OprD family => OccK9/OpdG</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">10,011</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center"><font class="snp_type_synonymous">S73S</font>&nbsp;(TC<font class="mutation_in_codon">C</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Outer membrane low permeability porin, OprD family => OccK9/OpdG</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">19,049</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">G45D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ABC transporter, substrate&#8209;binding protein (cluster 3, basic aa/glutamine/opines)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">26,680</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A68V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transcriptional regulator, AcrR family</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">32,193</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">16.2%</td>
<td align="center">intergenic&nbsp;(&#8209;38/+119)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Protease subunit of ATP&#8209;dependent Clp proteases/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">34,706</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">15.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">G182D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">38,994</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">14.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">A137V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Two&#8209;component system sensor histidine kinase/response regulator hybrid</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">41,365</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">16.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">H22Y</font>&nbsp;(<font class="mutation_in_codon">C</font>AC&rarr;<font class="mutation_in_codon">T</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">FIG00956084: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">51,286</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_synonymous">G293G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">4&#8209;alpha&#8209;glucanotransferase (amylomaltase) (EC 2.4.1.25)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">98,070</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_synonymous">L359L</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">T</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Aldehyde dehydrogenase (EC 1.2.1.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">99,808</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_synonymous">L332L</font>&nbsp;(CT<font class="mutation_in_codon">C</font>&rarr;CT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Oxidoreductase, GMC family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">107,049</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_synonymous">T104T</font>&nbsp;(AC<font class="mutation_in_codon">G</font>&rarr;AC<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Methylphosphotriester&#8209;DNA&#8209;&#8209;protein&#8209;cysteine S&#8209;methyltransferase (EC 2.1.1.n11) / ADA regulatory protein / Methylated&#8209;DNA&#8209;&#8209;protein&#8209;cysteine methyltransferase (EC 2.1.1.63)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">115,373</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">21.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">A245V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Allophanate hydrolase 2 subunit 2 (EC 3.5.1.54)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">124,365</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q271K</font>&nbsp;(<font class="mutation_in_codon">C</font>AG&rarr;<font class="mutation_in_codon">A</font>AG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Putative transmembrane sensor</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">136,935</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.1%</td>
<td align="center">intergenic&nbsp;(&#8209;86/+2)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Asparagine synthetase [glutamine&#8209;hydrolyzing] (EC 6.3.5.4)/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">137,224</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A330T</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">A</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">151,599</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">T493I</font>&nbsp;(A<font class="mutation_in_codon">C</font>C&rarr;A<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Sensory box/GGDEF family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">162,858</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">R7H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Multicopper oxidase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">170,257</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">P286S</font>&nbsp;(<font class="mutation_in_codon">C</font>CG&rarr;<font class="mutation_in_codon">T</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ABC transporter, permease protein 2 (cluster 5, nickel/peptides/opines)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">176,305</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.1%</td>
<td align="center"><font class="snp_type_synonymous">N218N</font>&nbsp;(AA<font class="mutation_in_codon">C</font>&rarr;AA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">TonB&#8209;dependent receptor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">184,278</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">23.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">L210V</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">G</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">184,513</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">17.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">P288Q</font>&nbsp;(C<font class="mutation_in_codon">C</font>G&rarr;C<font class="mutation_in_codon">A</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_746</td><!-- Seq_Id -->
<td align="right">191,443</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">D18N</font>&nbsp;(<font class="mutation_in_codon">G</font>AC&rarr;<font class="mutation_in_codon">A</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">FIG00957384: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_753</td><!-- Seq_Id -->
<td align="right">12,688</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">R279Q</font>&nbsp;(C<font class="mutation_in_codon">G</font>G&rarr;C<font class="mutation_in_codon">A</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">TonB&#8209;dependent receptor</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_753</td><!-- Seq_Id -->
<td align="right">37,973</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_nonsense">S76*</font>&nbsp;(T<font class="mutation_in_codon">C</font>G&rarr;T<font class="mutation_in_codon">A</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_753</td><!-- Seq_Id -->
<td align="right">39,575</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center">intergenic&nbsp;(+67/&#8209;56)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i></td>
<td align="left">hypothetical protein/FIG00954694: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_753</td><!-- Seq_Id -->
<td align="right">40,323</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">D139N</font>&nbsp;(<font class="mutation_in_codon">G</font>AT&rarr;<font class="mutation_in_codon">A</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">LysR family transcriptional regulator PA0739</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_753</td><!-- Seq_Id -->
<td align="right">40,338</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">R134S</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">A</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">LysR family transcriptional regulator PA0739</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_753</td><!-- Seq_Id -->
<td align="right">69,989</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">R145W</font>&nbsp;(<font class="mutation_in_codon">C</font>GG&rarr;<font class="mutation_in_codon">T</font>GG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Translation elongation factor LepA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_753</td><!-- Seq_Id -->
<td align="right">79,174</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">A15T</font>&nbsp;(<font class="mutation_in_codon">G</font>CT&rarr;<font class="mutation_in_codon">A</font>CT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ABC&#8209;type amino acid transport/signal transduction system, periplasmic component/domain</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_753</td><!-- Seq_Id -->
<td align="right">89,295</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center">intergenic&nbsp;(+292/&#8209;20)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Proline dehydrogenase (EC 1.5.5.2) / Delta&#8209;1&#8209;pyrroline&#8209;5&#8209;carboxylate dehydrogenase (EC 1.2.1.88)/Proline/sodium symporter PutP (TC 2.A.21.2.1) @ Propionate/sodium symporter</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_753</td><!-- Seq_Id -->
<td align="right">108,295</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">F140S</font>&nbsp;(T<font class="mutation_in_codon">T</font>C&rarr;T<font class="mutation_in_codon">C</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">2&#8209;methylcitrate synthase (EC 2.3.3.5)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_753</td><!-- Seq_Id -->
<td align="right">116,635</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_synonymous">P26P</font>&nbsp;(CC<font class="mutation_in_codon">C</font>&rarr;CC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Oxidoreductase, aldo/keto reductase family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">9,587</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">D349G</font>&nbsp;(G<font class="mutation_in_codon">A</font>C&rarr;G<font class="mutation_in_codon">G</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Multimodular transpeptidase&#8209;transglycosylase (EC 2.4.1.129) (EC 3.4.&#8209;.&#8209;)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">17,172</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">A63T</font>&nbsp;(<font class="mutation_in_codon">G</font>CG&rarr;<font class="mutation_in_codon">A</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Heme ABC transporter, permease protein HmuU</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">21,766</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">E737D</font>&nbsp;(GA<font class="mutation_in_codon">G</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">TonB&#8209;dependent hemin, ferrichrome receptor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">27,796</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">17.6%</td>
<td align="center">coding&nbsp;(27/477&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Preprotein translocase subunit SecD</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">29,075</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.8%</td>
<td align="center"><font class="snp_type_synonymous">R280R</font>&nbsp;(CG<font class="mutation_in_codon">C</font>&rarr;CG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Xanthine/uracil/thiamine/ascorbate permease family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">37,055</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_synonymous">V902V</font>&nbsp;(GT<font class="mutation_in_codon">G</font>&rarr;GT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Two&#8209;component sensor CbrA: intrcellular carbon:nitrogen balance</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">43,730</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">E63K</font>&nbsp;(<font class="mutation_in_codon">G</font>AG&rarr;<font class="mutation_in_codon">A</font>AG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Aspartate 1&#8209;decarboxylase (EC 4.1.1.11)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">45,441</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_synonymous">L476L</font>&nbsp;(CT<font class="mutation_in_codon">G</font>&rarr;CT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Glucose&#8209;6&#8209;phosphate isomerase (EC 5.3.1.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">51,289</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">R793H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">57,122</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">L50M</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">tRNA pseudouridine(55) synthase (EC 5.4.99.25)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">66,200</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center"><font class="snp_type_synonymous">T613T</font>&nbsp;(AC<font class="mutation_in_codon">C</font>&rarr;AC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Cell division&#8209;associated, ATP&#8209;dependent zinc metalloprotease FtsH</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">71,547</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">L626F</font>&nbsp;(<font class="mutation_in_codon">C</font>TT&rarr;<font class="mutation_in_codon">T</font>TT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Carbamoyl&#8209;phosphate synthase large chain (EC 6.3.5.5)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">76,552</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_synonymous">L290L</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">T</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Chaperone protein DnaJ</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">94,011</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A161V</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">96,240</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_synonymous">Q383Q</font>&nbsp;(CA<font class="mutation_in_codon">G</font>&rarr;CA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Sensory histidine kinase QseC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">100,616</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.7%</td>
<td align="center"><font class="snp_type_synonymous">L29L</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">T</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Response regulator c&#8209;di&#8209;GMP phosphodiesterase, RpfG family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">121,750</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">P583S</font>&nbsp;(<font class="mutation_in_codon">C</font>CC&rarr;<font class="mutation_in_codon">T</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Selenocysteine&#8209;specific translation elongation factor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">122,938</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">V187I</font>&nbsp;(<font class="mutation_in_codon">G</font>TC&rarr;<font class="mutation_in_codon">A</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Selenocysteine&#8209;specific translation elongation factor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">132,770</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">R140G</font>&nbsp;(<font class="mutation_in_codon">C</font>GC&rarr;<font class="mutation_in_codon">G</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">2,4&#8209;dienoyl&#8209;CoA reductase [NADPH] (EC 1.3.1.34)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">133,139</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">H263Y</font>&nbsp;(<font class="mutation_in_codon">C</font>AC&rarr;<font class="mutation_in_codon">T</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">2,4&#8209;dienoyl&#8209;CoA reductase [NADPH] (EC 1.3.1.34)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">134,650</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">A15V</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Integral membrane protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">137,771</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">A196V</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">dolichyl&#8209;phosphate&#8209;mannose&#8209;protein mannosyltransferase family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">139,686</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">12.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">M1K</font>&nbsp;(A<font class="mutation_in_codon">T</font>G&rarr;A<font class="mutation_in_codon">A</font>G)&nbsp;&dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">GtrA family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">142,410</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">T292I</font>&nbsp;(A<font class="mutation_in_codon">C</font>C&rarr;A<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Sodium&#8209;dependent phosphate transporter</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">150,556</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_synonymous">G72G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Dihydrolipoamide dehydrogenase (EC 1.8.1.4)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">156,586</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">C50W</font>&nbsp;(TG<font class="mutation_in_codon">C</font>&rarr;TG<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FIG016157: Similar to nicotianamine synthase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">177,162</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">19.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">D148Y</font>&nbsp;(<font class="mutation_in_codon">G</font>AC&rarr;<font class="mutation_in_codon">T</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">tRNA&#8209;dihydrouridine synthase DusB</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">183,826</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_synonymous">S868S</font>&nbsp;(TC<font class="mutation_in_codon">C</font>&rarr;TC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">probable sensor/response regulator hybrid</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">187,751</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.4%</td>
<td align="center"><font class="snp_type_synonymous">L272L</font>&nbsp;(CT<font class="mutation_in_codon">C</font>&rarr;CT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Urea ABC transporter, permease protein UrtB</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">195,233</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.9%</td>
<td align="center"><font class="snp_type_synonymous">L447L</font>&nbsp;(CT<font class="mutation_in_codon">C</font>&rarr;CT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Urease alpha subunit (EC 3.5.1.5)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">213,461</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">L41P</font>&nbsp;(C<font class="mutation_in_codon">T</font>G&rarr;C<font class="mutation_in_codon">C</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Uncharacterized MFS&#8209;type transporter</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">215,256</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_synonymous">E252E</font>&nbsp;(GA<font class="mutation_in_codon">G</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Flavodoxin reductases (ferredoxin&#8209;NADPH reductases) family 1</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">216,908</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center">intergenic&nbsp;(+109/&#8209;121)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Unsaturated fatty acid biosythesis repressor FabR, TetR family/Urease accessory protein UreE</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">224,754</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.6%</td>
<td align="center"><font class="snp_type_synonymous">F258F</font>&nbsp;(TT<font class="mutation_in_codon">C</font>&rarr;TT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Outer membrane low permeability porin, OprD family => OccK1/OpdK benzoate/histidine, carbenicillin and cefoxitin uptake</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">228,980</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">R386H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Benzoylformate decarboxylase (EC 4.1.1.7)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">230,099</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">R13L</font>&nbsp;(C<font class="mutation_in_codon">G</font>T&rarr;C<font class="mutation_in_codon">T</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Benzoylformate decarboxylase (EC 4.1.1.7)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">239,928</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">P225S</font>&nbsp;(<font class="mutation_in_codon">C</font>CG&rarr;<font class="mutation_in_codon">T</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">gamma&#8209;aminobutyric acid ABC transporter, permease protein 2</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">290,657</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">V286I</font>&nbsp;(<font class="mutation_in_codon">G</font>TC&rarr;<font class="mutation_in_codon">A</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">296,601</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_synonymous">R440R</font>&nbsp;(CG<font class="mutation_in_codon">C</font>&rarr;CG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Sensory box protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">302,974</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_synonymous">R434R</font>&nbsp;(CG<font class="mutation_in_codon">C</font>&rarr;CG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">DNA topoisomerase IV subunit A (EC 5.99.1.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">315,653</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.8%</td>
<td align="center"><font class="snp_type_synonymous">E154E</font>&nbsp;(GA<font class="mutation_in_codon">G</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">NAD(P)H oxidoreductase YrkL @ Putative NADPH&#8209;quinone reductase (modulator of drug activity B) @ Flavodoxin 2</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">316,665</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">R228C</font>&nbsp;(<font class="mutation_in_codon">C</font>GT&rarr;<font class="mutation_in_codon">T</font>GT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Aspartate aminotransferase (EC 2.6.1.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">398,035</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">15.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">M1302I</font>&nbsp;(AT<font class="mutation_in_codon">G</font>&rarr;AT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glutamate synthase [NADPH] large chain (EC 1.4.1.13)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">422,770</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_synonymous">L138L</font>&nbsp;(CT<font class="mutation_in_codon">G</font>&rarr;CT<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ATP&#8209;dependent protease subunit HslV (EC 3.4.25.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">433,965</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.5%</td>
<td align="center"><font class="snp_type_synonymous">R36R</font>&nbsp;(CG<font class="mutation_in_codon">C</font>&rarr;CG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Ubiquinone biosynthesis regulatory protein kinase UbiB</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">446,222</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_synonymous">L139L</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">T</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glucans biosynthesis glucosyltransferase H</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">448,128</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A27T</font>&nbsp;(<font class="mutation_in_codon">G</font>CA&rarr;<font class="mutation_in_codon">A</font>CA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glucans biosynthesis protein G precursor</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">450,337</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">G30S</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">A</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Nudix hydrolase family protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">451,404</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">S76T</font>&nbsp;(A<font class="mutation_in_codon">G</font>C&rarr;A<font class="mutation_in_codon">C</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glutamate/aspartate ABC transporter, substrate&#8209;binding protein GltI (TC 3.A.1.3.4)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">452,533</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">A256S</font>&nbsp;(<font class="mutation_in_codon">G</font>CC&rarr;<font class="mutation_in_codon">T</font>CC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;amino acid dehydrogenase (EC 1.4.99.6)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">473,418</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center"><font class="snp_type_synonymous">G68G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Putative transporter</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">498,391</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">E420K</font>&nbsp;(<font class="mutation_in_codon">G</font>AA&rarr;<font class="mutation_in_codon">A</font>AA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Glutamine synthetase type I (EC 6.3.1.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">508,106</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_synonymous">L88L</font>&nbsp;(CT<font class="mutation_in_codon">G</font>&rarr;CT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Rhodanese&#8209;related sulfurtransferase YibN</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">518,593</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">15.0%</td>
<td align="center"><font class="snp_type_synonymous">A138A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ABC&#8209;type amino acid transport/signal transduction system, periplasmic component/domain</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">527,294</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_synonymous">L259L</font>&nbsp;(CT<font class="mutation_in_codon">C</font>&rarr;CT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">A/G&#8209;specific adenine glycosylase (EC 3.2.2.&#8209;)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">539,409</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">W160R</font>&nbsp;(<font class="mutation_in_codon">T</font>GG&rarr;<font class="mutation_in_codon">C</font>GG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Multidrug efflux system EmrAB&#8209;OMF, inner&#8209;membrane proton/drug antiporter EmrB (MFS type)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">580,573</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_synonymous">K276K</font>&nbsp;(AA<font class="mutation_in_codon">G</font>&rarr;AA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Ribosomal protein S6&#8209;&#8209;L&#8209;glutamate ligase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">588,983</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center"><font class="snp_type_synonymous">N359N</font>&nbsp;(AA<font class="mutation_in_codon">C</font>&rarr;AA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">N&#8209;acetylglutamate synthase (EC 2.3.1.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">609,746</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_synonymous">R277R</font>&nbsp;(CG<font class="mutation_in_codon">G</font>&rarr;CG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">2&#8209;polyprenylphenol hydroxylase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">635,838</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">P132L</font>&nbsp;(C<font class="mutation_in_codon">C</font>G&rarr;C<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Porphobilinogen synthase (EC 4.2.1.24)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">639,265</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">16.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">P566S</font>&nbsp;(<font class="mutation_in_codon">C</font>CG&rarr;<font class="mutation_in_codon">T</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">High&#8209;affinity Fe2+/Pb2+ permease</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">640,415</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_synonymous">P182P</font>&nbsp;(CC<font class="mutation_in_codon">G</font>&rarr;CC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">High&#8209;affinity Fe2+/Pb2+ permease</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">648,381</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_synonymous">S132S</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Periplasmic thiol:disulfide oxidoreductase DsbB, required for DsbA reoxidation</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">653,561</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.1%</td>
<td align="center"><font class="snp_type_synonymous">A112A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Alginate biosynthesis two&#8209;component system response regulator AlgR</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">654,767</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_synonymous">L71L</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">T</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Alginate biosynthesis two&#8209;component system sensor histidine kinase AlgZ/FimS</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_754</td><!-- Seq_Id -->
<td align="right">661,794</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_synonymous">R154R</font>&nbsp;(CG<font class="mutation_in_codon">C</font>&rarr;CG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">membrane protein, putative</td>
</tr>
<!-- End Table Row -->
</table></body></html>