<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>Mutation Comparison</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<!--Output Html_Mutation_Table_String-->
<table border="0" cellspacing="1" cellpadding="3">
<tr><th colspan="7" align="left" class="mutation_header_row">Predicted mutations</th></tr><tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>

<!-- Item Lines -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">3</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">17.9%</td>
<td align="center">intergenic&nbsp;(&#8211;/&#8209;26)</td>
<td align="center"><i>&#8211;</i>&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">&#8211;/Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">6</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">14.5%</td>
<td align="center">intergenic&nbsp;(&#8211;/&#8209;23)</td>
<td align="center"><i>&#8211;</i>&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">&#8211;/Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">12</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">82.2%</td>
<td align="center">intergenic&nbsp;(&#8211;/&#8209;17)</td>
<td align="center"><i>&#8211;</i>&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">&#8211;/Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">45</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">13.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">I6T</font>&nbsp;(A<font class="mutation_in_codon">T</font>A&rarr;A<font class="mutation_in_codon">C</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">168</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">12.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q47R</font>&nbsp;(C<font class="mutation_in_codon">A</font>G&rarr;C<font class="mutation_in_codon">G</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">195</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">100%</td>
<td align="center"><font class="snp_type_nonsynonymous">T56I</font>&nbsp;(A<font class="mutation_in_codon">C</font>T&rarr;A<font class="mutation_in_codon">T</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">207</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">41.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">T60I</font>&nbsp;(A<font class="mutation_in_codon">C</font>A&rarr;A<font class="mutation_in_codon">T</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">225</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">100%</td>
<td align="center"><font class="snp_type_nonsynonymous">I66T</font>&nbsp;(A<font class="mutation_in_codon">T</font>A&rarr;A<font class="mutation_in_codon">C</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">237</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">T70I</font>&nbsp;(A<font class="mutation_in_codon">C</font>T&rarr;A<font class="mutation_in_codon">T</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">243</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">23.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">I72T</font>&nbsp;(A<font class="mutation_in_codon">T</font>A&rarr;A<font class="mutation_in_codon">C</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">246</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">A73V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">249</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">30.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">T74I</font>&nbsp;(A<font class="mutation_in_codon">C</font>T&rarr;A<font class="mutation_in_codon">T</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">258</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q77R</font>&nbsp;(C<font class="mutation_in_codon">A</font>G&rarr;C<font class="mutation_in_codon">G</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">264</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">14.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">A79V</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">279</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">42.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">I84T</font>&nbsp;(A<font class="mutation_in_codon">T</font>A&rarr;A<font class="mutation_in_codon">C</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">291</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">17.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">T88I</font>&nbsp;(A<font class="mutation_in_codon">C</font>T&rarr;A<font class="mutation_in_codon">T</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">294</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q89R</font>&nbsp;(C<font class="mutation_in_codon">A</font>G&rarr;C<font class="mutation_in_codon">G</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_153</td><!-- Seq_Id -->
<td align="right">324</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">15.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q99R</font>&nbsp;(C<font class="mutation_in_codon">A</font>G&rarr;C<font class="mutation_in_codon">G</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type I restriction&#8209;modification system, DNA&#8209;methyltransferase subunit M (EC 2.1.1.72) / Type I restriction&#8209;modification system, specificity subunit S</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_173</td><!-- Seq_Id -->
<td align="right">322</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">16.0%</td>
<td align="center">noncoding&nbsp;(314/2925&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_creation_tool=RNA_reps_LSU_rRNA</i>&nbsp;&rarr;</td>
<td align="left">LSU rRNA ## 23S rRNA, large subunit ribosomal RNA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_173</td><!-- Seq_Id -->
<td align="right">1,528</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">18.3%</td>
<td align="center">noncoding&nbsp;(1520/2925&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_creation_tool=RNA_reps_LSU_rRNA</i>&nbsp;&rarr;</td>
<td align="left">LSU rRNA ## 23S rRNA, large subunit ribosomal RNA</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_173</td><!-- Seq_Id -->
<td align="right">2,243</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">31.6%</td>
<td align="center">noncoding&nbsp;(2235/2925&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_creation_tool=RNA_reps_LSU_rRNA</i>&nbsp;&rarr;</td>
<td align="left">LSU rRNA ## 23S rRNA, large subunit ribosomal RNA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_268</td><!-- Seq_Id -->
<td align="right">700</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">14.2%</td>
<td align="center"><font class="snp_type_synonymous">Q9Q</font>&nbsp;(CA<font class="mutation_in_codon">G</font>&rarr;CA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Hypothetical protein, PVL orf52 homolog [SA bacteriophages 11, Mu50B]</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_268</td><!-- Seq_Id -->
<td align="right">706</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.9%</td>
<td align="center"><font class="snp_type_synonymous">I7I</font>&nbsp;(AT<font class="mutation_in_codon">A</font>&rarr;AT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Hypothetical protein, PVL orf52 homolog [SA bacteriophages 11, Mu50B]</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_268</td><!-- Seq_Id -->
<td align="right">718</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.5%</td>
<td align="center"><font class="snp_type_synonymous">N3N</font>&nbsp;(AA<font class="mutation_in_codon">C</font>&rarr;AA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Hypothetical protein, PVL orf52 homolog [SA bacteriophages 11, Mu50B]</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_271</td><!-- Seq_Id -->
<td align="right">1,146</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">25.2%</td>
<td align="center"><font class="snp_type_synonymous">L13L</font>&nbsp;(CT<font class="mutation_in_codon">A</font>&rarr;CT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ORF101</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_271</td><!-- Seq_Id -->
<td align="right">1,164</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">33.5%</td>
<td align="center"><font class="snp_type_synonymous">A19A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ORF101</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_276</td><!-- Seq_Id -->
<td align="right">2,164</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">48.1%</td>
<td align="center">intergenic&nbsp;(+133/&#8211;)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;<i>&#8211;</i></td>
<td align="left">hypothetical protein/&#8211;</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_279</td><!-- Seq_Id -->
<td align="right">12</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">43.6%</td>
<td align="center">intergenic&nbsp;(&#8211;/&#8209;211)</td>
<td align="center"><i>&#8211;</i>&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">&#8211;/Cadmium resistance protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_279</td><!-- Seq_Id -->
<td align="right">8,198</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">26.9%</td>
<td align="center">intergenic&nbsp;(+22/&#8211;)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;<i>&#8211;</i></td>
<td align="left">Tn552 transposase/&#8211;</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_279</td><!-- Seq_Id -->
<td align="right">8,204</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">23.1%</td>
<td align="center">intergenic&nbsp;(+28/&#8211;)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;<i>&#8211;</i></td>
<td align="left">Tn552 transposase/&#8211;</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_279</td><!-- Seq_Id -->
<td align="right">8,209</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">20.8%</td>
<td align="center">intergenic&nbsp;(+33/&#8211;)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;<i>&#8211;</i></td>
<td align="left">Tn552 transposase/&#8211;</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_279</td><!-- Seq_Id -->
<td align="right">8,210</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">20.8%</td>
<td align="center">intergenic&nbsp;(+34/&#8211;)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;<i>&#8211;</i></td>
<td align="left">Tn552 transposase/&#8211;</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_279</td><!-- Seq_Id -->
<td align="right">8,215</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">20.5%</td>
<td align="center">intergenic&nbsp;(+39/&#8211;)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;<i>&#8211;</i></td>
<td align="left">Tn552 transposase/&#8211;</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_279</td><!-- Seq_Id -->
<td align="right">8,219</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">21.3%</td>
<td align="center">intergenic&nbsp;(+43/&#8211;)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;<i>&#8211;</i></td>
<td align="left">Tn552 transposase/&#8211;</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_279</td><!-- Seq_Id -->
<td align="right">8,221:1</td><!-- Position -->
<td align="center">+A</td><!-- Cell Mutation -->
<td align="right">20.6%</td>
<td align="center">intergenic&nbsp;(+45/&#8211;)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;<i>&#8211;</i></td>
<td align="left">Tn552 transposase/&#8211;</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_279</td><!-- Seq_Id -->
<td align="right">8,222</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">22.5%</td>
<td align="center">intergenic&nbsp;(+46/&#8211;)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;<i>&#8211;</i></td>
<td align="left">Tn552 transposase/&#8211;</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_279</td><!-- Seq_Id -->
<td align="right">8,240</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">25.9%</td>
<td align="center">intergenic&nbsp;(+64/&#8211;)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;<i>&#8211;</i></td>
<td align="left">Tn552 transposase/&#8211;</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_289</td><!-- Seq_Id -->
<td align="right">6,408</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">12.8%</td>
<td align="center">intergenic&nbsp;(+120/+53)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Transcriptional regulator, HxlR family/Mobile element protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_289</td><!-- Seq_Id -->
<td align="right">6,411</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">12.3%</td>
<td align="center">intergenic&nbsp;(+123/+50)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Transcriptional regulator, HxlR family/Mobile element protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_289</td><!-- Seq_Id -->
<td align="right">6,426</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">18.6%</td>
<td align="center">intergenic&nbsp;(+138/+35)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Transcriptional regulator, HxlR family/Mobile element protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_289</td><!-- Seq_Id -->
<td align="right">6,430</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">19.0%</td>
<td align="center">intergenic&nbsp;(+142/+31)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Transcriptional regulator, HxlR family/Mobile element protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_289</td><!-- Seq_Id -->
<td align="right">6,432</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">18.1%</td>
<td align="center">intergenic&nbsp;(+144/+29)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Transcriptional regulator, HxlR family/Mobile element protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center">k141_293</td><!-- Seq_Id -->
<td align="right">18,324</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">100%</td>
<td align="center">intergenic&nbsp;(&#8209;361/&#8211;)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;<i>&#8211;</i></td>
<td align="left">Toprim domain protein/&#8211;</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_296</td><!-- Seq_Id -->
<td align="right">478</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">9.2%</td>
<td align="center"><font class="snp_type_synonymous">N159N</font>&nbsp;(AA<font class="mutation_in_codon">T</font>&rarr;AA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Phage tail fiber</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_296</td><!-- Seq_Id -->
<td align="right">482</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">L161M</font>&nbsp;(<font class="mutation_in_codon">T</font>TG&rarr;<font class="mutation_in_codon">A</font>TG)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Phage tail fiber</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_296</td><!-- Seq_Id -->
<td align="right">484</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center"><font class="snp_type_synonymous">L161L</font>&nbsp;(TT<font class="mutation_in_codon">G</font>&rarr;TT<font class="mutation_in_codon">A</font>)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Phage tail fiber</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_296</td><!-- Seq_Id -->
<td align="right">976</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">8.8%</td>
<td align="center">intergenic&nbsp;(+49/&#8209;7)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Phage protein/Phage holin</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_297</td><!-- Seq_Id -->
<td align="right">10,161</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">L299S</font>&nbsp;(T<font class="mutation_in_codon">T</font>G&rarr;T<font class="mutation_in_codon">C</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phage major capsid protein of Caudovirales</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_298</td><!-- Seq_Id -->
<td align="right">18,512</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center">intergenic&nbsp;(&#8209;68/&#8209;302)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Ribonuclease HIII (EC 3.1.26.4)/Z&#8209;ring&#8209;associated protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_298</td><!-- Seq_Id -->
<td align="right">31,405</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_synonymous">G150G</font>&nbsp;(GG<font class="mutation_in_codon">A</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Nucleoside 5&#8209;triphosphatase RdgB (dHAPTP, dITP, XTP&#8209;specific) (EC 3.6.1.66)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_300</td><!-- Seq_Id -->
<td align="right">13,050</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_synonymous">R3R</font>&nbsp;(CG<font class="mutation_in_codon">T</font>&rarr;CG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Spermidine/putrescine import ABC transporter substrate&#8209;binding protein PotD (TC 3.A.1.11.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_300</td><!-- Seq_Id -->
<td align="right">15,886</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">K132N</font>&nbsp;(AA<font class="mutation_in_codon">G</font>&rarr;AA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_304</td><!-- Seq_Id -->
<td align="right">38,323</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">53.8%</td>
<td align="center">intergenic&nbsp;(+11/&#8209;6)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">tRNA&#8209;Val&#8209;TAC/tRNA&#8209;Thr&#8209;TGT</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_304</td><!-- Seq_Id -->
<td align="right">38,328</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">51.9%</td>
<td align="center">intergenic&nbsp;(+16/&#8209;1)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">tRNA&#8209;Val&#8209;TAC/tRNA&#8209;Thr&#8209;TGT</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_304</td><!-- Seq_Id -->
<td align="right">38,406</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">37.9%</td>
<td align="center">intergenic&nbsp;(+2/&#8209;5)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">tRNA&#8209;Thr&#8209;TGT/tRNA&#8209;Lys&#8209;TTT</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_304</td><!-- Seq_Id -->
<td align="right">38,408</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">37.1%</td>
<td align="center">intergenic&nbsp;(+4/&#8209;3)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">tRNA&#8209;Thr&#8209;TGT/tRNA&#8209;Lys&#8209;TTT</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_304</td><!-- Seq_Id -->
<td align="right">38,591</td><!-- Position -->
<td align="center">+TTT</td><!-- Cell Mutation -->
<td align="right">33.7%</td>
<td align="center">noncoding&nbsp;(75/75&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&rarr;</td>
<td align="left">tRNA&#8209;Gly&#8209;GCC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_304</td><!-- Seq_Id -->
<td align="right">38,596</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">29.0%</td>
<td align="center">intergenic&nbsp;(+5/&#8209;3)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">tRNA&#8209;Gly&#8209;GCC/tRNA&#8209;Leu&#8209;TAA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_304</td><!-- Seq_Id -->
<td align="right">38,597</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">29.1%</td>
<td align="center">intergenic&nbsp;(+6/&#8209;2)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">tRNA&#8209;Gly&#8209;GCC/tRNA&#8209;Leu&#8209;TAA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_304</td><!-- Seq_Id -->
<td align="right">38,784</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">18.2%</td>
<td align="center">intergenic&nbsp;(+15/&#8209;6)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">tRNA&#8209;Arg&#8209;ACG/tRNA&#8209;Pro&#8209;TGG</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_304</td><!-- Seq_Id -->
<td align="right">38,785</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">18.3%</td>
<td align="center">intergenic&nbsp;(+16/&#8209;5)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">tRNA&#8209;Arg&#8209;ACG/tRNA&#8209;Pro&#8209;TGG</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_304</td><!-- Seq_Id -->
<td align="right">38,864</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">31.9%</td>
<td align="center">intergenic&nbsp;(+1/&#8209;23)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">tRNA&#8209;Pro&#8209;TGG/tRNA&#8209;Ala&#8209;TGC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_304</td><!-- Seq_Id -->
<td align="right">38,867</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">32.4%</td>
<td align="center">intergenic&nbsp;(+4/&#8209;20)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">tRNA&#8209;Pro&#8209;TGG/tRNA&#8209;Ala&#8209;TGC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_304</td><!-- Seq_Id -->
<td align="right">38,869</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">32.4%</td>
<td align="center">intergenic&nbsp;(+6/&#8209;18)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">tRNA&#8209;Pro&#8209;TGG/tRNA&#8209;Ala&#8209;TGC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_304</td><!-- Seq_Id -->
<td align="right">38,872</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">32.3%</td>
<td align="center">intergenic&nbsp;(+9/&#8209;15)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">tRNA&#8209;Pro&#8209;TGG/tRNA&#8209;Ala&#8209;TGC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_304</td><!-- Seq_Id -->
<td align="right">38,873</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">32.0%</td>
<td align="center">intergenic&nbsp;(+10/&#8209;14)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">tRNA&#8209;Pro&#8209;TGG/tRNA&#8209;Ala&#8209;TGC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_304</td><!-- Seq_Id -->
<td align="right">38,879</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">32.4%</td>
<td align="center">intergenic&nbsp;(+16/&#8209;8)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">tRNA&#8209;Pro&#8209;TGG/tRNA&#8209;Ala&#8209;TGC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_305</td><!-- Seq_Id -->
<td align="right">7,525</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">S65P</font>&nbsp;(<font class="mutation_in_codon">T</font>CA&rarr;<font class="mutation_in_codon">C</font>CA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Hypothetical protein SAV1852</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_305</td><!-- Seq_Id -->
<td align="right">11,218</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q158R</font>&nbsp;(C<font class="mutation_in_codon">A</font>A&rarr;C<font class="mutation_in_codon">G</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Two&#8209;component system sensor histidine kinase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_305</td><!-- Seq_Id -->
<td align="right">21,808</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center"><font class="snp_type_synonymous">A101A</font>&nbsp;(GC<font class="mutation_in_codon">C</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Foldase protein PrsA precursor (EC 5.2.1.8) @ Foldase clustered with pyrimidine conversion</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_305</td><!-- Seq_Id -->
<td align="right">32,235</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">51.0%</td>
<td align="center">intergenic&nbsp;(+400/&#8209;1)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">Hypothetical protein SAV1831/tRNA&#8209;Met&#8209;CAT</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_309</td><!-- Seq_Id -->
<td align="right">33,395</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">E335K</font>&nbsp;(<font class="mutation_in_codon">G</font>AA&rarr;<font class="mutation_in_codon">A</font>AA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Membrane anchored protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_309</td><!-- Seq_Id -->
<td align="right">47,047</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">T237N</font>&nbsp;(A<font class="mutation_in_codon">C</font>T&rarr;A<font class="mutation_in_codon">A</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phage DNA helicase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_312</td><!-- Seq_Id -->
<td align="right">9,515</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">N72H</font>&nbsp;(<font class="mutation_in_codon">A</font>AT&rarr;<font class="mutation_in_codon">C</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Zn peptidase with DNA binding</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_314</td><!-- Seq_Id -->
<td align="right">23,016</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">A171T</font>&nbsp;(<font class="mutation_in_codon">G</font>CA&rarr;<font class="mutation_in_codon">A</font>CA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Cell envelope stress response system LiaFSR, response regulator LiaR(VraR)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_314</td><!-- Seq_Id -->
<td align="right">31,620</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">H22N</font>&nbsp;(<font class="mutation_in_codon">C</font>AC&rarr;<font class="mutation_in_codon">A</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Bacterial non&#8209;heme ferritin (EC 1.16.3.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_314</td><!-- Seq_Id -->
<td align="right">71,844</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">D163Y</font>&nbsp;(<font class="mutation_in_codon">G</font>AT&rarr;<font class="mutation_in_codon">T</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_314</td><!-- Seq_Id -->
<td align="right">75,881</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">F129S</font>&nbsp;(T<font class="mutation_in_codon">T</font>T&rarr;T<font class="mutation_in_codon">C</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ABC transporter, ATP&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_318</td><!-- Seq_Id -->
<td align="right">33,747</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">K206E</font>&nbsp;(<font class="mutation_in_codon">A</font>AG&rarr;<font class="mutation_in_codon">G</font>AG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Arginase (EC 3.5.3.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_319</td><!-- Seq_Id -->
<td align="right">17,511</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_synonymous">F265F</font>&nbsp;(TT<font class="mutation_in_codon">C</font>&rarr;TT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_319</td><!-- Seq_Id -->
<td align="right">18,888</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_synonymous">S422S</font>&nbsp;(TC<font class="mutation_in_codon">T</font>&rarr;TC<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Dihydrolipoamide succinyltransferase component (E2) of 2&#8209;oxoglutarate dehydrogenase complex (EC 2.3.1.61)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_319</td><!-- Seq_Id -->
<td align="right">38,138</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">G40R</font>&nbsp;(<font class="mutation_in_codon">G</font>GG&rarr;<font class="mutation_in_codon">A</font>GG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_319</td><!-- Seq_Id -->
<td align="right">40,502</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">A1978V</font>&nbsp;(G<font class="mutation_in_codon">C</font>A&rarr;G<font class="mutation_in_codon">T</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Putative Staphylococcal surface anchored protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_319</td><!-- Seq_Id -->
<td align="right">41,125</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_synonymous">I1770I</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Putative Staphylococcal surface anchored protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_319</td><!-- Seq_Id -->
<td align="right">123,662</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_synonymous">L9L</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein YpbB</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_322</td><!-- Seq_Id -->
<td align="right">5,556</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">T111A</font>&nbsp;(<font class="mutation_in_codon">A</font>CA&rarr;<font class="mutation_in_codon">G</font>CA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phosphoenolpyruvate&#8209;protein phosphotransferase of PTS system (EC 2.7.3.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_322</td><!-- Seq_Id -->
<td align="right">33,425</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">51.2%</td>
<td align="center">coding&nbsp;(172/1194&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FmtA protein involved in methicillin resistance</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_322</td><!-- Seq_Id -->
<td align="right">68,682</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">D297Y</font>&nbsp;(<font class="mutation_in_codon">G</font>AT&rarr;<font class="mutation_in_codon">T</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Serine protease, DegP/HtrA, do&#8209;like (EC 3.4.21.&#8209;)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_322</td><!-- Seq_Id -->
<td align="right">72,156</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">A106S</font>&nbsp;(<font class="mutation_in_codon">G</font>CT&rarr;<font class="mutation_in_codon">T</font>CT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Peptide chain release factor 3</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_322</td><!-- Seq_Id -->
<td align="right">74,519</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center">intergenic&nbsp;(&#8209;323/&#8209;110)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">UDP&#8209;N&#8209;acetylmuramoy&#8209;dipeptide&#8209;&#8209;L&#8209;lysine ligase (EC 6.3.2.7)/Processive diacylglycerol beta&#8209;glucosyltransferase (EC 2.4.1.315)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_322</td><!-- Seq_Id -->
<td align="right">92,569</td><!-- Position -->
<td align="center">&Delta;4&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">11.7%</td>
<td align="center">coding&nbsp;(641&#8209;644/1809&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Oligoendopeptidase F</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_322</td><!-- Seq_Id -->
<td align="right">114,895</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">13.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">R125G</font>&nbsp;(<font class="mutation_in_codon">A</font>GG&rarr;<font class="mutation_in_codon">G</font>GG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Threonylcarbamoyl&#8209;AMP synthase (EC 2.7.7.87)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_322</td><!-- Seq_Id -->
<td align="right">160,825</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">A27E</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">A</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">NADH dehydrogenase (EC 1.6.99.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">96,208</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">A70D</font>&nbsp;(G<font class="mutation_in_codon">C</font>T&rarr;G<font class="mutation_in_codon">A</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Cyclic&#8209;di&#8209;AMP phosphodiesterase GdpP</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,456</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_synonymous">D609D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,459</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_synonymous">S610S</font>&nbsp;(TC<font class="mutation_in_codon">G</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,468</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_synonymous">D613D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,471</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">8.2%</td>
<td align="center"><font class="snp_type_synonymous">S614S</font>&nbsp;(AG<font class="mutation_in_codon">T</font>&rarr;AG<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,498</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">17.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">E623D</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,504</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">16.8%</td>
<td align="center"><font class="snp_type_synonymous">D625D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,516</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">17.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">E629D</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,528</td><!-- Position -->
<td align="center">28&nbsp;bp&rarr;88&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">21.3%</td>
<td align="center">coding&nbsp;(1899&#8209;1926/2634&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,669</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_synonymous">S680S</font>&nbsp;(AG<font class="mutation_in_codon">T</font>&rarr;AG<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,672</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.0%</td>
<td align="center"><font class="snp_type_synonymous">D681D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,675</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_synonymous">S682S</font>&nbsp;(TC<font class="mutation_in_codon">G</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,696</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.6%</td>
<td align="center"><font class="snp_type_synonymous">D689D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,702</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">11.6%</td>
<td align="center"><font class="snp_type_synonymous">D691D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,714</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">11.9%</td>
<td align="center"><font class="snp_type_synonymous">D695D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,738:1</td><!-- Position -->
<td align="center">+T</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center">coding&nbsp;(2109/2634&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,738:2</td><!-- Position -->
<td align="center">+C</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center">coding&nbsp;(2109/2634&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,738:3</td><!-- Position -->
<td align="center">+A</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center">coding&nbsp;(2109/2634&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,738:4</td><!-- Position -->
<td align="center">+G</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center">coding&nbsp;(2109/2634&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,738:5</td><!-- Position -->
<td align="center">+A</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center">coding&nbsp;(2109/2634&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,738:6</td><!-- Position -->
<td align="center">+T</td><!-- Cell Mutation -->
<td align="right">9.7%</td>
<td align="center">coding&nbsp;(2109/2634&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,759</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.2%</td>
<td align="center"><font class="snp_type_synonymous">S710S</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,762</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center"><font class="snp_type_synonymous">D711D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,771</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_synonymous">S714S</font>&nbsp;(TC<font class="mutation_in_codon">A</font>&rarr;TC<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,786</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.4%</td>
<td align="center"><font class="snp_type_synonymous">D719D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,918</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.0%</td>
<td align="center"><font class="snp_type_synonymous">D763D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">210,930</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_synonymous">D767D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_324</td><!-- Seq_Id -->
<td align="right">218,680</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">M429I</font>&nbsp;(AT<font class="mutation_in_codon">G</font>&rarr;AT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ABC transporter&#8209;like sensor and permease protein SAV2622</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">46,698</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_synonymous">E142E</font>&nbsp;(GA<font class="mutation_in_codon">G</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Deoxyguanosine kinase (EC 2.7.1.113)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">51,436</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">D582N</font>&nbsp;(<font class="mutation_in_codon">G</font>AT&rarr;<font class="mutation_in_codon">A</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">51,477</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.8%</td>
<td align="center"><font class="snp_type_synonymous">N595N</font>&nbsp;(AA<font class="mutation_in_codon">C</font>&rarr;AA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">51,603</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.6%</td>
<td align="center"><font class="snp_type_synonymous">T637T</font>&nbsp;(AC<font class="mutation_in_codon">T</font>&rarr;AC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">51,702</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">100%</td>
<td align="center"><font class="snp_type_synonymous">I670I</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">51,870</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.9%</td>
<td align="center"><font class="snp_type_synonymous">S726S</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">51,897</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">18.2%</td>
<td align="center"><font class="snp_type_synonymous">D735D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">51,909</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.8%</td>
<td align="center"><font class="snp_type_synonymous">D739D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">51,939</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">17.4%</td>
<td align="center"><font class="snp_type_synonymous">D749D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">51,945</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">22.3%</td>
<td align="center"><font class="snp_type_synonymous">D751D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">51,945</td><!-- Position -->
<td align="center">+TCAGAT</td><!-- Cell Mutation -->
<td align="right">18.2%</td>
<td align="center">coding&nbsp;(2253/2658&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">51,951</td><!-- Position -->
<td align="center">+60 bp</td><!-- Cell Mutation -->
<td align="right">18.3%</td>
<td align="center">coding&nbsp;(2259/2658&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">51,951</td><!-- Position -->
<td align="center">+144 bp</td><!-- Cell Mutation -->
<td align="right">20.1%</td>
<td align="center">coding&nbsp;(2259/2658&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">51,984</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">32.0%</td>
<td align="center"><font class="snp_type_synonymous">S764S</font>&nbsp;(TC<font class="mutation_in_codon">G</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">51,993</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.6%</td>
<td align="center"><font class="snp_type_synonymous">D767D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">52,026:1</td><!-- Position -->
<td align="center">+G</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center">coding&nbsp;(2334/2658&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">52,026:2</td><!-- Position -->
<td align="center">+A</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center">coding&nbsp;(2334/2658&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">52,026:3</td><!-- Position -->
<td align="center">+T</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center">coding&nbsp;(2334/2658&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">52,026:4</td><!-- Position -->
<td align="center">+T</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center">coding&nbsp;(2334/2658&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">52,026:5</td><!-- Position -->
<td align="center">+C</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center">coding&nbsp;(2334/2658&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">52,026:6</td><!-- Position -->
<td align="center">+A</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center">coding&nbsp;(2334/2658&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">52,029</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">14.2%</td>
<td align="center"><font class="snp_type_synonymous">D779D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">52,035</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">14.2%</td>
<td align="center"><font class="snp_type_synonymous">D781D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">52,041</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">10.8%</td>
<td align="center"><font class="snp_type_synonymous">D783D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">52,047</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">13.9%</td>
<td align="center"><font class="snp_type_synonymous">D785D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">52,074</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">13.8%</td>
<td align="center"><font class="snp_type_synonymous">S794S</font>&nbsp;(TC<font class="mutation_in_codon">A</font>&rarr;TC<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">52,101</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.9%</td>
<td align="center"><font class="snp_type_synonymous">D803D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">52,107</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">26.0%</td>
<td align="center"><font class="snp_type_synonymous">D805D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">52,134</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">26.9%</td>
<td align="center"><font class="snp_type_synonymous">S814S</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_325</td><!-- Seq_Id -->
<td align="right">52,308</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.2%</td>
<td align="center"><font class="snp_type_synonymous">G872G</font>&nbsp;(GG<font class="mutation_in_codon">T</font>&rarr;GG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_326</td><!-- Seq_Id -->
<td align="right">33,212</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">S76Y</font>&nbsp;(T<font class="mutation_in_codon">C</font>T&rarr;T<font class="mutation_in_codon">A</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">UPF0118 membrane protein YubA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_326</td><!-- Seq_Id -->
<td align="right">42,238</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center">intergenic&nbsp;(+337/&#8209;39)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Acyl&#8209;phosphate:glycerol&#8209;3&#8209;phosphate O&#8209;acyltransferase PlsY (EC 2.3.1.n3)/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_326</td><!-- Seq_Id -->
<td align="right">46,708</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.1%</td>
<td align="center">intergenic&nbsp;(&#8209;576/+371)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Aconitate hydratase (EC 4.2.1.3)/Glycine betaine transporter OpuD</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_326</td><!-- Seq_Id -->
<td align="right">54,737</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center">intergenic&nbsp;(&#8209;209/+69)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">UPF0154 protein YneF/Transketolase (EC 2.2.1.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_326</td><!-- Seq_Id -->
<td align="right">58,732</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">G268R</font>&nbsp;(<font class="mutation_in_codon">G</font>GT&rarr;<font class="mutation_in_codon">C</font>GT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_326</td><!-- Seq_Id -->
<td align="right">67,211</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsense">Q124*</font>&nbsp;(<font class="mutation_in_codon">C</font>AA&rarr;<font class="mutation_in_codon">T</font>AA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Homoserine kinase (EC 2.7.1.39)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_326</td><!-- Seq_Id -->
<td align="right">112,105</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center"><font class="snp_type_synonymous">T449T</font>&nbsp;(AC<font class="mutation_in_codon">G</font>&rarr;AC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Ribonuclease Y</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_326</td><!-- Seq_Id -->
<td align="right">136,469</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center">intergenic&nbsp;(&#8209;17/+4)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Transcription termination protein NusA/Bacterial ribosome SSU maturation protein RimP</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_326</td><!-- Seq_Id -->
<td align="right">165,554</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">P118T</font>&nbsp;(<font class="mutation_in_codon">C</font>CT&rarr;<font class="mutation_in_codon">A</font>CT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">LSU ribosomal maturation GTPase RbgA (B. subtilis YlqF)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_326</td><!-- Seq_Id -->
<td align="right">197,480</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">A41T</font>&nbsp;(<font class="mutation_in_codon">G</font>CA&rarr;<font class="mutation_in_codon">A</font>CA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Peptide deformylase (EC 3.5.1.88)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_326</td><!-- Seq_Id -->
<td align="right">205,561</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q300R</font>&nbsp;(C<font class="mutation_in_codon">A</font>G&rarr;C<font class="mutation_in_codon">G</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Fibronectin/fibrinogen&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_326</td><!-- Seq_Id -->
<td align="right">206,362</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.2%</td>
<td align="center">intergenic&nbsp;(+2/&#8209;74)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i></td>
<td align="left">Fibronectin/fibrinogen&#8209;binding protein/FIG01108508: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_326</td><!-- Seq_Id -->
<td align="right">222,292</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">D607N</font>&nbsp;(<font class="mutation_in_codon">G</font>AC&rarr;<font class="mutation_in_codon">A</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Isoleucyl&#8209;tRNA synthetase (EC 6.1.1.5)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_326</td><!-- Seq_Id -->
<td align="right">240,041</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.8%</td>
<td align="center"><font class="snp_type_synonymous">D287D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glucosaminyl&#8209;malate:cysteine ligase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_326</td><!-- Seq_Id -->
<td align="right">247,981</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">G201C</font>&nbsp;(<font class="mutation_in_codon">G</font>GT&rarr;<font class="mutation_in_codon">T</font>GT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Carbamate kinase (EC 2.7.2.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_332</td><!-- Seq_Id -->
<td align="right">14,336</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">E141D</font>&nbsp;(GA<font class="mutation_in_codon">G</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Hypothetical protein SAV2178</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_332</td><!-- Seq_Id -->
<td align="right">14,396</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_synonymous">V121V</font>&nbsp;(GT<font class="mutation_in_codon">A</font>&rarr;GT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Hypothetical protein SAV2178</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_332</td><!-- Seq_Id -->
<td align="right">70,829</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">12.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">I65M</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">LSU ribosomal protein L2p (L8e)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_332</td><!-- Seq_Id -->
<td align="right">92,464</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_synonymous">Q56Q</font>&nbsp;(CA<font class="mutation_in_codon">A</font>&rarr;CA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Molybdenum cofactor guanylyltransferase (EC 2.7.7.77)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_332</td><!-- Seq_Id -->
<td align="right">153,829</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">I55T</font>&nbsp;(A<font class="mutation_in_codon">T</font>A&rarr;A<font class="mutation_in_codon">C</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Fosfomycin resistance protein FosB</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center">k141_332</td><!-- Seq_Id -->
<td align="right">156,457:1</td><!-- Position -->
<td align="center">+A</td><!-- Cell Mutation -->
<td align="right">100%</td>
<td align="center">coding&nbsp;(892/1260&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_332</td><!-- Seq_Id -->
<td align="right">164,371</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center">intergenic&nbsp;(+155/&#8209;178)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">DNA&#8209;3&#8209;methyladenine glycosylase II (EC 3.2.2.21)/Sodium/glutamate symporter</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_335</td><!-- Seq_Id -->
<td align="right">35,685</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center">intergenic&nbsp;(&#8209;89/+724)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Central glycolytic genes regulator/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_335</td><!-- Seq_Id -->
<td align="right">99,015</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_synonymous">Q558Q</font>&nbsp;(CA<font class="mutation_in_codon">A</font>&rarr;CA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Lipoteichoic acid synthase LtaS Type Ib</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_335</td><!-- Seq_Id -->
<td align="right">104,912</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">L23S</font>&nbsp;(T<font class="mutation_in_codon">T</font>A&rarr;T<font class="mutation_in_codon">C</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Para&#8209;aminobenzoate synthase, aminase component (EC 2.6.1.85)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_335</td><!-- Seq_Id -->
<td align="right">108,334</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">15.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">Y175H</font>&nbsp;(<font class="mutation_in_codon">T</font>AT&rarr;<font class="mutation_in_codon">C</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_336</td><!-- Seq_Id -->
<td align="right">21,104</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">D20G</font>&nbsp;(G<font class="mutation_in_codon">A</font>T&rarr;G<font class="mutation_in_codon">G</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transposase, IS4 family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_336</td><!-- Seq_Id -->
<td align="right">95,432</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">V76F</font>&nbsp;(<font class="mutation_in_codon">G</font>TT&rarr;<font class="mutation_in_codon">T</font>TT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;3&#8209;phosphoglycerate dehydrogenase (EC 1.1.1.95)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_336</td><!-- Seq_Id -->
<td align="right">184,566</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">A320V</font>&nbsp;(G<font class="mutation_in_codon">C</font>T&rarr;G<font class="mutation_in_codon">T</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">S&#8209;adenosylmethionine:tRNA ribosyltransferase&#8209;isomerase (EC 2.4.99.17)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_336</td><!-- Seq_Id -->
<td align="right">199,036</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center"><font class="snp_type_synonymous">L567L</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Aspartyl&#8209;tRNA synthetase (EC 6.1.1.12)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_336</td><!-- Seq_Id -->
<td align="right">203,197</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">P267L</font>&nbsp;(C<font class="mutation_in_codon">C</font>G&rarr;C<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Flavin&#8209;utilizing monoxygenase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_336</td><!-- Seq_Id -->
<td align="right">236,431</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">N358D</font>&nbsp;(<font class="mutation_in_codon">A</font>AT&rarr;<font class="mutation_in_codon">G</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">DNA internalization&#8209;related competence protein ComEC/Rec2</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_336</td><!-- Seq_Id -->
<td align="right">294,141</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_synonymous">S6S</font>&nbsp;(TC<font class="mutation_in_codon">C</font>&rarr;TC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcription termination protein NusB</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_336</td><!-- Seq_Id -->
<td align="right">305,396</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">A73T</font>&nbsp;(<font class="mutation_in_codon">G</font>CA&rarr;<font class="mutation_in_codon">A</font>CA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_336</td><!-- Seq_Id -->
<td align="right">323,700</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">M264T</font>&nbsp;(A<font class="mutation_in_codon">T</font>G&rarr;A<font class="mutation_in_codon">C</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Site&#8209;specific tyrosine recombinase XerD</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_337</td><!-- Seq_Id -->
<td align="right">124,675</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_synonymous">G448G</font>&nbsp;(GG<font class="mutation_in_codon">G</font>&rarr;GG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Virulence&#8209;associated cell&#8209;wall&#8209;anchored protein SasG (LPXTG motif), binding to squamous nasal epithelial cells</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_337</td><!-- Seq_Id -->
<td align="right">131,656</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">F171Y</font>&nbsp;(T<font class="mutation_in_codon">T</font>T&rarr;T<font class="mutation_in_codon">A</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">DNA/RNA helicase of DEAD/DEAH box family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_337</td><!-- Seq_Id -->
<td align="right">159,649</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_synonymous">I65I</font>&nbsp;(AT<font class="mutation_in_codon">T</font>&rarr;AT<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_337</td><!-- Seq_Id -->
<td align="right">177,268</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">L281S</font>&nbsp;(T<font class="mutation_in_codon">T</font>A&rarr;T<font class="mutation_in_codon">C</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Uncharacterized MFS&#8209;type transporter</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_337</td><!-- Seq_Id -->
<td align="right">203,356</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center"><font class="snp_type_synonymous">E100E</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Pimeloyl&#8209;CoA synthase (EC 6.2.1.14)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_337</td><!-- Seq_Id -->
<td align="right">207,189</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.8%</td>
<td align="center"><font class="snp_type_synonymous">Y183Y</font>&nbsp;(TA<font class="mutation_in_codon">T</font>&rarr;TA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Cytolytic pore&#8209;forming protein S component => Gamma&#8209;hemolysin HlgA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_339</td><!-- Seq_Id -->
<td align="right">642</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">E19G</font>&nbsp;(G<font class="mutation_in_codon">A</font>A&rarr;G<font class="mutation_in_codon">G</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_339</td><!-- Seq_Id -->
<td align="right">94,206</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center"><font class="snp_type_synonymous">M1M</font>&nbsp;(AT<font class="mutation_in_codon">G</font>&rarr;AT<font class="mutation_in_codon">T</font>)&nbsp;&dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">FIG01108149: hypothetical protein</td>
</tr>
<!-- End Table Row -->
</table></body></html>