<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>Mutation Comparison</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<!--Output Html_Mutation_Table_String-->
<table border="0" cellspacing="1" cellpadding="3">
<tr><th colspan="7" align="left" class="mutation_header_row">Predicted mutations</th></tr><tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>

<!-- Item Lines -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_100</td><!-- Seq_Id -->
<td align="right">2,151</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">E485D</font>&nbsp;(GA<font class="mutation_in_codon">G</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Nitric oxide reductase activation protein NorD</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_100</td><!-- Seq_Id -->
<td align="right">2,891</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">A239S</font>&nbsp;(<font class="mutation_in_codon">G</font>CT&rarr;<font class="mutation_in_codon">T</font>CT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Nitric oxide reductase activation protein NorD</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_101</td><!-- Seq_Id -->
<td align="right">10,622</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">N147T</font>&nbsp;(A<font class="mutation_in_codon">A</font>T&rarr;A<font class="mutation_in_codon">C</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Hypothetical protein SAV1803</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_101</td><!-- Seq_Id -->
<td align="right">10,632</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">G144S</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">A</font>GC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Hypothetical protein SAV1803</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_101</td><!-- Seq_Id -->
<td align="right">10,698</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">H122N</font>&nbsp;(<font class="mutation_in_codon">C</font>AT&rarr;<font class="mutation_in_codon">A</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Hypothetical protein SAV1803</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_103</td><!-- Seq_Id -->
<td align="right">3,257</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.6%</td>
<td align="center">intergenic&nbsp;(+55/&#8209;142)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Cold shock protein of CSP family/Protein msa (Modulator of sarA)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_103</td><!-- Seq_Id -->
<td align="right">8,488</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_synonymous">A220A</font>&nbsp;(GC<font class="mutation_in_codon">A</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">4&#8209;hydroxy&#8209;tetrahydrodipicolinate reductase (EC 1.17.1.8)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_103</td><!-- Seq_Id -->
<td align="right">15,138</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.3%</td>
<td align="center"><font class="snp_type_synonymous">D80D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Uncharacterized S1 RNA binding domain protein YitL</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_106</td><!-- Seq_Id -->
<td align="right">49</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">10.5%</td>
<td align="center"><font class="snp_type_synonymous">L89L</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">G</font>)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transposase for insertion sequence&#8209;like element IS431mec</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_106</td><!-- Seq_Id -->
<td align="right">51</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">9.8%</td>
<td align="center"><font class="snp_type_synonymous">L89L</font>&nbsp;(<font class="mutation_in_codon">T</font>TA&rarr;<font class="mutation_in_codon">C</font>TA)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transposase for insertion sequence&#8209;like element IS431mec</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_106</td><!-- Seq_Id -->
<td align="right">55</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.0%</td>
<td align="center"><font class="snp_type_synonymous">S87S</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transposase for insertion sequence&#8209;like element IS431mec</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_106</td><!-- Seq_Id -->
<td align="right">57</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">S87C</font>&nbsp;(<font class="mutation_in_codon">A</font>GC&rarr;<font class="mutation_in_codon">T</font>GC)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transposase for insertion sequence&#8209;like element IS431mec</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_106</td><!-- Seq_Id -->
<td align="right">61</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">10.0%</td>
<td align="center"><font class="snp_type_synonymous">K85K</font>&nbsp;(AA<font class="mutation_in_codon">A</font>&rarr;AA<font class="mutation_in_codon">G</font>)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transposase for insertion sequence&#8209;like element IS431mec</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_106</td><!-- Seq_Id -->
<td align="right">63</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">K85Q</font>&nbsp;(<font class="mutation_in_codon">A</font>AA&rarr;<font class="mutation_in_codon">C</font>AA)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Transposase for insertion sequence&#8209;like element IS431mec</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_107</td><!-- Seq_Id -->
<td align="right">10,874</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">P711L</font>&nbsp;(C<font class="mutation_in_codon">C</font>T&rarr;C<font class="mutation_in_codon">T</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">DNA primase, phage associated</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_108</td><!-- Seq_Id -->
<td align="right">24,297</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">D177E</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Fumarate hydratase class II (EC 4.2.1.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_108</td><!-- Seq_Id -->
<td align="right">24,306</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_synonymous">D174D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Fumarate hydratase class II (EC 4.2.1.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_108</td><!-- Seq_Id -->
<td align="right">29,171</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_synonymous">V147V</font>&nbsp;(GT<font class="mutation_in_codon">A</font>&rarr;GT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glutamine ABC transporter, ATP&#8209;binding protein GlnQ</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_108</td><!-- Seq_Id -->
<td align="right">29,180</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">7.2%</td>
<td align="center"><font class="snp_type_synonymous">K144K</font>&nbsp;(AA<font class="mutation_in_codon">A</font>&rarr;AA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glutamine ABC transporter, ATP&#8209;binding protein GlnQ</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_111</td><!-- Seq_Id -->
<td align="right">9,350</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center">intergenic&nbsp;(&#8209;7/&#8209;201)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/Putative esterase</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_111</td><!-- Seq_Id -->
<td align="right">16,963</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">7.2%</td>
<td align="center">intergenic&nbsp;(+106/&#8209;125)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/Efflux ABC transporter, ATP&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_111</td><!-- Seq_Id -->
<td align="right">16,965</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.9%</td>
<td align="center">intergenic&nbsp;(+108/&#8209;123)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/Efflux ABC transporter, ATP&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_111</td><!-- Seq_Id -->
<td align="right">16,967</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center">intergenic&nbsp;(+110/&#8209;121)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/Efflux ABC transporter, ATP&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_111</td><!-- Seq_Id -->
<td align="right">16,968</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.9%</td>
<td align="center">intergenic&nbsp;(+111/&#8209;120)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/Efflux ABC transporter, ATP&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_111</td><!-- Seq_Id -->
<td align="right">16,980</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.2%</td>
<td align="center">intergenic&nbsp;(+123/&#8209;108)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/Efflux ABC transporter, ATP&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_111</td><!-- Seq_Id -->
<td align="right">17,013</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">8.2%</td>
<td align="center">intergenic&nbsp;(+156/&#8209;75)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/Efflux ABC transporter, ATP&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_111</td><!-- Seq_Id -->
<td align="right">17,041</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center">intergenic&nbsp;(+184/&#8209;47)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/Efflux ABC transporter, ATP&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_111</td><!-- Seq_Id -->
<td align="right">23,316</td><!-- Position -->
<td align="center">+GAGAAT</td><!-- Cell Mutation -->
<td align="right">29.5%</td>
<td align="center">coding&nbsp;(564/1260&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_112</td><!-- Seq_Id -->
<td align="right">162</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">R8K</font>&nbsp;(A<font class="mutation_in_codon">G</font>A&rarr;A<font class="mutation_in_codon">A</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transposase, IS4 family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_112</td><!-- Seq_Id -->
<td align="right">18,898</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center"><font class="snp_type_synonymous">V587V</font>&nbsp;(GT<font class="mutation_in_codon">C</font>&rarr;GT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">FtsK/SpoIIIE family protein, putative EssC/YukB component of Type VII secretion system</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_112</td><!-- Seq_Id -->
<td align="right">18,917</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_synonymous">R594R</font>&nbsp;(<font class="mutation_in_codon">C</font>GA&rarr;<font class="mutation_in_codon">A</font>GA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">FtsK/SpoIIIE family protein, putative EssC/YukB component of Type VII secretion system</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_112</td><!-- Seq_Id -->
<td align="right">22,718</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_synonymous">K131K</font>&nbsp;(AA<font class="mutation_in_codon">G</font>&rarr;AA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">SAV0291 homolog within ESAT&#8209;6 gene cluster</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center">k141_113</td><!-- Seq_Id -->
<td align="right">1,286</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">100%</td>
<td align="center"><font class="snp_type_synonymous">I428I</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative major teichoic acid biosynthesis protein C</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_115</td><!-- Seq_Id -->
<td align="right">4,122</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">T4N</font>&nbsp;(A<font class="mutation_in_codon">C</font>T&rarr;A<font class="mutation_in_codon">A</font>T)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein within prophage</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_115</td><!-- Seq_Id -->
<td align="right">4,123</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_synonymous">T4T</font>&nbsp;(AC<font class="mutation_in_codon">T</font>&rarr;AC<font class="mutation_in_codon">A</font>)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein within prophage</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_115</td><!-- Seq_Id -->
<td align="right">4,148</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">V13I</font>&nbsp;(<font class="mutation_in_codon">G</font>TA&rarr;<font class="mutation_in_codon">A</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein within prophage</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_115</td><!-- Seq_Id -->
<td align="right">8,431</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_synonymous">C6C</font>&nbsp;(TG<font class="mutation_in_codon">T</font>&rarr;TG<font class="mutation_in_codon">C</font>)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Cytolytic pore&#8209;forming protein F component => Leukocidin LukF&#8209;PV</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_115</td><!-- Seq_Id -->
<td align="right">8,432</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">C6Y</font>&nbsp;(T<font class="mutation_in_codon">G</font>T&rarr;T<font class="mutation_in_codon">A</font>T)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Cytolytic pore&#8209;forming protein F component => Leukocidin LukF&#8209;PV</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_115</td><!-- Seq_Id -->
<td align="right">8,436</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_synonymous">L5L</font>&nbsp;(<font class="mutation_in_codon">C</font>TA&rarr;<font class="mutation_in_codon">T</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Cytolytic pore&#8209;forming protein F component => Leukocidin LukF&#8209;PV</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_115</td><!-- Seq_Id -->
<td align="right">14,142</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_synonymous">V203V</font>&nbsp;(GT<font class="mutation_in_codon">G</font>&rarr;GT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">KtrAB potassium uptake system, integral membrane component KtrB</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_115</td><!-- Seq_Id -->
<td align="right">27,177</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.6%</td>
<td align="center"><font class="snp_type_synonymous">I73I</font>&nbsp;(AT<font class="mutation_in_codon">A</font>&rarr;AT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Accessory gene regulator C (sensor histidine kinase)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_115</td><!-- Seq_Id -->
<td align="right">45,124</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_synonymous">P529P</font>&nbsp;(CC<font class="mutation_in_codon">A</font>&rarr;CC<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Dihydroxy&#8209;acid dehydratase (EC 4.2.1.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_115</td><!-- Seq_Id -->
<td align="right">48,270</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">F286S</font>&nbsp;(T<font class="mutation_in_codon">T</font>C&rarr;T<font class="mutation_in_codon">C</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Ketol&#8209;acid reductoisomerase (NADP(+)) (EC 1.1.1.86)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_115</td><!-- Seq_Id -->
<td align="right">51,618</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.2%</td>
<td align="center"><font class="snp_type_synonymous">D193D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">3&#8209;isopropylmalate dehydratase large subunit (EC 4.2.1.33)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_115</td><!-- Seq_Id -->
<td align="right">51,678</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_synonymous">D213D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">3&#8209;isopropylmalate dehydratase large subunit (EC 4.2.1.33)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_115</td><!-- Seq_Id -->
<td align="right">53,570</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_synonymous">G191G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Threonine dehydratase biosynthetic (EC 4.3.1.19)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_115</td><!-- Seq_Id -->
<td align="right">53,573</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_synonymous">G192G</font>&nbsp;(GG<font class="mutation_in_codon">T</font>&rarr;GG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Threonine dehydratase biosynthetic (EC 4.3.1.19)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_116</td><!-- Seq_Id -->
<td align="right">9,034</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center">intergenic&nbsp;(&#8209;358/+113)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i></td>
<td align="left">PTS system, trehalose&#8209;specific IIB component (EC 2.7.1.201) / PTS system, trehalose&#8209;specific IIC component/tRNA&#8209;Ser&#8209;TGA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_116</td><!-- Seq_Id -->
<td align="right">47,204</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">I147V</font>&nbsp;(<font class="mutation_in_codon">A</font>TA&rarr;<font class="mutation_in_codon">G</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Tandem lipoprotein within Pathogenicity island</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_121</td><!-- Seq_Id -->
<td align="right">157</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">13.8%</td>
<td align="center">intergenic&nbsp;(&#8211;/+621)</td>
<td align="center"><i>&#8211;</i>&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">&#8211;/Peroxide stress regulator PerR, FUR family</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_121</td><!-- Seq_Id -->
<td align="right">8,138</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">D38Y</font>&nbsp;(<font class="mutation_in_codon">G</font>AT&rarr;<font class="mutation_in_codon">T</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">DUF402 family nucleoside diphosphatase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_121</td><!-- Seq_Id -->
<td align="right">11,881</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">G487V</font>&nbsp;(G<font class="mutation_in_codon">G</font>G&rarr;G<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Teichoic acid export ATP&#8209;binding protein TagH (EC 3.6.3.40)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_121</td><!-- Seq_Id -->
<td align="right">12,954</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_synonymous">G129G</font>&nbsp;(GG<font class="mutation_in_codon">T</font>&rarr;GG<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Teichoic acid export ATP&#8209;binding protein TagH (EC 3.6.3.40)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_121</td><!-- Seq_Id -->
<td align="right">49,047</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">M127V</font>&nbsp;(<font class="mutation_in_codon">A</font>TG&rarr;<font class="mutation_in_codon">G</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Heptaprenylglyceryl phosphate synthase (EC 2.5.1.n9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_121</td><!-- Seq_Id -->
<td align="right">49,093</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_synonymous">E111E</font>&nbsp;(GA<font class="mutation_in_codon">G</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Heptaprenylglyceryl phosphate synthase (EC 2.5.1.n9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_121</td><!-- Seq_Id -->
<td align="right">56,507</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.3%</td>
<td align="center"><font class="snp_type_synonymous">L261L</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Nicotinate phosphoribosyltransferase (EC 6.3.4.21)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_121</td><!-- Seq_Id -->
<td align="right">64,009</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">D203G</font>&nbsp;(G<font class="mutation_in_codon">A</font>T&rarr;G<font class="mutation_in_codon">G</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Manganese&#8209;dependent inorganic pyrophosphatase (EC 3.6.1.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_122</td><!-- Seq_Id -->
<td align="right">2,836</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">S57T</font>&nbsp;(<font class="mutation_in_codon">T</font>CA&rarr;<font class="mutation_in_codon">A</font>CA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcriptional activator rinB, phage associated</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_122</td><!-- Seq_Id -->
<td align="right">2,895</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_synonymous">G17G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ORF120</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_122</td><!-- Seq_Id -->
<td align="right">2,904</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_synonymous">L20L</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ORF120</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_122</td><!-- Seq_Id -->
<td align="right">2,910</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.1%</td>
<td align="center"><font class="snp_type_synonymous">A22A</font>&nbsp;(GC<font class="mutation_in_codon">C</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ORF120</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_122</td><!-- Seq_Id -->
<td align="right">2,925</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.5%</td>
<td align="center"><font class="snp_type_synonymous">G27G</font>&nbsp;(GG<font class="mutation_in_codon">T</font>&rarr;GG<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ORF120</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_122</td><!-- Seq_Id -->
<td align="right">2,935</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">I31V</font>&nbsp;(<font class="mutation_in_codon">A</font>TT&rarr;<font class="mutation_in_codon">G</font>TT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ORF120</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_122</td><!-- Seq_Id -->
<td align="right">2,946</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.8%</td>
<td align="center"><font class="snp_type_synonymous">R34R</font>&nbsp;(CG<font class="mutation_in_codon">A</font>&rarr;CG<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ORF120</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_122</td><!-- Seq_Id -->
<td align="right">6,027</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">V234G</font>&nbsp;(G<font class="mutation_in_codon">T</font>T&rarr;G<font class="mutation_in_codon">G</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Phage portal (connector) protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_122</td><!-- Seq_Id -->
<td align="right">6,076</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_synonymous">N250N</font>&nbsp;(AA<font class="mutation_in_codon">T</font>&rarr;AA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Phage portal (connector) protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_122</td><!-- Seq_Id -->
<td align="right">6,096</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">D257G</font>&nbsp;(G<font class="mutation_in_codon">A</font>T&rarr;G<font class="mutation_in_codon">G</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Phage portal (connector) protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_123</td><!-- Seq_Id -->
<td align="right">18,187</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">K150N</font>&nbsp;(AA<font class="mutation_in_codon">A</font>&rarr;AA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phage lysin, N&#8209;acetylmuramoyl&#8209;L&#8209;alanine amidase (EC 3.5.1.28)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_123</td><!-- Seq_Id -->
<td align="right">20,213</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">I23T</font>&nbsp;(A<font class="mutation_in_codon">T</font>T&rarr;A<font class="mutation_in_codon">C</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phage protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_124</td><!-- Seq_Id -->
<td align="right">8,783</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_synonymous">G91G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Arsenite/antimonite:H+ antiporter ArsB</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_124</td><!-- Seq_Id -->
<td align="right">8,978</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_synonymous">L156L</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Arsenite/antimonite:H+ antiporter ArsB</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_124</td><!-- Seq_Id -->
<td align="right">13,508</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">S5P</font>&nbsp;(<font class="mutation_in_codon">T</font>CA&rarr;<font class="mutation_in_codon">C</font>CA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Hypothetical protein SAV1779</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_125</td><!-- Seq_Id -->
<td align="right">36,013</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_synonymous">G624G</font>&nbsp;(GG<font class="mutation_in_codon">G</font>&rarr;GG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcriptional regulator, AraC family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_125</td><!-- Seq_Id -->
<td align="right">37,585</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.3%</td>
<td align="center"><font class="snp_type_synonymous">A352A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">N&#8209;acyl&#8209;L&#8209;amino acid amidohydrolase (EC 3.5.1.14)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_125</td><!-- Seq_Id -->
<td align="right">43,410</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center">intergenic&nbsp;(&#8209;57/+117)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Oleate hydratase (EC 4.2.1.53)/Permease of the drug/metabolite transporter (DMT) superfamily</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_125</td><!-- Seq_Id -->
<td align="right">49,549</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.1%</td>
<td align="center"><font class="snp_type_synonymous">L245L</font>&nbsp;(<font class="mutation_in_codon">C</font>TA&rarr;<font class="mutation_in_codon">T</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Protein A, von Willebrand factor binding protein Spa</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_126</td><!-- Seq_Id -->
<td align="right">24,969</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">I459F</font>&nbsp;(<font class="mutation_in_codon">A</font>TC&rarr;<font class="mutation_in_codon">T</font>TC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">FIG006789: Stage V sporulation protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_127</td><!-- Seq_Id -->
<td align="right">11,080</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">7.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">L379S</font>&nbsp;(T<font class="mutation_in_codon">T</font>G&rarr;T<font class="mutation_in_codon">C</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ATP&#8209;dependent DNA helicase RecS (RecQ family)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_127</td><!-- Seq_Id -->
<td align="right">32,418</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_synonymous">N100N</font>&nbsp;(AA<font class="mutation_in_codon">T</font>&rarr;AA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">MazG nucleotide pyrophosphohydrolase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_127</td><!-- Seq_Id -->
<td align="right">32,976</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_synonymous">G100G</font>&nbsp;(GG<font class="mutation_in_codon">G</font>&rarr;GG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">UDP&#8209;N&#8209;acetylglucosamine:L&#8209;malate glycosyltransferase</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_127</td><!-- Seq_Id -->
<td align="right">44,680</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">30.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">A132T</font>&nbsp;(<font class="mutation_in_codon">G</font>CA&rarr;<font class="mutation_in_codon">A</font>CA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Multimodular transpeptidase&#8209;transglycosylase (EC 2.4.1.129) (EC 3.4.&#8209;.&#8209;) / Penicillin&#8209;binding protein 1A/1B (PBP1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_127</td><!-- Seq_Id -->
<td align="right">59,236</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_synonymous">G193G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Serine/threonine exchanger SteT</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_127</td><!-- Seq_Id -->
<td align="right">61,210</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">F358L</font>&nbsp;(TT<font class="mutation_in_codon">C</font>&rarr;TT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">MFS&#8209;type transporter quinolone resistance protein NorB/NorC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_127</td><!-- Seq_Id -->
<td align="right">74,306</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center"><font class="snp_type_synonymous">T4127T</font>&nbsp;(AC<font class="mutation_in_codon">A</font>&rarr;AC<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative Staphylococcal surface anchored protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_127</td><!-- Seq_Id -->
<td align="right">79,722</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">N5933H</font>&nbsp;(<font class="mutation_in_codon">A</font>AT&rarr;<font class="mutation_in_codon">C</font>AT)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative Staphylococcal surface anchored protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_127</td><!-- Seq_Id -->
<td align="right">79,723</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">N5933S</font>&nbsp;(A<font class="mutation_in_codon">A</font>T&rarr;A<font class="mutation_in_codon">G</font>T)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative Staphylococcal surface anchored protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_127</td><!-- Seq_Id -->
<td align="right">79,736</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_synonymous">D5937D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative Staphylococcal surface anchored protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_127</td><!-- Seq_Id -->
<td align="right">79,772</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">D5949E</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative Staphylococcal surface anchored protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_127</td><!-- Seq_Id -->
<td align="right">79,784</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">E5953D</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative Staphylococcal surface anchored protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_127</td><!-- Seq_Id -->
<td align="right">79,805</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center"><font class="snp_type_synonymous">D5960D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative Staphylococcal surface anchored protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_127</td><!-- Seq_Id -->
<td align="right">105,074</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_synonymous">L191L</font>&nbsp;(CT<font class="mutation_in_codon">C</font>&rarr;CT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">UDP&#8209;N&#8209;acetylglucosamine&#8209;&#8209;N&#8209;acetylmuramyl&#8209; (pentapeptide) pyrophosphoryl&#8209;undecaprenol N&#8209;acetylglucosamine transferase (EC 2.4.1.227)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_128</td><!-- Seq_Id -->
<td align="right">17,102</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">D3H</font>&nbsp;(<font class="mutation_in_codon">G</font>AT&rarr;<font class="mutation_in_codon">C</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcriptional regulator</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_128</td><!-- Seq_Id -->
<td align="right">59,220</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.4%</td>
<td align="center"><font class="snp_type_synonymous">A387A</font>&nbsp;(GC<font class="mutation_in_codon">C</font>&rarr;GC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Hypoxanthine/guanine permease PbuG</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_128</td><!-- Seq_Id -->
<td align="right">59,253</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_synonymous">Y398Y</font>&nbsp;(TA<font class="mutation_in_codon">T</font>&rarr;TA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Hypoxanthine/guanine permease PbuG</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_128</td><!-- Seq_Id -->
<td align="right">75,317</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">R178L</font>&nbsp;(C<font class="mutation_in_codon">G</font>T&rarr;C<font class="mutation_in_codon">T</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ATPase component of general energizing module of ECF transporters</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_128</td><!-- Seq_Id -->
<td align="right">76,381</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">K264R</font>&nbsp;(A<font class="mutation_in_codon">A</font>A&rarr;A<font class="mutation_in_codon">G</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ATPase component of general energizing module of ECF transporters</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_128</td><!-- Seq_Id -->
<td align="right">83,182</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">G13C</font>&nbsp;(<font class="mutation_in_codon">G</font>GT&rarr;<font class="mutation_in_codon">T</font>GT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_128</td><!-- Seq_Id -->
<td align="right">86,128</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_synonymous">Y129Y</font>&nbsp;(TA<font class="mutation_in_codon">T</font>&rarr;TA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Acetolactate synthase, catabolic (EC 2.2.1.6)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_128</td><!-- Seq_Id -->
<td align="right">96,833</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.8%</td>
<td align="center">intergenic&nbsp;(+284/+7)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/NAD&#8209;dependent protein deacetylase of SIR2 family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_128</td><!-- Seq_Id -->
<td align="right">101,116</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_synonymous">L88L</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Tagatose 1,6&#8209;bisphosphate aldolase (EC 4.1.2.40)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_128</td><!-- Seq_Id -->
<td align="right">101,119</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_synonymous">L89L</font>&nbsp;(CT<font class="mutation_in_codon">T</font>&rarr;CT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Tagatose 1,6&#8209;bisphosphate aldolase (EC 4.1.2.40)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_128</td><!-- Seq_Id -->
<td align="right">101,671</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_synonymous">G273G</font>&nbsp;(GG<font class="mutation_in_codon">G</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Tagatose 1,6&#8209;bisphosphate aldolase (EC 4.1.2.40)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_129</td><!-- Seq_Id -->
<td align="right">144</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">30.0%</td>
<td align="center">intergenic&nbsp;(&#8211;/&#8209;140)</td>
<td align="center"><i>&#8211;</i>&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">&#8211;/Transposase, IS4 family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_129</td><!-- Seq_Id -->
<td align="right">57,351</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.6%</td>
<td align="center"><font class="snp_type_synonymous">C18C</font>&nbsp;(TG<font class="mutation_in_codon">C</font>&rarr;TG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">L&#8209;cystine ABC transporter, substrate&#8209;binding protein TcyA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_129</td><!-- Seq_Id -->
<td align="right">64,940</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_synonymous">N25N</font>&nbsp;(AA<font class="mutation_in_codon">T</font>&rarr;AA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&rarr;</td>
<td align="left">FIG01110006: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_129</td><!-- Seq_Id -->
<td align="right">65,226</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center">intergenic&nbsp;(+235/&#8209;143)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">FIG01110006: hypothetical protein/Cytolytic pore&#8209;forming protein S component => Gamma&#8209;hemolysin HlgC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_129</td><!-- Seq_Id -->
<td align="right">81,174</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_synonymous">P328P</font>&nbsp;(CC<font class="mutation_in_codon">G</font>&rarr;CC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Teicoplanin resistance transporter, TcaB family => TcaB2</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_129</td><!-- Seq_Id -->
<td align="right">128,923</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center">intergenic&nbsp;(&#8209;124/+127)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/Short&#8209;chain alcohol dehydrogenase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">10,432</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">G81V</font>&nbsp;(G<font class="mutation_in_codon">G</font>A&rarr;G<font class="mutation_in_codon">T</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Ribonuclease HII (EC 3.1.26.4)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">19,821</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">9.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">F198L</font>&nbsp;(<font class="mutation_in_codon">T</font>TT&rarr;<font class="mutation_in_codon">C</font>TT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Methylenetetrahydrofolate&#8209;&#8209;tRNA&#8209;(uracil&#8209;5&#8209;)&#8209; methyltransferase TrmFO (EC 2.1.1.74)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">20,177</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_synonymous">N316N</font>&nbsp;(AA<font class="mutation_in_codon">T</font>&rarr;AA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Methylenetetrahydrofolate&#8209;&#8209;tRNA&#8209;(uracil&#8209;5&#8209;)&#8209; methyltransferase TrmFO (EC 2.1.1.74)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">21,016</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.7%</td>
<td align="center"><font class="snp_type_synonymous">H21H</font>&nbsp;(CA<font class="mutation_in_codon">C</font>&rarr;CA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Site&#8209;specific tyrosine recombinase XerC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">32,217</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.6%</td>
<td align="center"><font class="snp_type_synonymous">A160A</font>&nbsp;(GC<font class="mutation_in_codon">A</font>&rarr;GC<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Prolyl&#8209;tRNA synthetase (EC 6.1.1.15), bacterial type</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">32,727</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.6%</td>
<td align="center"><font class="snp_type_synonymous">P330P</font>&nbsp;(CC<font class="mutation_in_codon">T</font>&rarr;CC<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Prolyl&#8209;tRNA synthetase (EC 6.1.1.15), bacterial type</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">36,764</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.0%</td>
<td align="center"><font class="snp_type_synonymous">G1022G</font>&nbsp;(GG<font class="mutation_in_codon">T</font>&rarr;GG<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">DNA polymerase III polC&#8209;type (EC 2.7.7.7)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">45,425</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">D294Y</font>&nbsp;(<font class="mutation_in_codon">G</font>AC&rarr;<font class="mutation_in_codon">T</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">FMN adenylyltransferase (EC 2.7.7.2) / Riboflavin kinase (EC 2.7.1.26)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">53,543</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">S213L</font>&nbsp;(T<font class="mutation_in_codon">C</font>A&rarr;T<font class="mutation_in_codon">T</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transcriptional regulator in cluster with Zn&#8209;dependent hydrolase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">58,791</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_synonymous">G170G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">CDP&#8209;diacylglycerol&#8209;&#8209;glycerol&#8209;3&#8209;phosphate 3&#8209;phosphatidyltransferase (EC 2.7.8.5)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">65,878</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_synonymous">T299T</font>&nbsp;(AC<font class="mutation_in_codon">C</font>&rarr;AC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">2&#8209;oxoglutarate/2&#8209;oxoacid ferredoxin oxidoreductase, gamma subunit (EC 1.2.7.&#8209;) / 2&#8209;oxoglutarate/2&#8209;oxoacid ferredoxin oxidoreductase, alpha subunit (EC 1.2.7.&#8209;)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">67,247</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">G169C</font>&nbsp;(<font class="mutation_in_codon">G</font>GT&rarr;<font class="mutation_in_codon">T</font>GT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">2&#8209;oxoglutarate/2&#8209;oxoacid ferredoxin oxidoreductase, beta subunit (EC 1.2.7.&#8209;)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">69,701</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">N9K</font>&nbsp;(AA<font class="mutation_in_codon">C</font>&rarr;AA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">74,626</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">P379S</font>&nbsp;(<font class="mutation_in_codon">C</font>CA&rarr;<font class="mutation_in_codon">T</font>CA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">DNA mismatch repair protein MutL</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">74,636</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">E382A</font>&nbsp;(G<font class="mutation_in_codon">A</font>A&rarr;G<font class="mutation_in_codon">C</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">DNA mismatch repair protein MutL</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">74,716</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">A409T</font>&nbsp;(<font class="mutation_in_codon">G</font>CT&rarr;<font class="mutation_in_codon">A</font>CT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">DNA mismatch repair protein MutL</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">96,738</td><!-- Position -->
<td align="center">&Delta;3&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">6.3%</td>
<td align="center">coding&nbsp;(875&#8209;877/1026&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Low&#8209;specificity L&#8209;threonine aldolase (EC 4.1.2.48)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">98,146</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">14.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">F3Y</font>&nbsp;(T<font class="mutation_in_codon">T</font>T&rarr;T<font class="mutation_in_codon">A</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Cardiolipin synthase, bacterial type ClsA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">98,159</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">14.3%</td>
<td align="center"><font class="snp_type_synonymous">N7N</font>&nbsp;(AA<font class="mutation_in_codon">C</font>&rarr;AA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Cardiolipin synthase, bacterial type ClsA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">98,171</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_synonymous">T11T</font>&nbsp;(AC<font class="mutation_in_codon">G</font>&rarr;AC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Cardiolipin synthase, bacterial type ClsA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">98,172</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_synonymous">L12L</font>&nbsp;(<font class="mutation_in_codon">T</font>TA&rarr;<font class="mutation_in_codon">C</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Cardiolipin synthase, bacterial type ClsA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">98,180</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_synonymous">T14T</font>&nbsp;(AC<font class="mutation_in_codon">T</font>&rarr;AC<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Cardiolipin synthase, bacterial type ClsA</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">102,648</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">10.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">F52I</font>&nbsp;(<font class="mutation_in_codon">T</font>TT&rarr;<font class="mutation_in_codon">A</font>TT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Two&#8209;component transcriptional response regulator, LuxR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">106,480</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_synonymous">V310V</font>&nbsp;(GT<font class="mutation_in_codon">T</font>&rarr;GT<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Aspartokinase (EC 2.7.2.4)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">106,483</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.4%</td>
<td align="center"><font class="snp_type_synonymous">T309T</font>&nbsp;(AC<font class="mutation_in_codon">C</font>&rarr;AC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Aspartokinase (EC 2.7.2.4)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">111,662</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">N248D</font>&nbsp;(<font class="mutation_in_codon">A</font>AT&rarr;<font class="mutation_in_codon">G</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Hydrolase, HAD superfamily</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">111,679</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center"><font class="snp_type_synonymous">D253D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Hydrolase, HAD superfamily</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">133,826</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">E829G</font>&nbsp;(G<font class="mutation_in_codon">A</font>A&rarr;G<font class="mutation_in_codon">G</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Aconitate hydratase (EC 4.2.1.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">140,101</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_synonymous">G575G</font>&nbsp;(GG<font class="mutation_in_codon">A</font>&rarr;GG<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">DNA topoisomerase IV subunit A (EC 5.99.1.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">144,760</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">9.2%</td>
<td align="center"><font class="snp_type_synonymous">S242S</font>&nbsp;(TC<font class="mutation_in_codon">G</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">UPF0118 membrane protein YubA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">164,223</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center">intergenic&nbsp;(+427/&#8209;262)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">tRNA&#8209;dependent lipid II&#8209;GlyGlyGly glycyltransferase (EC 2.3.2.18) @ tRNA&#8209;dependent lipid II&#8209;GlyGlyGlyGly glycyltransferase (EC 2.3.2.18) @ FemB, factor involved in methicillin resistance/Putative hydrolase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_130</td><!-- Seq_Id -->
<td align="right">166,159</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">R147H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ABC&#8209;type dipeptide/oligopeptide/nickel transport system, ATP&#8209;binding protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_131</td><!-- Seq_Id -->
<td align="right">51</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">16.7%</td>
<td align="center"><font class="snp_type_synonymous">R17R</font>&nbsp;(CG<font class="mutation_in_codon">T</font>&rarr;CG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transposase, IS4 family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_131</td><!-- Seq_Id -->
<td align="right">99</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">54.1%</td>
<td align="center"><font class="snp_type_synonymous">A33A</font>&nbsp;(GC<font class="mutation_in_codon">C</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transposase, IS4 family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_131</td><!-- Seq_Id -->
<td align="right">103</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">37.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">D35N</font>&nbsp;(<font class="mutation_in_codon">G</font>AT&rarr;<font class="mutation_in_codon">A</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transposase, IS4 family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_131</td><!-- Seq_Id -->
<td align="right">136</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">27.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">F46I</font>&nbsp;(<font class="mutation_in_codon">T</font>TT&rarr;<font class="mutation_in_codon">A</font>TT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transposase, IS4 family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_131</td><!-- Seq_Id -->
<td align="right">183</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">32.1%</td>
<td align="center"><font class="snp_type_synonymous">I61I</font>&nbsp;(AT<font class="mutation_in_codon">T</font>&rarr;AT<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transposase, IS4 family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_131</td><!-- Seq_Id -->
<td align="right">189</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">29.9%</td>
<td align="center"><font class="snp_type_synonymous">F63F</font>&nbsp;(TT<font class="mutation_in_codon">C</font>&rarr;TT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transposase, IS4 family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_131</td><!-- Seq_Id -->
<td align="right">224</td><!-- Position -->
<td align="center">&Delta;21&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">29.8%</td>
<td align="center">coding&nbsp;(224&#8209;244/321&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Transposase, IS4 family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_131</td><!-- Seq_Id -->
<td align="right">33,690</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_synonymous">Q349Q</font>&nbsp;(CA<font class="mutation_in_codon">G</font>&rarr;CA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">UDP&#8209;N&#8209;acetylmuramate&#8209;&#8209;L&#8209;alanine ligase (EC 6.3.2.8)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_131</td><!-- Seq_Id -->
<td align="right">36,692</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_synonymous">V13V</font>&nbsp;(GT<font class="mutation_in_codon">A</font>&rarr;GT<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Chorismate mutase I (EC 5.4.99.5) / 2&#8209;keto&#8209;3&#8209;deoxy&#8209;D&#8209;arabino&#8209;heptulosonate&#8209;7&#8209;phosphate synthase I beta (EC 2.5.1.54)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_131</td><!-- Seq_Id -->
<td align="right">36,758</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.2%</td>
<td align="center"><font class="snp_type_synonymous">E35E</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Chorismate mutase I (EC 5.4.99.5) / 2&#8209;keto&#8209;3&#8209;deoxy&#8209;D&#8209;arabino&#8209;heptulosonate&#8209;7&#8209;phosphate synthase I beta (EC 2.5.1.54)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_131</td><!-- Seq_Id -->
<td align="right">38,974</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_synonymous">A230A</font>&nbsp;(GC<font class="mutation_in_codon">C</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Catabolite control protein A</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_131</td><!-- Seq_Id -->
<td align="right">53,174</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center">intergenic&nbsp;(+103/&#8209;25)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Acyl&#8209;ACP:1&#8209;acyl&#8209;sn&#8209;glycerol&#8209;3&#8209;phosphate acyltransferase (EC 2.3.1.n4)/PTS system, N&#8209;acetylglucosamine&#8209;specific IIC component / PTS system, N&#8209;acetylglucosamine&#8209;specific IIB component (EC 2.7.1.193)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_131</td><!-- Seq_Id -->
<td align="right">77,265</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.6%</td>
<td align="center"><font class="snp_type_synonymous">T167T</font>&nbsp;(AC<font class="mutation_in_codon">C</font>&rarr;AC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">3'&#8209;to&#8209;5' oligoribonuclease A, Bacillus type</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">3,584</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_synonymous">E310E</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Hypothetical protein SAV2141</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">5,131</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center">intergenic&nbsp;(&#8209;618/+54)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Hypothetical protein SAV2141/Hypothetical protein SAV2142</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">5,160</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center">intergenic&nbsp;(&#8209;647/+25)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Hypothetical protein SAV2141/Hypothetical protein SAV2142</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">5,162</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.5%</td>
<td align="center">intergenic&nbsp;(&#8209;649/+23)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Hypothetical protein SAV2141/Hypothetical protein SAV2142</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">7,749</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.0%</td>
<td align="center"><font class="snp_type_synonymous">S38S</font>&nbsp;(AG<font class="mutation_in_codon">C</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Zn(II) or Co(II)&#8209;specific transcriptional repressor protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">18,283</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.0%</td>
<td align="center"><font class="snp_type_synonymous">F17F</font>&nbsp;(TT<font class="mutation_in_codon">T</font>&rarr;TT<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">PTS system, mannitol&#8209;specific IIC component / PTS system, mannitol&#8209;specific IIB component (EC 2.7.1.197)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">20,507</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_synonymous">L234L</font>&nbsp;(CT<font class="mutation_in_codon">C</font>&rarr;CT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Mannitol operon activator, BglG family</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">27,851</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center">intergenic&nbsp;(&#8209;161/+86)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">FmtB (Mrp) protein involved in methicillin resistance and cell wall biosynthesis/FmtB (Mrp) protein involved in methicillin resistance and cell wall biosynthesis</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">27,856</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center">intergenic&nbsp;(&#8209;166/+81)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">FmtB (Mrp) protein involved in methicillin resistance and cell wall biosynthesis/FmtB (Mrp) protein involved in methicillin resistance and cell wall biosynthesis</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">27,979</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.0%</td>
<td align="center"><font class="snp_type_synonymous">G1089G</font>&nbsp;(GG<font class="mutation_in_codon">A</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FmtB (Mrp) protein involved in methicillin resistance and cell wall biosynthesis</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">27,980</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">G1089A</font>&nbsp;(G<font class="mutation_in_codon">G</font>A&rarr;G<font class="mutation_in_codon">C</font>A)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FmtB (Mrp) protein involved in methicillin resistance and cell wall biosynthesis</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">28,054</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_synonymous">P1064P</font>&nbsp;(CC<font class="mutation_in_codon">T</font>&rarr;CC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FmtB (Mrp) protein involved in methicillin resistance and cell wall biosynthesis</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">28,057</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_synonymous">T1063T</font>&nbsp;(AC<font class="mutation_in_codon">A</font>&rarr;AC<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FmtB (Mrp) protein involved in methicillin resistance and cell wall biosynthesis</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">28,063</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_synonymous">A1061A</font>&nbsp;(GC<font class="mutation_in_codon">C</font>&rarr;GC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FmtB (Mrp) protein involved in methicillin resistance and cell wall biosynthesis</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">28,066</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_synonymous">A1060A</font>&nbsp;(GC<font class="mutation_in_codon">A</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FmtB (Mrp) protein involved in methicillin resistance and cell wall biosynthesis</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_132</td><!-- Seq_Id -->
<td align="right">28,069</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_synonymous">I1059I</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">FmtB (Mrp) protein involved in methicillin resistance and cell wall biosynthesis</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">26,995</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center">intergenic&nbsp;(+107/&#8209;44)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Cell division trigger factor (EC 5.2.1.8)/ATP&#8209;dependent Clp protease ATP&#8209;binding subunit ClpX</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">36,912</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.6%</td>
<td align="center"><font class="snp_type_synonymous">H185H</font>&nbsp;(CA<font class="mutation_in_codon">T</font>&rarr;CA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">DNA&#8209;3&#8209;methyladenine glycosylase (EC 3.2.2.20)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">47,212</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.6%</td>
<td align="center"><font class="snp_type_synonymous">N75N</font>&nbsp;(AA<font class="mutation_in_codon">C</font>&rarr;AA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">FIG139598: Potential ribosomal protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">49,662</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_synonymous">G134G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">ACT domain&#8209;containing protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">60,307</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_synonymous">R164R</font>&nbsp;(CG<font class="mutation_in_codon">C</font>&rarr;CG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Guanosine&#8209;3',5'&#8209;bis(diphosphate) 3'&#8209;pyrophosphohydrolase (EC 3.1.7.2) / GTP pyrophosphokinase (EC 2.7.6.5), (p)ppGpp synthetase II</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">60,355</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_synonymous">P180P</font>&nbsp;(CC<font class="mutation_in_codon">A</font>&rarr;CC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Guanosine&#8209;3',5'&#8209;bis(diphosphate) 3'&#8209;pyrophosphohydrolase (EC 3.1.7.2) / GTP pyrophosphokinase (EC 2.7.6.5), (p)ppGpp synthetase II</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">61,073</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">P420S</font>&nbsp;(<font class="mutation_in_codon">C</font>CG&rarr;<font class="mutation_in_codon">T</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Guanosine&#8209;3',5'&#8209;bis(diphosphate) 3'&#8209;pyrophosphohydrolase (EC 3.1.7.2) / GTP pyrophosphokinase (EC 2.7.6.5), (p)ppGpp synthetase II</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">62,927</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center"><font class="snp_type_synonymous">D147D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">LytH protein involved in methicillin resistance / N&#8209;acetylmuramoyl&#8209;L&#8209;alanine amidase (EC 3.5.1.28) domain</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">87,001</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_synonymous">G142G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Allophanate hydrolase 2 subunit 1 (EC 3.5.1.54)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">88,016</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_synonymous">Q239Q</font>&nbsp;(CA<font class="mutation_in_codon">G</font>&rarr;CA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Allophanate hydrolase 2 subunit 2 (EC 3.5.1.54)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">95,588</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center">intergenic&nbsp;(&#8209;312/&#8209;2)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/5'&#8209;methylthioadenosine nucleosidase (EC 3.2.2.16) @ S&#8209;adenosylhomocysteine nucleosidase (EC 3.2.2.9)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">130,488</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center">intergenic&nbsp;(+142/&#8209;82)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">DNA primase DnaG/RNA polymerase sigma factor RpoD</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">146,216</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">P111Q</font>&nbsp;(C<font class="mutation_in_codon">C</font>A&rarr;C<font class="mutation_in_codon">A</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Late competence protein ComGA, access of DNA to ComEA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">157,180</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">V91I</font>&nbsp;(<font class="mutation_in_codon">G</font>TA&rarr;<font class="mutation_in_codon">A</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Aminopeptidase YpdF (MP&#8209;, MA&#8209;, MS&#8209;, AP&#8209;, NP&#8209; specific)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">161,845</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">7.6%</td>
<td align="center"><font class="snp_type_synonymous">E52E</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Exodeoxyribonuclease VII large subunit (EC 3.1.11.6)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">165,749</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_synonymous">L247L</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">DNA repair protein RecN</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">171,254</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">A331T</font>&nbsp;(<font class="mutation_in_codon">G</font>CT&rarr;<font class="mutation_in_codon">A</font>CT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Dihydrolipoamide acyltransferase component of branched&#8209;chain alpha&#8209;keto acid dehydrogenase complex (EC 2.3.1.168)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">171,268</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_synonymous">Q335Q</font>&nbsp;(CA<font class="mutation_in_codon">G</font>&rarr;CA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Dihydrolipoamide acyltransferase component of branched&#8209;chain alpha&#8209;keto acid dehydrogenase complex (EC 2.3.1.168)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">171,271</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_synonymous">G336G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Dihydrolipoamide acyltransferase component of branched&#8209;chain alpha&#8209;keto acid dehydrogenase complex (EC 2.3.1.168)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">171,274</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.3%</td>
<td align="center"><font class="snp_type_synonymous">G337G</font>&nbsp;(GG<font class="mutation_in_codon">T</font>&rarr;GG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Dihydrolipoamide acyltransferase component of branched&#8209;chain alpha&#8209;keto acid dehydrogenase complex (EC 2.3.1.168)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">171,283</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_synonymous">T340T</font>&nbsp;(AC<font class="mutation_in_codon">G</font>&rarr;AC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Dihydrolipoamide acyltransferase component of branched&#8209;chain alpha&#8209;keto acid dehydrogenase complex (EC 2.3.1.168)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">171,286</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center"><font class="snp_type_synonymous">V341V</font>&nbsp;(GT<font class="mutation_in_codon">A</font>&rarr;GT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Dihydrolipoamide acyltransferase component of branched&#8209;chain alpha&#8209;keto acid dehydrogenase complex (EC 2.3.1.168)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">181,344</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.9%</td>
<td align="center"><font class="snp_type_synonymous">F428F</font>&nbsp;(TT<font class="mutation_in_codon">T</font>&rarr;TT<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glucose&#8209;6&#8209;phosphate 1&#8209;dehydrogenase (EC 1.1.1.49)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">182,688</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center">intergenic&nbsp;(&#8209;61/&#8209;61)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i></td>
<td align="left">Glucose&#8209;6&#8209;phosphate 1&#8209;dehydrogenase (EC 1.1.1.49)/FIG01109470: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">182,710</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center">intergenic&nbsp;(&#8209;83/&#8209;39)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i></td>
<td align="left">Glucose&#8209;6&#8209;phosphate 1&#8209;dehydrogenase (EC 1.1.1.49)/FIG01109470: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_133</td><!-- Seq_Id -->
<td align="right">192,903</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_nonsense">Q72*</font>&nbsp;(<font class="mutation_in_codon">C</font>AA&rarr;<font class="mutation_in_codon">T</font>AA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Staphylococcal respiratory response protein SrrB</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_134</td><!-- Seq_Id -->
<td align="right">1,174</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.6%</td>
<td align="center">intergenic&nbsp;(&#8209;140/+72)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">oxidoreductase of aldo/keto reductase family, subgroup 1/Hypothetical protein SAV1789</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_134</td><!-- Seq_Id -->
<td align="right">1,179</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center">intergenic&nbsp;(&#8209;145/+67)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">oxidoreductase of aldo/keto reductase family, subgroup 1/Hypothetical protein SAV1789</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_134</td><!-- Seq_Id -->
<td align="right">1,214</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">7.4%</td>
<td align="center">intergenic&nbsp;(&#8209;180/+32)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">oxidoreductase of aldo/keto reductase family, subgroup 1/Hypothetical protein SAV1789</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_134</td><!-- Seq_Id -->
<td align="right">2,940</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">K178R</font>&nbsp;(A<font class="mutation_in_codon">A</font>A&rarr;A<font class="mutation_in_codon">G</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">S&#8209;adenosylmethionine synthetase (EC 2.5.1.6)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_135</td><!-- Seq_Id -->
<td align="right">1,830</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">L162I</font>&nbsp;(<font class="mutation_in_codon">T</font>TA&rarr;<font class="mutation_in_codon">A</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein within pathogenicity island</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_135</td><!-- Seq_Id -->
<td align="right">2,444</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">Y113C</font>&nbsp;(T<font class="mutation_in_codon">A</font>T&rarr;T<font class="mutation_in_codon">G</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">[Genomic island nu Sa beta2]</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_136</td><!-- Seq_Id -->
<td align="right">38,418</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">T247S</font>&nbsp;(A<font class="mutation_in_codon">C</font>T&rarr;A<font class="mutation_in_codon">G</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Two&#8209;component system YycFG regulatory protein YycI</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_136</td><!-- Seq_Id -->
<td align="right">40,726</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">K281T</font>&nbsp;(A<font class="mutation_in_codon">A</font>G&rarr;A<font class="mutation_in_codon">C</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Virulence&#8209;associated cell&#8209;wall&#8209;anchored protein SasH (LPXTG motif); 5'&#8209;nucleotidase (EC 3.1.3.5)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_137</td><!-- Seq_Id -->
<td align="right">5,345</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">S150N</font>&nbsp;(A<font class="mutation_in_codon">G</font>T&rarr;A<font class="mutation_in_codon">A</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Exotoxin 15</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_137</td><!-- Seq_Id -->
<td align="right">7,643</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_synonymous">S197S</font>&nbsp;(TC<font class="mutation_in_codon">T</font>&rarr;TC<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">Exotoxin 8</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_137</td><!-- Seq_Id -->
<td align="right">7,657</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">L193V</font>&nbsp;(<font class="mutation_in_codon">C</font>TG&rarr;<font class="mutation_in_codon">G</font>TG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">Exotoxin 8</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_137</td><!-- Seq_Id -->
<td align="right">7,660</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">N192Y</font>&nbsp;(<font class="mutation_in_codon">A</font>AT&rarr;<font class="mutation_in_codon">T</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">Exotoxin 8</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_137</td><!-- Seq_Id -->
<td align="right">7,663</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">Y191N</font>&nbsp;(<font class="mutation_in_codon">T</font>AC&rarr;<font class="mutation_in_codon">A</font>AC)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">Exotoxin 8</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_137</td><!-- Seq_Id -->
<td align="right">22,580</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center">intergenic&nbsp;(+87/&#8209;56)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_137</td><!-- Seq_Id -->
<td align="right">28,407</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_nonsense">E115*</font>&nbsp;(<font class="mutation_in_codon">G</font>AA&rarr;<font class="mutation_in_codon">T</font>AA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Alkyl hydroperoxide reductase protein C (EC 1.11.1.15)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_137</td><!-- Seq_Id -->
<td align="right">79,492</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">H19R</font>&nbsp;(C<font class="mutation_in_codon">A</font>T&rarr;C<font class="mutation_in_codon">G</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Macro domain protein YmdB, possibly ADP&#8209;ribose binding</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_137</td><!-- Seq_Id -->
<td align="right">82,191</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">M325I</font>&nbsp;(AT<font class="mutation_in_codon">G</font>&rarr;AT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Lipoylation&#8209;related NADH&#8209;dependent flavin oxidoreductase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_137</td><!-- Seq_Id -->
<td align="right">87,444</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.6%</td>
<td align="center">intergenic&nbsp;(+83/+44)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Predicted histidine uptake transporter/N&#8209;acetylmannosamine&#8209;6&#8209;phosphate 2&#8209;epimerase (EC 5.1.3.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_137</td><!-- Seq_Id -->
<td align="right">88,469</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.0%</td>
<td align="center"><font class="snp_type_synonymous">L58L</font>&nbsp;(CT<font class="mutation_in_codon">G</font>&rarr;CT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Sialic acid utilization regulator, RpiR family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_138</td><!-- Seq_Id -->
<td align="right">486</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.4%</td>
<td align="center">intergenic&nbsp;(&#8209;152/+495)</td>
<td align="center"><i>rasttk_feature_creation_tool=tRNAscan&#8209;SE</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">tRNA&#8209;Leu&#8209;GAG/Putative metallopeptidase (Zinc) SprT family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_138</td><!-- Seq_Id -->
<td align="right">16,420</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center">intergenic&nbsp;(&#8209;151/&#8209;121)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Potassium&#8209;transporting ATPase A chain (EC 3.6.3.12) (TC 3.A.3.7.1)/Osmosensitive K+ channel histidine kinase KdpD</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_138</td><!-- Seq_Id -->
<td align="right">30,230</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center">intergenic&nbsp;(+516/+197)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Metal&#8209;dependent phosphohydrolase/Inner membrane protein translocase and chaperone YidC, short form OxaI&#8209;like</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_138</td><!-- Seq_Id -->
<td align="right">40,224</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.0%</td>
<td align="center"><font class="snp_type_synonymous">L377L</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ATP synthase beta chain (EC 3.6.3.14)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_138</td><!-- Seq_Id -->
<td align="right">60,215</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.9%</td>
<td align="center">intergenic&nbsp;(&#8209;251/+215)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">UDP&#8209;N&#8209;acetylglucosamine 1&#8209;carboxyvinyltransferase (EC 2.5.1.7)/Fructose&#8209;bisphosphate aldolase class II (EC 4.1.2.13)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_138</td><!-- Seq_Id -->
<td align="right">60,219</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.8%</td>
<td align="center">intergenic&nbsp;(&#8209;255/+211)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">UDP&#8209;N&#8209;acetylglucosamine 1&#8209;carboxyvinyltransferase (EC 2.5.1.7)/Fructose&#8209;bisphosphate aldolase class II (EC 4.1.2.13)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_138</td><!-- Seq_Id -->
<td align="right">60,220</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.7%</td>
<td align="center">intergenic&nbsp;(&#8209;256/+210)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">UDP&#8209;N&#8209;acetylglucosamine 1&#8209;carboxyvinyltransferase (EC 2.5.1.7)/Fructose&#8209;bisphosphate aldolase class II (EC 4.1.2.13)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_138</td><!-- Seq_Id -->
<td align="right">66,123</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">W38C</font>&nbsp;(TG<font class="mutation_in_codon">G</font>&rarr;TG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Pantothenate kinase type II, eukaryotic (EC 2.7.1.33)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_138</td><!-- Seq_Id -->
<td align="right">70,644</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center">intergenic&nbsp;(&#8209;254/&#8209;57)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">HmrA protein involved in methicillin resistance / amidohydrolase of M40 family/S&#8209;ribosylhomocysteine lyase (EC 4.4.1.21) @ Autoinducer&#8209;2 production protein LuxS</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_138</td><!-- Seq_Id -->
<td align="right">70,682</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center">intergenic&nbsp;(&#8209;292/&#8209;19)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">HmrA protein involved in methicillin resistance / amidohydrolase of M40 family/S&#8209;ribosylhomocysteine lyase (EC 4.4.1.21) @ Autoinducer&#8209;2 production protein LuxS</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_138</td><!-- Seq_Id -->
<td align="right">70,745</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_synonymous">K15K</font>&nbsp;(AA<font class="mutation_in_codon">G</font>&rarr;AA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">S&#8209;ribosylhomocysteine lyase (EC 4.4.1.21) @ Autoinducer&#8209;2 production protein LuxS</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">4,959</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">7.6%</td>
<td align="center"><font class="snp_type_synonymous">L290L</font>&nbsp;(<font class="mutation_in_codon">T</font>TA&rarr;<font class="mutation_in_codon">C</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">HoxN/HupN/NixA family nickel/cobalt transporter</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">21,677</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.8%</td>
<td align="center">intergenic&nbsp;(+247/&#8209;58)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/ATP phosphoribosyltransferase regulatory subunit (EC 2.4.2.17)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">29,426</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.6%</td>
<td align="center">intergenic&nbsp;(+394/&#8209;324)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">FIG01108155: hypothetical protein/Triacylglycerol lipase (EC 3.1.1.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">29,430</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center">intergenic&nbsp;(+398/&#8209;320)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">FIG01108155: hypothetical protein/Triacylglycerol lipase (EC 3.1.1.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">29,457</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center">intergenic&nbsp;(+425/&#8209;293)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">FIG01108155: hypothetical protein/Triacylglycerol lipase (EC 3.1.1.3)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">86,913</td><!-- Position -->
<td align="center">+42 bp</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center">coding&nbsp;(1685/2700&nbsp;nt)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">87,127</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">D633E</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Clumping factor ClfB, fibrinogen binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">112,794</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_synonymous">S511S</font>&nbsp;(TC<font class="mutation_in_codon">A</font>&rarr;TC<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Choline dehydrogenase (EC 1.1.99.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">132,237</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.2%</td>
<td align="center"><font class="snp_type_synonymous">E75E</font>&nbsp;(GA<font class="mutation_in_codon">G</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Epoxyqueuosine reductase (EC 1.17.99.6) QueH</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">132,240</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center"><font class="snp_type_synonymous">Y76Y</font>&nbsp;(TA<font class="mutation_in_codon">C</font>&rarr;TA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Epoxyqueuosine reductase (EC 1.17.99.6) QueH</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">138,789</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_synonymous">I192I</font>&nbsp;(AT<font class="mutation_in_codon">A</font>&rarr;AT<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Similar to ferrous iron transporter protein B</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">147,669</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">S12L</font>&nbsp;(T<font class="mutation_in_codon">C</font>A&rarr;T<font class="mutation_in_codon">T</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">FIG01108446: hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">179,522</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center">intergenic&nbsp;(&#8209;183/&#8209;71)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Hydroxymethylglutaryl&#8209;CoA synthase (EC 2.3.3.10)/Hydroxymethylglutaryl&#8209;CoA reductase (EC 1.1.1.34)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">179,655</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_synonymous">V21V</font>&nbsp;(GT<font class="mutation_in_codon">T</font>&rarr;GT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Hydroxymethylglutaryl&#8209;CoA reductase (EC 1.1.1.34)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">197,823</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.0%</td>
<td align="center"><font class="snp_type_synonymous">E106E</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Acetyltransferase, GNAT family</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">197,826</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_synonymous">V105V</font>&nbsp;(GT<font class="mutation_in_codon">G</font>&rarr;GT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Acetyltransferase, GNAT family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">197,836</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">H102R</font>&nbsp;(C<font class="mutation_in_codon">A</font>T&rarr;C<font class="mutation_in_codon">G</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Acetyltransferase, GNAT family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">197,838</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">H101Q</font>&nbsp;(CA<font class="mutation_in_codon">C</font>&rarr;CA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Acetyltransferase, GNAT family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">197,843</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_synonymous">L100L</font>&nbsp;(<font class="mutation_in_codon">C</font>TA&rarr;<font class="mutation_in_codon">T</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Acetyltransferase, GNAT family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">203,103</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center">intergenic&nbsp;(&#8209;255/&#8209;15)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">D&#8209;lactate dehydrogenase (EC 1.1.1.28)/Oxygen&#8209;insensitive NAD(P)H nitroreductase (EC 1.&#8209;.&#8209;.&#8209;) / Dihydropteridine reductase (EC 1.5.1.34)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">211,897</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_synonymous">Y77Y</font>&nbsp;(TA<font class="mutation_in_codon">C</font>&rarr;TA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Putative membrane&#8209;spanning protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_139</td><!-- Seq_Id -->
<td align="right">216,258</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_synonymous">T44T</font>&nbsp;(AC<font class="mutation_in_codon">A</font>&rarr;AC<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Uncharacterized MFS&#8209;type transporter</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_140</td><!-- Seq_Id -->
<td align="right">14,731</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">D271E</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glycosyl transferase family protein, putative</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_140</td><!-- Seq_Id -->
<td align="right">14,737</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center"><font class="snp_type_synonymous">L269L</font>&nbsp;(CT<font class="mutation_in_codon">A</font>&rarr;CT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glycosyl transferase family protein, putative</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_140</td><!-- Seq_Id -->
<td align="right">50,012</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center"><font class="snp_type_synonymous">E391E</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">3&#8209;ketoacyl&#8209;CoA thiolase [fadN&#8209;fadA&#8209;fadE operon] (EC 2.3.1.16)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_140</td><!-- Seq_Id -->
<td align="right">98,496</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">I233V</font>&nbsp;(<font class="mutation_in_codon">A</font>TA&rarr;<font class="mutation_in_codon">G</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Pyruvate decarboxylase (EC 4.1.1.1); Alpha&#8209;keto&#8209;acid decarboxylase (EC 4.1.1.&#8209;)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_140</td><!-- Seq_Id -->
<td align="right">105,251</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">N346K</font>&nbsp;(AA<font class="mutation_in_codon">C</font>&rarr;AA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Glutamate N&#8209;acetyltransferase (EC 2.3.1.35) @ N&#8209;acetylglutamate synthase (EC 2.3.1.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_140</td><!-- Seq_Id -->
<td align="right">105,287</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_synonymous">V358V</font>&nbsp;(GT<font class="mutation_in_codon">T</font>&rarr;GT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Glutamate N&#8209;acetyltransferase (EC 2.3.1.35) @ N&#8209;acetylglutamate synthase (EC 2.3.1.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_140</td><!-- Seq_Id -->
<td align="right">105,332</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_synonymous">N373N</font>&nbsp;(AA<font class="mutation_in_codon">T</font>&rarr;AA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Glutamate N&#8209;acetyltransferase (EC 2.3.1.35) @ N&#8209;acetylglutamate synthase (EC 2.3.1.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_140</td><!-- Seq_Id -->
<td align="right">127,243</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center">intergenic&nbsp;(&#8209;322/&#8209;105)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Aldehyde dehydrogenase B (EC 1.2.1.22)/Inner membrane protein YbaN</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_140</td><!-- Seq_Id -->
<td align="right">136,559</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">Y86F</font>&nbsp;(T<font class="mutation_in_codon">A</font>T&rarr;T<font class="mutation_in_codon">T</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_si milarity</i>&nbsp;&larr;</td>
<td align="left">Capsular polysaccharide synthesis enzyme Cap5I</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">10,304</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_synonymous">F93F</font>&nbsp;(TT<font class="mutation_in_codon">T</font>&rarr;TT<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Cysteine desulfurase (EC 2.8.1.7) => SufS</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">11,030</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.5%</td>
<td align="center"><font class="snp_type_synonymous">S325S</font>&nbsp;(TC<font class="mutation_in_codon">G</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Iron&#8209;sulfur cluster assembly protein SufD</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">19,306</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center"><font class="snp_type_synonymous">L205L</font>&nbsp;(TT<font class="mutation_in_codon">G</font>&rarr;TT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">22,695</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">S84L</font>&nbsp;(T<font class="mutation_in_codon">C</font>G&rarr;T<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">3&#8209;dehydroquinate dehydratase I (EC 4.2.1.10)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">48,545</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_synonymous">D496D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">2,3&#8209;bisphosphoglycerate&#8209;independent phosphoglycerate mutase (EC 5.4.2.12)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">70,107</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_synonymous">A226A</font>&nbsp;(GC<font class="mutation_in_codon">C</font>&rarr;GC<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Excinuclease ABC subunit A</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">74,181</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_synonymous">S254S</font>&nbsp;(TC<font class="mutation_in_codon">T</font>&rarr;TC<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">LysM domain protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">76,362</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center">intergenic&nbsp;(&#8209;19/+414)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Peptide chain release factor 2 @ programmed frameshift&#8209;containing/Protein translocase subunit SecA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">111,683</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_synonymous">G272G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glycine betaine ABC transport system, permease protein OpuAB / Glycine betaine ABC transport system, glycine betaine&#8209;binding protein OpuAC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">122,532</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">L62F</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Aminodeoxychorismate lyase (EC 4.1.3.38)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">123,149</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.3%</td>
<td align="center"><font class="snp_type_synonymous">L240L</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Para&#8209;aminobenzoate synthase, aminase component (EC 2.6.1.85)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">142,815</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">S101I</font>&nbsp;(A<font class="mutation_in_codon">G</font>C&rarr;A<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">150,467</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.6%</td>
<td align="center">intergenic&nbsp;(+68/+59)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Transcriptional regulator MgrA (Regulator of autolytic activity)/Efflux ABC transporter for glutathione/L&#8209;cysteine, essential for assembly of bd&#8209;type respiratory oxidases => CydC subunit</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">153,488</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_synonymous">L114L</font>&nbsp;(<font class="mutation_in_codon">C</font>TA&rarr;<font class="mutation_in_codon">T</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Efflux ABC transporter for glutathione/L&#8209;cysteine, essential for assembly of bd&#8209;type respiratory oxidases => CydD subunit</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">157,111</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_synonymous">K109K</font>&nbsp;(AA<font class="mutation_in_codon">A</font>&rarr;AA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Acetyltransferase, GNAT family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">169,336</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.6%</td>
<td align="center">intergenic&nbsp;(&#8209;234/&#8209;127)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/Secretory antigen precursor SsaA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">179,229</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_synonymous">Y158Y</font>&nbsp;(TA<font class="mutation_in_codon">T</font>&rarr;TA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Acetyltransferase, GNAT family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">179,409</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center">intergenic&nbsp;(+147/+62)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Acetyltransferase, GNAT family/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">179,431</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center">intergenic&nbsp;(+169/+40)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Acetyltransferase, GNAT family/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">179,452</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center">intergenic&nbsp;(+190/+19)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Acetyltransferase, GNAT family/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">198,845</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q23R</font>&nbsp;(C<font class="mutation_in_codon">A</font>G&rarr;C<font class="mutation_in_codon">G</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Teichoic acid translocation permease protein TagG</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">239,930</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.8%</td>
<td align="center"><font class="snp_type_synonymous">V326V</font>&nbsp;(GT<font class="mutation_in_codon">A</font>&rarr;GT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Arginyl&#8209;tRNA synthetase (EC 6.1.1.19)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">259,074</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_synonymous">G192G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Putative Dihydrolipoamide dehydrogenase (EC 1.8.1.4); Mercuric ion reductase (EC 1.16.1.1); PF00070 family, FAD&#8209;dependent NAD(P)&#8209;disulphide oxidoreductase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">260,532</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center"><font class="snp_type_synonymous">G312G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phosphomevalonate kinase (EC 2.7.4.2)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">260,942</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.5%</td>
<td align="center"><font class="snp_type_synonymous">L176L</font>&nbsp;(<font class="mutation_in_codon">C</font>TA&rarr;<font class="mutation_in_codon">T</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phosphomevalonate kinase (EC 2.7.4.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">267,763</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_synonymous">R165R</font>&nbsp;(CG<font class="mutation_in_codon">T</font>&rarr;CG<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">267,772</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.3%</td>
<td align="center"><font class="snp_type_synonymous">H168H</font>&nbsp;(CA<font class="mutation_in_codon">C</font>&rarr;CA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">267,844</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.6%</td>
<td align="center"><font class="snp_type_synonymous">I192I</font>&nbsp;(AT<font class="mutation_in_codon">T</font>&rarr;AT<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,665</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">D175E</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,689</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_synonymous">G167G</font>&nbsp;(GG<font class="mutation_in_codon">A</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,695</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_synonymous">A165A</font>&nbsp;(GC<font class="mutation_in_codon">A</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,698</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_synonymous">V164V</font>&nbsp;(GT<font class="mutation_in_codon">A</font>&rarr;GT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,704</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.3%</td>
<td align="center"><font class="snp_type_synonymous">V162V</font>&nbsp;(GT<font class="mutation_in_codon">T</font>&rarr;GT<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,722</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.5%</td>
<td align="center"><font class="snp_type_synonymous">V156V</font>&nbsp;(GT<font class="mutation_in_codon">T</font>&rarr;GT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,725</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_synonymous">S155S</font>&nbsp;(TC<font class="mutation_in_codon">T</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,734</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.2%</td>
<td align="center"><font class="snp_type_synonymous">T152T</font>&nbsp;(AC<font class="mutation_in_codon">C</font>&rarr;AC<font class="mutation_in_codon">A</font>)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,735</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">T152N</font>&nbsp;(A<font class="mutation_in_codon">C</font>C&rarr;A<font class="mutation_in_codon">A</font>C)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,737</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">R151S</font>&nbsp;(AG<font class="mutation_in_codon">A</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,739</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center"><font class="snp_type_synonymous">R151R</font>&nbsp;(<font class="mutation_in_codon">A</font>GA&rarr;<font class="mutation_in_codon">C</font>GA)&nbsp;&Dagger;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,743</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">7.1%</td>
<td align="center"><font class="snp_type_synonymous">S149S</font>&nbsp;(AG<font class="mutation_in_codon">T</font>&rarr;AG<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,746</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">E148D</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,755</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.5%</td>
<td align="center"><font class="snp_type_synonymous">S145S</font>&nbsp;(TC<font class="mutation_in_codon">A</font>&rarr;TC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,761</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.3%</td>
<td align="center"><font class="snp_type_synonymous">G143G</font>&nbsp;(GG<font class="mutation_in_codon">G</font>&rarr;GG<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,767</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center"><font class="snp_type_synonymous">A141A</font>&nbsp;(GC<font class="mutation_in_codon">A</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">280,791</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.8%</td>
<td align="center"><font class="snp_type_synonymous">V133V</font>&nbsp;(GT<font class="mutation_in_codon">A</font>&rarr;GT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">D&#8209;arabino&#8209;3&#8209;hexulose 6&#8209;phosphate formaldehyde&#8209;lyase (EC 4.1.2.43)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">297,955</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_synonymous">T314T</font>&nbsp;(AC<font class="mutation_in_codon">A</font>&rarr;AC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Adhesin of unknown specificity SdrC</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">304,352</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_synonymous">E171E</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Branched&#8209;chain amino acid aminotransferase (EC 2.6.1.42)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">305,367</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.8%</td>
<td align="center"><font class="snp_type_synonymous">Y266Y</font>&nbsp;(TA<font class="mutation_in_codon">C</font>&rarr;TA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">L&#8209;threonine 3&#8209;dehydrogenase (EC 1.1.1.103)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_141</td><!-- Seq_Id -->
<td align="right">305,385</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.6%</td>
<td align="center"><font class="snp_type_synonymous">P260P</font>&nbsp;(CC<font class="mutation_in_codon">C</font>&rarr;CC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">L&#8209;threonine 3&#8209;dehydrogenase (EC 1.1.1.103)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">15,024</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">F198L</font>&nbsp;(TT<font class="mutation_in_codon">C</font>&rarr;TT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Malonyl CoA&#8209;acyl carrier protein transacylase (EC 2.3.1.39)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">38,630</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center">intergenic&nbsp;(&#8209;215/&#8209;60)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Guanylate kinase (EC 2.7.4.8)/Fibronectin/fibrinogen&#8209;binding protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">43,587</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.0%</td>
<td align="center"><font class="snp_type_synonymous">V910V</font>&nbsp;(GT<font class="mutation_in_codon">G</font>&rarr;GT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Carbamoyl&#8209;phosphate synthase large chain (EC 6.3.5.5)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">53,933</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">17.6%</td>
<td align="center">intergenic&nbsp;(&#8209;462/&#8209;107)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Lipoprotein signal peptidase (EC 3.4.23.36)/Transposase, IS4 family</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">62,354</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center">intergenic&nbsp;(&#8209;123/+137)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">FIG00003370: Multicopper polyphenol oxidase/Cell division protein FtsZ</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">103,938</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_synonymous">K582K</font>&nbsp;(AA<font class="mutation_in_codon">G</font>&rarr;AA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Recombination inhibitory protein MutS2</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">104,784</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">D300E</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Recombination inhibitory protein MutS2</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">108,819</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">E62K</font>&nbsp;(<font class="mutation_in_codon">G</font>AA&rarr;<font class="mutation_in_codon">A</font>AA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Ribonuclease HIII (EC 3.1.26.4)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">110,567</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.8%</td>
<td align="center"><font class="snp_type_synonymous">E527E</font>&nbsp;(GA<font class="mutation_in_codon">A</font>&rarr;GA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phenylalanyl&#8209;tRNA synthetase beta chain (EC 6.1.1.20)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">128,722</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">V297I</font>&nbsp;(<font class="mutation_in_codon">G</font>TT&rarr;<font class="mutation_in_codon">A</font>TT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Allergen V5/Tpx&#8209;1 related</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">139,707</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.9%</td>
<td align="center"><font class="snp_type_synonymous">Q573Q</font>&nbsp;(CA<font class="mutation_in_codon">A</font>&rarr;CA<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">GTP&#8209;binding protein TypA/BipA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">140,883</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.0%</td>
<td align="center"><font class="snp_type_synonymous">K181K</font>&nbsp;(AA<font class="mutation_in_codon">G</font>&rarr;AA<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">GTP&#8209;binding protein TypA/BipA</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">145,071</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="right">7.9%</td>
<td align="center">intergenic&nbsp;(+65/+17)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Manganese transport protein MntH/hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">148,371</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">W132L</font>&nbsp;(T<font class="mutation_in_codon">G</font>G&rarr;T<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Spermidine/putrescine import ABC transporter permease protein PotC (TC 3.A.1.11.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">153,465</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_synonymous">A344A</font>&nbsp;(GC<font class="mutation_in_codon">T</font>&rarr;GC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">165,717</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">G327D</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">A</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phosphoenolpyruvate&#8209;protein phosphotransferase of PTS system (EC 2.7.3.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">172,367</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">L101V</font>&nbsp;(<font class="mutation_in_codon">T</font>TA&rarr;<font class="mutation_in_codon">G</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Duplicated ATPase component YkoD of energizing module of thiamin&#8209;regulated ECF transporter for HydroxyMethylPyrimidine</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">175,436</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">N115Y</font>&nbsp;(<font class="mutation_in_codon">A</font>AT&rarr;<font class="mutation_in_codon">T</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phosphoribosylamine&#8209;&#8209;glycine ligase (EC 6.3.4.13)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">181,261</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_synonymous">Y425Y</font>&nbsp;(TA<font class="mutation_in_codon">C</font>&rarr;TA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phosphoribosylformylglycinamidine synthase, synthetase subunit (EC 6.3.5.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">181,317</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">F407L</font>&nbsp;(<font class="mutation_in_codon">T</font>TT&rarr;<font class="mutation_in_codon">C</font>TT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Phosphoribosylformylglycinamidine synthase, synthetase subunit (EC 6.3.5.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">186,438</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">G158A</font>&nbsp;(G<font class="mutation_in_codon">G</font>T&rarr;G<font class="mutation_in_codon">C</font>T)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Methenyltetrahydrofolate cyclohydrolase (EC 3.5.4.9) / Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">192,546:1</td><!-- Position -->
<td align="center">+A</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center">intergenic&nbsp;(+56/+480)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Cytochrome aa3&#8209;600 menaquinol oxidase subunit IV/FmtA protein involved in methicillin resistance</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">196,438</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center">intergenic&nbsp;(+33/&#8209;122)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/GNAT family acetyltransferase YjcF</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">196,441</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center">intergenic&nbsp;(+36/&#8209;119)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;&nbsp;/&nbsp;&rarr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">hypothetical protein/GNAT family acetyltransferase YjcF</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">199,796</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center"><font class="snp_type_synonymous">L859L</font>&nbsp;(<font class="mutation_in_codon">T</font>TA&rarr;<font class="mutation_in_codon">C</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">N&#8209;acetylmuramoyl&#8209;L&#8209;alanine amidase (EC 3.5.1.28) / Endo&#8209;beta&#8209;N&#8209;acetylglucosaminidase (EC 3.2.1.96) @ Bifunctional autolysin Atl</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">203,235</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">D87Y</font>&nbsp;(<font class="mutation_in_codon">G</font>AT&rarr;<font class="mutation_in_codon">T</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">N&#8209;acetyl&#8209;L,L&#8209;diaminopimelate aminotransferase (EC 2.6.1.&#8209;)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">203,643</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q223K</font>&nbsp;(<font class="mutation_in_codon">C</font>AA&rarr;<font class="mutation_in_codon">A</font>AA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">N&#8209;acetyl&#8209;L,L&#8209;diaminopimelate aminotransferase (EC 2.6.1.&#8209;)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">203,714</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">7.9%</td>
<td align="center"><font class="snp_type_synonymous">G246G</font>&nbsp;(GG<font class="mutation_in_codon">A</font>&rarr;GG<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">N&#8209;acetyl&#8209;L,L&#8209;diaminopimelate aminotransferase (EC 2.6.1.&#8209;)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">206,755</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.5%</td>
<td align="center"><font class="snp_type_synonymous">D335D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Staphopain B precursor (EC 3.4.22.&#8209;)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">206,773</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.8%</td>
<td align="center"><font class="snp_type_synonymous">A341A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Staphopain B precursor (EC 3.4.22.&#8209;)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">209,090</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center"><font class="snp_type_synonymous">G22G</font>&nbsp;(GG<font class="mutation_in_codon">A</font>&rarr;GG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">2&#8209;succinyl&#8209;6&#8209;hydroxy&#8209;2,4&#8209;cyclohexadiene&#8209;1&#8209; carboxylate synthase (EC 4.2.99.20)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">209,099</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">F19L</font>&nbsp;(TT<font class="mutation_in_codon">T</font>&rarr;TT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">2&#8209;succinyl&#8209;6&#8209;hydroxy&#8209;2,4&#8209;cyclohexadiene&#8209;1&#8209; carboxylate synthase (EC 4.2.99.20)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">211,631</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">G178E</font>&nbsp;(G<font class="mutation_in_codon">G</font>A&rarr;G<font class="mutation_in_codon">A</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Isochorismate synthase (EC 5.4.4.2) @ Menaquinone&#8209;specific isochorismate synthase (EC 5.4.4.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">212,096</td><!-- Position -->
<td align="center">G&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_nonsense">S23*</font>&nbsp;(T<font class="mutation_in_codon">C</font>A&rarr;T<font class="mutation_in_codon">A</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Isochorismate synthase (EC 5.4.4.2) @ Menaquinone&#8209;specific isochorismate synthase (EC 5.4.4.2)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">223,018</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center">intergenic&nbsp;(&#8209;1274/+70)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Putative membrane&#8209;spanning protein/hypothetical protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">238,031</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_synonymous">T336T</font>&nbsp;(AC<font class="mutation_in_codon">T</font>&rarr;AC<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Processive diacylglycerol beta&#8209;glucosyltransferase (EC 2.4.1.315)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">244,576</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.9%</td>
<td align="center"><font class="snp_type_synonymous">N156N</font>&nbsp;(AA<font class="mutation_in_codon">C</font>&rarr;AA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Enoyl&#8209;[acyl&#8209;carrier&#8209;protein] reductase [NADH] (EC 1.3.1.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">247,973</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_synonymous">S196S</font>&nbsp;(TC<font class="mutation_in_codon">G</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Mg/Co/Ni transporter MgtE, CBS domain&#8209;containing</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">248,042</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center"><font class="snp_type_synonymous">A173A</font>&nbsp;(GC<font class="mutation_in_codon">A</font>&rarr;GC<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Mg/Co/Ni transporter MgtE, CBS domain&#8209;containing</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">248,890</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">R182S</font>&nbsp;(AG<font class="mutation_in_codon">G</font>&rarr;AG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Similar to ribosomal large subunit pseudouridine synthase D, Bacillus subtilis YjbO type</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">248,944</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_synonymous">G164G</font>&nbsp;(GG<font class="mutation_in_codon">T</font>&rarr;GG<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Similar to ribosomal large subunit pseudouridine synthase D, Bacillus subtilis YjbO type</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">251,878</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">7.7%</td>
<td align="center"><font class="snp_type_nonsynonymous">T146A</font>&nbsp;(<font class="mutation_in_codon">A</font>CG&rarr;<font class="mutation_in_codon">G</font>CG)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Adenylate cyclase (EC 4.6.1.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">263,891</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center"><font class="snp_type_synonymous">S420S</font>&nbsp;(TC<font class="mutation_in_codon">T</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Oligopeptide ABC transporter, substrate&#8209;binding protein OppA (TC 3.A.1.5.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">287,033</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">8.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">F245L</font>&nbsp;(TT<font class="mutation_in_codon">T</font>&rarr;TT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">CoA&#8209;disulfide reductase (EC 1.8.1.14)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">290,266</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">10.1%</td>
<td align="center"><font class="snp_type_nonsynonymous">R987Q</font>&nbsp;(C<font class="mutation_in_codon">G</font>A&rarr;C<font class="mutation_in_codon">A</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ATP&#8209;dependent helicase/nuclease AddAB, subunit A</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">290,354</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">8.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">D958N</font>&nbsp;(<font class="mutation_in_codon">G</font>AT&rarr;<font class="mutation_in_codon">A</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">ATP&#8209;dependent helicase/nuclease AddAB, subunit A</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">299,293</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_synonymous">S307S</font>&nbsp;(TC<font class="mutation_in_codon">T</font>&rarr;TC<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glucose&#8209;6&#8209;phosphate isomerase (EC 5.3.1.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">299,308</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.4%</td>
<td align="center"><font class="snp_type_synonymous">T302T</font>&nbsp;(AC<font class="mutation_in_codon">A</font>&rarr;AC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glucose&#8209;6&#8209;phosphate isomerase (EC 5.3.1.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">299,317</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.6%</td>
<td align="center"><font class="snp_type_synonymous">A299A</font>&nbsp;(GC<font class="mutation_in_codon">C</font>&rarr;GC<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glucose&#8209;6&#8209;phosphate isomerase (EC 5.3.1.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">299,332</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.6%</td>
<td align="center"><font class="snp_type_synonymous">I294I</font>&nbsp;(AT<font class="mutation_in_codon">C</font>&rarr;AT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glucose&#8209;6&#8209;phosphate isomerase (EC 5.3.1.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">299,341</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.6%</td>
<td align="center"><font class="snp_type_synonymous">F291F</font>&nbsp;(TT<font class="mutation_in_codon">C</font>&rarr;TT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glucose&#8209;6&#8209;phosphate isomerase (EC 5.3.1.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">299,419</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.2%</td>
<td align="center"><font class="snp_type_nonsynonymous">L265F</font>&nbsp;(TT<font class="mutation_in_codon">G</font>&rarr;TT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Glucose&#8209;6&#8209;phosphate isomerase (EC 5.3.1.9)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">306,426</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center"><font class="snp_type_synonymous">G310G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;</td>
<td align="left">Ornithine aminotransferase (EC 2.6.1.13)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">312,579</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">F488L</font>&nbsp;(TT<font class="mutation_in_codon">T</font>&rarr;TT<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Na(+) H(+) antiporter subunit A</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">320,248</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.8%</td>
<td align="center">intergenic&nbsp;(&#8209;200/+211)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Histidine permease YuiF/Cytosol aminopeptidase PepA (EC 3.4.11.1)</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">320,253</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center">intergenic&nbsp;(&#8209;205/+206)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Histidine permease YuiF/Cytosol aminopeptidase PepA (EC 3.4.11.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">320,254</td><!-- Position -->
<td align="center">A&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center">intergenic&nbsp;(&#8209;206/+205)</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&larr;&nbsp;/&nbsp;&larr;&nbsp;<i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i></td>
<td align="left">Histidine permease YuiF/Cytosol aminopeptidase PepA (EC 3.4.11.1)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_142</td><!-- Seq_Id -->
<td align="right">324,789</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">5.1%</td>
<td align="center"><font class="snp_type_synonymous">F46F</font>&nbsp;(TT<font class="mutation_in_codon">T</font>&rarr;TT<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Type 2 NADH dehydrogenase (EC 1.6.99.3)</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_78</td><!-- Seq_Id -->
<td align="right">56</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">34.3%</td>
<td align="center"><font class="snp_type_nonsynonymous">R19Q</font>&nbsp;(C<font class="mutation_in_codon">G</font>A&rarr;C<font class="mutation_in_codon">A</font>A)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Repetitive hypothetical protein near ESAT cluster, SA0282 homolog</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_78</td><!-- Seq_Id -->
<td align="right">58</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">32.5%</td>
<td align="center"><font class="snp_type_nonsynonymous">Q20E</font>&nbsp;(<font class="mutation_in_codon">C</font>AA&rarr;<font class="mutation_in_codon">G</font>AA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Repetitive hypothetical protein near ESAT cluster, SA0282 homolog</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_78</td><!-- Seq_Id -->
<td align="right">63</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">31.8%</td>
<td align="center"><font class="snp_type_synonymous">N21N</font>&nbsp;(AA<font class="mutation_in_codon">T</font>&rarr;AA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Repetitive hypothetical protein near ESAT cluster, SA0282 homolog</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_78</td><!-- Seq_Id -->
<td align="right">72</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="right">28.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">K24N</font>&nbsp;(AA<font class="mutation_in_codon">G</font>&rarr;AA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Repetitive hypothetical protein near ESAT cluster, SA0282 homolog</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_78</td><!-- Seq_Id -->
<td align="right">84</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">26.6%</td>
<td align="center"><font class="snp_type_synonymous">F28F</font>&nbsp;(TT<font class="mutation_in_codon">T</font>&rarr;TT<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Repetitive hypothetical protein near ESAT cluster, SA0282 homolog</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_78</td><!-- Seq_Id -->
<td align="right">91</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">24.6%</td>
<td align="center"><font class="snp_type_nonsynonymous">L31I</font>&nbsp;(<font class="mutation_in_codon">T</font>TA&rarr;<font class="mutation_in_codon">A</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Repetitive hypothetical protein near ESAT cluster, SA0282 homolog</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_78</td><!-- Seq_Id -->
<td align="right">101</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">T34M</font>&nbsp;(A<font class="mutation_in_codon">C</font>G&rarr;A<font class="mutation_in_codon">T</font>G)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Repetitive hypothetical protein near ESAT cluster, SA0282 homolog</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_78</td><!-- Seq_Id -->
<td align="right">458</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">7.8%</td>
<td align="center"><font class="snp_type_nonsynonymous">G93S</font>&nbsp;(<font class="mutation_in_codon">G</font>GT&rarr;<font class="mutation_in_codon">A</font>GT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Repetitive hypothetical protein near ESAT cluster, SA0282 homolog</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_78</td><!-- Seq_Id -->
<td align="right">481</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">29.7%</td>
<td align="center"><font class="snp_type_synonymous">C100C</font>&nbsp;(TG<font class="mutation_in_codon">C</font>&rarr;TG<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Repetitive hypothetical protein near ESAT cluster, SA0282 homolog</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_78</td><!-- Seq_Id -->
<td align="right">490</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="right">30.7%</td>
<td align="center"><font class="snp_type_synonymous">D103D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Repetitive hypothetical protein near ESAT cluster, SA0282 homolog</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_89</td><!-- Seq_Id -->
<td align="right">585</td><!-- Position -->
<td align="center">C&rarr;G</td><!-- Cell Mutation -->
<td align="right">6.2%</td>
<td align="center"><font class="snp_type_synonymous">P155P</font>&nbsp;(CC<font class="mutation_in_codon">C</font>&rarr;CC<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Mobile element protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_89</td><!-- Seq_Id -->
<td align="right">882</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">7.0%</td>
<td align="center"><font class="snp_type_synonymous">R254R</font>&nbsp;(AG<font class="mutation_in_codon">A</font>&rarr;AG<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Mobile element protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_89</td><!-- Seq_Id -->
<td align="right">940</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="right">5.9%</td>
<td align="center"><font class="snp_type_nonsynonymous">D274N</font>&nbsp;(<font class="mutation_in_codon">G</font>AT&rarr;<font class="mutation_in_codon">A</font>AT)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Mobile element protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_89</td><!-- Seq_Id -->
<td align="right">972</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">6.1%</td>
<td align="center"><font class="snp_type_synonymous">V284V</font>&nbsp;(GT<font class="mutation_in_codon">C</font>&rarr;GT<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Mobile element protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_89</td><!-- Seq_Id -->
<td align="right">1,094</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">8.4%</td>
<td align="center"><font class="snp_type_nonsynonymous">H325R</font>&nbsp;(C<font class="mutation_in_codon">A</font>C&rarr;C<font class="mutation_in_codon">G</font>C)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Mobile element protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_89</td><!-- Seq_Id -->
<td align="right">1,120</td><!-- Position -->
<td align="center">C&rarr;A</td><!-- Cell Mutation -->
<td align="right">11.8%</td>
<td align="center"><font class="snp_type_synonymous">R334R</font>&nbsp;(<font class="mutation_in_codon">C</font>GA&rarr;<font class="mutation_in_codon">A</font>GA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Mobile element protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_89</td><!-- Seq_Id -->
<td align="right">1,128</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="right">9.2%</td>
<td align="center"><font class="snp_type_synonymous">D336D</font>&nbsp;(GA<font class="mutation_in_codon">C</font>&rarr;GA<font class="mutation_in_codon">T</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Mobile element protein</td>
</tr>
<!-- End Table Row -->
<tr>
<th>seq&nbsp;id</th>
<th>position</th>
<th>mutation</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>


<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_96</td><!-- Seq_Id -->
<td align="right">5,969</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center"><font class="snp_type_synonymous">V52V</font>&nbsp;(GT<font class="mutation_in_codon">T</font>&rarr;GT<font class="mutation_in_codon">A</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Phage DNA replication protein O</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_96</td><!-- Seq_Id -->
<td align="right">8,004</td><!-- Position -->
<td align="center">A&rarr;G</td><!-- Cell Mutation -->
<td align="right">5.7%</td>
<td align="center"><font class="snp_type_synonymous">G8G</font>&nbsp;(GG<font class="mutation_in_codon">A</font>&rarr;GG<font class="mutation_in_codon">G</font>)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Hypothetical protein, SAB1734c homolog [SA bacteriophages 11, Mu50B]</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="polymorphism_table_row">
<td align="center">k141_96</td><!-- Seq_Id -->
<td align="right">8,011</td><!-- Position -->
<td align="center">A&rarr;C</td><!-- Cell Mutation -->
<td align="right">6.0%</td>
<td align="center"><font class="snp_type_nonsynonymous">I11L</font>&nbsp;(<font class="mutation_in_codon">A</font>TA&rarr;<font class="mutation_in_codon">C</font>TA)&nbsp;</td>
<td align="center"><i>rasttk_feature_annotation_tool=annotate_proteins_km er_v2</i>&nbsp;&rarr;</td>
<td align="left">Hypothetical protein, SAB1734c homolog [SA bacteriophages 11, Mu50B]</td>
</tr>
<!-- End Table Row -->
</table></body></html>