<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Summary Statistics</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.35.1&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<h2>Read File Information</h2>
<table border="0" cellspace="1" cellpadding="5">
<tr ><th></th><th>read file</th><th>reads</th><th>bases</th><th>passed&nbsp;filters</th><th>average</th><th>longest</th><th>mapped</th></tr>
<tr ><td><a href="calibration/W100_D1_qc_decon_pe.1.error_rates.pdf">errors</a></td><td>W100_D1_qc_decon_pe.1</td><td align="right">5,151</td><td align="right">772,650</td><td align="right">100.0%</td><td align="right">150.0&nbsp;bases</td><td align="right">150&nbsp;bases</td><td align="right">13.6%</td></tr><tr ><td><a href="calibration/W100_D1_qc_decon_pe.2.error_rates.pdf">errors</a></td><td>W100_D1_qc_decon_pe.2</td><td align="right">5,151</td><td align="right">772,650</td><td align="right">100.0%</td><td align="right">150.0&nbsp;bases</td><td align="right">150&nbsp;bases</td><td align="right">13.6%</td></tr><tr class="highlight_table_row"><td></td><td><b>total</b></td><td align="right"><b>10,302</b></td><td align="right"><b>1,545,300</b></td><td align="right">100.0%</td><td align="right">150.0&nbsp;bases</td><td align="right"><b>150</b>&nbsp;bases</td><td align="right">13.6%</td></tr></table><h2>Reference Sequence Information</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5" >
<tr><th></th><th></th><th>seq id</th><th>length</th><th align="center">fit mean</th><th align="center">fit dispersion</th><th align="center">% mapped reads</th><th align="left">description</th></tr>
<tr class="gray_table_row"><td><a href="evidence/k141_73.overview.png">coverage</a></td><td><a href="calibration/0.unique_coverage.pdf">distribution</a></td><td>k141_73</td><td align="right">28,052</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.4%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_80.overview.png">coverage</a></td><td><a href="calibration/1.unique_coverage.pdf">distribution</a></td><td>k141_80</td><td align="right">19,878</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_144.overview.png">coverage</a></td><td><a href="calibration/2.unique_coverage.pdf">distribution</a></td><td>k141_144</td><td align="right">5,529</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_149.overview.png">coverage</a></td><td><a href="calibration/3.unique_coverage.pdf">distribution</a></td><td>k141_149</td><td align="right">2,532</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_259.overview.png">coverage</a></td><td><a href="calibration/4.unique_coverage.pdf">distribution</a></td><td>k141_259</td><td align="right">2,879</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_300.overview.png">coverage</a></td><td><a href="calibration/5.unique_coverage.pdf">distribution</a></td><td>k141_300</td><td align="right">2,565</td><td align="center">NA</td><td align="center">NA</td><td align="center">1.9%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_319.overview.png">coverage</a></td><td><a href="calibration/6.unique_coverage.pdf">distribution</a></td><td>k141_319</td><td align="right">49,599</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.9%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_322.overview.png">coverage</a></td><td><a href="calibration/7.unique_coverage.pdf">distribution</a></td><td>k141_322</td><td align="right">2,605</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_371.overview.png">coverage</a></td><td><a href="calibration/8.unique_coverage.pdf">distribution</a></td><td>k141_371</td><td align="right">4,384</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_391.overview.png">coverage</a></td><td><a href="calibration/9.unique_coverage.pdf">distribution</a></td><td>k141_391</td><td align="right">3,346</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_399.overview.png">coverage</a></td><td><a href="calibration/10.unique_coverage.pdf">distribution</a></td><td>k141_399</td><td align="right">6,440</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.1%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_412.overview.png">coverage</a></td><td><a href="calibration/11.unique_coverage.pdf">distribution</a></td><td>k141_412</td><td align="right">6,606</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_417.overview.png">coverage</a></td><td><a href="calibration/12.unique_coverage.pdf">distribution</a></td><td>k141_417</td><td align="right">5,274</td><td align="center">NA</td><td align="center">NA</td><td align="center">1.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_423.overview.png">coverage</a></td><td><a href="calibration/13.unique_coverage.pdf">distribution</a></td><td>k141_423</td><td align="right">5,736</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_427.overview.png">coverage</a></td><td><a href="calibration/14.unique_coverage.pdf">distribution</a></td><td>k141_427</td><td align="right">8,181</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_430.overview.png">coverage</a></td><td><a href="calibration/15.unique_coverage.pdf">distribution</a></td><td>k141_430</td><td align="right">6,132</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_437.overview.png">coverage</a></td><td><a href="calibration/16.unique_coverage.pdf">distribution</a></td><td>k141_437</td><td align="right">4,117</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.1%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_444.overview.png">coverage</a></td><td><a href="calibration/17.unique_coverage.pdf">distribution</a></td><td>k141_444</td><td align="right">6,380</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_450.overview.png">coverage</a></td><td><a href="calibration/18.unique_coverage.pdf">distribution</a></td><td>k141_450</td><td align="right">4,836</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_455.overview.png">coverage</a></td><td><a href="calibration/19.unique_coverage.pdf">distribution</a></td><td>k141_455</td><td align="right">2,988</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.1%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_458.overview.png">coverage</a></td><td><a href="calibration/20.unique_coverage.pdf">distribution</a></td><td>k141_458</td><td align="right">9,601</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_462.overview.png">coverage</a></td><td><a href="calibration/21.unique_coverage.pdf">distribution</a></td><td>k141_462</td><td align="right">14,985</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.2%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_465.overview.png">coverage</a></td><td><a href="calibration/22.unique_coverage.pdf">distribution</a></td><td>k141_465</td><td align="right">16,071</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.5%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_470.overview.png">coverage</a></td><td><a href="calibration/23.unique_coverage.pdf">distribution</a></td><td>k141_470</td><td align="right">11,305</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_472.overview.png">coverage</a></td><td><a href="calibration/24.unique_coverage.pdf">distribution</a></td><td>k141_472</td><td align="right">18,050</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_474.overview.png">coverage</a></td><td><a href="calibration/25.unique_coverage.pdf">distribution</a></td><td>k141_474</td><td align="right">9,515</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.1%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_476.overview.png">coverage</a></td><td><a href="calibration/26.unique_coverage.pdf">distribution</a></td><td>k141_476</td><td align="right">16,415</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.4%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_480.overview.png">coverage</a></td><td><a href="calibration/27.unique_coverage.pdf">distribution</a></td><td>k141_480</td><td align="right">10,455</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.1%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_482.overview.png">coverage</a></td><td><a href="calibration/28.unique_coverage.pdf">distribution</a></td><td>k141_482</td><td align="right">29,301</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_484.overview.png">coverage</a></td><td><a href="calibration/29.unique_coverage.pdf">distribution</a></td><td>k141_484</td><td align="right">36,330</td><td align="center">NA</td><td align="center">NA</td><td align="center">1.3%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_486.overview.png">coverage</a></td><td><a href="calibration/30.unique_coverage.pdf">distribution</a></td><td>k141_486</td><td align="right">3,820</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_491.overview.png">coverage</a></td><td><a href="calibration/31.unique_coverage.pdf">distribution</a></td><td>k141_491</td><td align="right">2,822</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_493.overview.png">coverage</a></td><td><a href="calibration/32.unique_coverage.pdf">distribution</a></td><td>k141_493</td><td align="right">64,587</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.7%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_495.overview.png">coverage</a></td><td><a href="calibration/33.unique_coverage.pdf">distribution</a></td><td>k141_495</td><td align="right">70,707</td><td align="center">NA</td><td align="center">NA</td><td align="center">1.1%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_508.overview.png">coverage</a></td><td><a href="calibration/34.unique_coverage.pdf">distribution</a></td><td>k141_508</td><td align="right">30,548</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.3%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_509.overview.png">coverage</a></td><td><a href="calibration/35.unique_coverage.pdf">distribution</a></td><td>k141_509</td><td align="right">59,140</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.6%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_510.overview.png">coverage</a></td><td><a href="calibration/36.unique_coverage.pdf">distribution</a></td><td>k141_510</td><td align="right">47,961</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.1%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_519.overview.png">coverage</a></td><td><a href="calibration/37.unique_coverage.pdf">distribution</a></td><td>k141_519</td><td align="right">56,451</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.6%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_525.overview.png">coverage</a></td><td><a href="calibration/38.unique_coverage.pdf">distribution</a></td><td>k141_525</td><td align="right">90,917</td><td align="center">NA</td><td align="center">NA</td><td align="center">1.6%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_529.overview.png">coverage</a></td><td><a href="calibration/39.unique_coverage.pdf">distribution</a></td><td>k141_529</td><td align="right">58,477</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.4%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_533.overview.png">coverage</a></td><td><a href="calibration/40.unique_coverage.pdf">distribution</a></td><td>k141_533</td><td align="right">58,895</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.6%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_535.overview.png">coverage</a></td><td><a href="calibration/41.unique_coverage.pdf">distribution</a></td><td>k141_535</td><td align="right">23,115</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.3%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_536.overview.png">coverage</a></td><td><a href="calibration/42.unique_coverage.pdf">distribution</a></td><td>k141_536</td><td align="right">95,072</td><td align="center">NA</td><td align="center">NA</td><td align="center">1.4%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_537.overview.png">coverage</a></td><td><a href="calibration/43.unique_coverage.pdf">distribution</a></td><td>k141_537</td><td align="right">78,695</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.9%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_538.overview.png">coverage</a></td><td><a href="calibration/44.unique_coverage.pdf">distribution</a></td><td>k141_538</td><td align="right">32,887</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.6%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_540.overview.png">coverage</a></td><td><a href="calibration/45.unique_coverage.pdf">distribution</a></td><td>k141_540</td><td align="right">110,043</td><td align="center">NA</td><td align="center">NA</td><td align="center">1.3%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_544.overview.png">coverage</a></td><td><a href="calibration/46.unique_coverage.pdf">distribution</a></td><td>k141_544</td><td align="right">86,995</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.9%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_546.overview.png">coverage</a></td><td><a href="calibration/47.unique_coverage.pdf">distribution</a></td><td>k141_546</td><td align="right">34,885</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.1%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_549.overview.png">coverage</a></td><td><a href="calibration/48.unique_coverage.pdf">distribution</a></td><td>k141_549</td><td align="right">130,854</td><td align="center">NA</td><td align="center">NA</td><td align="center">1.4%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_551.overview.png">coverage</a></td><td><a href="calibration/49.unique_coverage.pdf">distribution</a></td><td>k141_551</td><td align="right">64,488</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.3%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_553.overview.png">coverage</a></td><td><a href="calibration/50.unique_coverage.pdf">distribution</a></td><td>k141_553</td><td align="right">47,502</td><td align="center">NA</td><td align="center">NA</td><td align="center">1.3%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_554.overview.png">coverage</a></td><td><a href="calibration/51.unique_coverage.pdf">distribution</a></td><td>k141_554</td><td align="right">2,801</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_556.overview.png">coverage</a></td><td><a href="calibration/52.unique_coverage.pdf">distribution</a></td><td>k141_556</td><td align="right">91,833</td><td align="center">NA</td><td align="center">NA</td><td align="center">1.9%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_557.overview.png">coverage</a></td><td><a href="calibration/53.unique_coverage.pdf">distribution</a></td><td>k141_557</td><td align="right">3,154</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_558.overview.png">coverage</a></td><td><a href="calibration/54.unique_coverage.pdf">distribution</a></td><td>k141_558</td><td align="right">195,927</td><td align="center">NA</td><td align="center">NA</td><td align="center">3.1%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_561.overview.png">coverage</a></td><td><a href="calibration/55.unique_coverage.pdf">distribution</a></td><td>k141_561</td><td align="right">187,506</td><td align="center">NA</td><td align="center">NA</td><td align="center">3.3%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_564.overview.png">coverage</a></td><td><a href="calibration/56.unique_coverage.pdf">distribution</a></td><td>k141_564</td><td align="right">85,442</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.6%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_568.overview.png">coverage</a></td><td><a href="calibration/57.unique_coverage.pdf">distribution</a></td><td>k141_568</td><td align="right">5,659</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.1%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_571.overview.png">coverage</a></td><td><a href="calibration/58.unique_coverage.pdf">distribution</a></td><td>k141_571</td><td align="right">2,255</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.1%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_577.overview.png">coverage</a></td><td><a href="calibration/59.unique_coverage.pdf">distribution</a></td><td>k141_577</td><td align="right">14,424</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.4%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_580.overview.png">coverage</a></td><td><a href="calibration/60.unique_coverage.pdf">distribution</a></td><td>k141_580</td><td align="right">207,473</td><td align="center">NA</td><td align="center">NA</td><td align="center">2.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_583.overview.png">coverage</a></td><td><a href="calibration/61.unique_coverage.pdf">distribution</a></td><td>k141_583</td><td align="right">185,542</td><td align="center">NA</td><td align="center">NA</td><td align="center">3.6%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_586.overview.png">coverage</a></td><td><a href="calibration/62.unique_coverage.pdf">distribution</a></td><td>k141_586</td><td align="right">180,930</td><td align="center">NA</td><td align="center">NA</td><td align="center">2.7%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_587.overview.png">coverage</a></td><td><a href="calibration/63.unique_coverage.pdf">distribution</a></td><td>k141_587</td><td align="right">30,926</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_591.overview.png">coverage</a></td><td><a href="calibration/64.unique_coverage.pdf">distribution</a></td><td>k141_591</td><td align="right">323,027</td><td align="center">NA</td><td align="center">NA</td><td align="center">5.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_592.overview.png">coverage</a></td><td><a href="calibration/65.unique_coverage.pdf">distribution</a></td><td>k141_592</td><td align="right">84,107</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.3%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_593.overview.png">coverage</a></td><td><a href="calibration/66.unique_coverage.pdf">distribution</a></td><td>k141_593</td><td align="right">208,805</td><td align="center">NA</td><td align="center">NA</td><td align="center">1.6%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_594.overview.png">coverage</a></td><td><a href="calibration/67.unique_coverage.pdf">distribution</a></td><td>k141_594</td><td align="right">189,829</td><td align="center">NA</td><td align="center">NA</td><td align="center">4.3%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_596.overview.png">coverage</a></td><td><a href="calibration/68.unique_coverage.pdf">distribution</a></td><td>k141_596</td><td align="right">407,688</td><td align="center">NA</td><td align="center">NA</td><td align="center">7.9%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_603.overview.png">coverage</a></td><td><a href="calibration/69.unique_coverage.pdf">distribution</a></td><td>k141_603</td><td align="right">187,884</td><td align="center">NA</td><td align="center">NA</td><td align="center">2.8%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_604.overview.png">coverage</a></td><td><a href="calibration/70.unique_coverage.pdf">distribution</a></td><td>k141_604</td><td align="right">74,912</td><td align="center">NA</td><td align="center">NA</td><td align="center">1.6%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_605.overview.png">coverage</a></td><td><a href="calibration/71.unique_coverage.pdf">distribution</a></td><td>k141_605</td><td align="right">379,147</td><td align="center">NA</td><td align="center">NA</td><td align="center">6.5%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_608.overview.png">coverage</a></td><td><a href="calibration/72.unique_coverage.pdf">distribution</a></td><td>k141_608</td><td align="right">534,602</td><td align="center">NA</td><td align="center">NA</td><td align="center">8.9%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_609.overview.png">coverage</a></td><td><a href="calibration/73.unique_coverage.pdf">distribution</a></td><td>k141_609</td><td align="right">231,289</td><td align="center">NA</td><td align="center">NA</td><td align="center">4.0%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_622.overview.png">coverage</a></td><td><a href="calibration/74.unique_coverage.pdf">distribution</a></td><td>k141_622</td><td align="right">116,598</td><td align="center">NA</td><td align="center">NA</td><td align="center">1.1%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_623.overview.png">coverage</a></td><td><a href="calibration/75.unique_coverage.pdf">distribution</a></td><td>k141_623</td><td align="right">268,506</td><td align="center">NA</td><td align="center">NA</td><td align="center">2.2%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_624.overview.png">coverage</a></td><td><a href="calibration/76.unique_coverage.pdf">distribution</a></td><td>k141_624</td><td align="right">154,649</td><td align="center">NA</td><td align="center">NA</td><td align="center">2.8%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_625.overview.png">coverage</a></td><td><a href="calibration/77.unique_coverage.pdf">distribution</a></td><td>k141_625</td><td align="right">71,762</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.8%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_626.overview.png">coverage</a></td><td><a href="calibration/78.unique_coverage.pdf">distribution</a></td><td>k141_626</td><td align="right">291,064</td><td align="center">NA</td><td align="center">NA</td><td align="center">2.6%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_631.overview.png">coverage</a></td><td><a href="calibration/79.unique_coverage.pdf">distribution</a></td><td>k141_631</td><td align="right">42,414</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.5%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_634.overview.png">coverage</a></td><td><a href="calibration/80.unique_coverage.pdf">distribution</a></td><td>k141_634</td><td align="right">303,106</td><td align="center">NA</td><td align="center">NA</td><td align="center">3.6%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="gray_table_row"><td><a href="evidence/k141_636.overview.png">coverage</a></td><td><a href="calibration/81.unique_coverage.pdf">distribution</a></td><td>k141_636</td><td align="right">233,691</td><td align="center">NA</td><td align="center">NA</td><td align="center">4.4%</td><td>Pseudomonas aeruginosa UCIMC22</td></tr><tr class="highlight_table_row"><td></td><td></td><td><b>total</b></td><td align="right"><b>6,895,900</b></td><td></td><td></td><td align="center">100.0%</td><td></td></tr>
</table>
<p><b>fit dispersion</b> is the ratio of the variance to the mean for the negative binomial fit. It is =1 for Poisson and >1 for over-dispersed data.
<p><span class="gray_table_row">Insufficient coverage</span> Reference sequence counted as entirely deleted due to low coverage. Try either the <code>-t,--targeted-sequencing</code> or the <code>-c,--contig-reference </code> option if you want mutations called for these reference sequences.
<h2>New Junction Evidence</h2>
<p>
<h3>Junction Candidates Tested</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">limit</th><th align="center">actual</th></tr><tr><td>Number of alignment pairs examined for constructing junction candidates</td><td align="center">&le;&nbsp;100000</td><td align="center">3</td></tr><tr><td>Coverage evenness (position-hash) score of junction candidates</td><td align="center">&ge;&nbsp;2</td><td align="center">&ge;&nbsp;0</td></tr><tr><td>Test this many junction candidates (n). May be smaller if not enough passed the coverage evenness threshold</td><td align="center">100&nbsp;&le;&nbsp;n&nbsp;&le;&nbsp;5000</td><td align="center">0</td></tr><tr><td>Total length of all junction candidates (factor times the reference genome length)</td><td align="center">&le;&nbsp;0.1</td><td align="center">0.000</td></tr></table>
<h3>Junction Skew Score Calculation</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr ><th>reference sequence</th><th>pr(no read start)</th></tr>
<tr >
<td>k141_73</td><td align="center">0.99989</td></tr><tr >
<td>k141_80</td><td align="center">NA</td></tr><tr >
<td>k141_144</td><td align="center">NA</td></tr><tr >
<td>k141_149</td><td align="center">NA</td></tr><tr >
<td>k141_259</td><td align="center">NA</td></tr><tr >
<td>k141_300</td><td align="center">0.99532</td></tr><tr >
<td>k141_319</td><td align="center">NA</td></tr><tr >
<td>k141_322</td><td align="center">NA</td></tr><tr >
<td>k141_371</td><td align="center">NA</td></tr><tr >
<td>k141_391</td><td align="center">NA</td></tr><tr >
<td>k141_399</td><td align="center">NA</td></tr><tr >
<td>k141_412</td><td align="center">NA</td></tr><tr >
<td>k141_417</td><td align="center">0.99915</td></tr><tr >
<td>k141_423</td><td align="center">NA</td></tr><tr >
<td>k141_427</td><td align="center">NA</td></tr><tr >
<td>k141_430</td><td align="center">NA</td></tr><tr >
<td>k141_437</td><td align="center">0.99976</td></tr><tr >
<td>k141_444</td><td align="center">NA</td></tr><tr >
<td>k141_450</td><td align="center">NA</td></tr><tr >
<td>k141_455</td><td align="center">0.99967</td></tr><tr >
<td>k141_458</td><td align="center">NA</td></tr><tr >
<td>k141_462</td><td align="center">NA</td></tr><tr >
<td>k141_465</td><td align="center">0.99987</td></tr><tr >
<td>k141_470</td><td align="center">NA</td></tr><tr >
<td>k141_472</td><td align="center">NA</td></tr><tr >
<td>k141_474</td><td align="center">NA</td></tr><tr >
<td>k141_476</td><td align="center">0.99988</td></tr><tr >
<td>k141_480</td><td align="center">0.99990</td></tr><tr >
<td>k141_482</td><td align="center">NA</td></tr><tr >
<td>k141_484</td><td align="center">0.99975</td></tr><tr >
<td>k141_486</td><td align="center">NA</td></tr><tr >
<td>k141_491</td><td align="center">NA</td></tr><tr >
<td>k141_493</td><td align="center">NA</td></tr><tr >
<td>k141_495</td><td align="center">0.99990</td></tr><tr >
<td>k141_508</td><td align="center">0.99995</td></tr><tr >
<td>k141_509</td><td align="center">0.99993</td></tr><tr >
<td>k141_510</td><td align="center">NA</td></tr><tr >
<td>k141_519</td><td align="center">0.99993</td></tr><tr >
<td>k141_525</td><td align="center">0.99991</td></tr><tr >
<td>k141_529</td><td align="center">0.99996</td></tr><tr >
<td>k141_533</td><td align="center">0.99993</td></tr><tr >
<td>k141_535</td><td align="center">NA</td></tr><tr >
<td>k141_536</td><td align="center">0.99991</td></tr><tr >
<td>k141_537</td><td align="center">0.99994</td></tr><tr >
<td>k141_538</td><td align="center">0.99989</td></tr><tr >
<td>k141_540</td><td align="center">NA</td></tr><tr >
<td>k141_544</td><td align="center">0.99994</td></tr><tr >
<td>k141_546</td><td align="center">NA</td></tr><tr >
<td>k141_549</td><td align="center">0.99993</td></tr><tr >
<td>k141_551</td><td align="center">0.99997</td></tr><tr >
<td>k141_553</td><td align="center">0.99982</td></tr><tr >
<td>k141_554</td><td align="center">NA</td></tr><tr >
<td>k141_556</td><td align="center">0.99987</td></tr><tr >
<td>k141_557</td><td align="center">NA</td></tr><tr >
<td>k141_558</td><td align="center">0.99990</td></tr><tr >
<td>k141_561</td><td align="center">0.99989</td></tr><tr >
<td>k141_564</td><td align="center">0.99995</td></tr><tr >
<td>k141_568</td><td align="center">NA</td></tr><tr >
<td>k141_571</td><td align="center">NA</td></tr><tr >
<td>k141_577</td><td align="center">NA</td></tr><tr >
<td>k141_580</td><td align="center">0.99994</td></tr><tr >
<td>k141_583</td><td align="center">0.99987</td></tr><tr >
<td>k141_586</td><td align="center">0.99990</td></tr><tr >
<td>k141_587</td><td align="center">NA</td></tr><tr >
<td>k141_591</td><td align="center">0.99991</td></tr><tr >
<td>k141_592</td><td align="center">0.99998</td></tr><tr >
<td>k141_593</td><td align="center">0.99995</td></tr><tr >
<td>k141_594</td><td align="center">0.99986</td></tr><tr >
<td>k141_596</td><td align="center">0.99988</td></tr><tr >
<td>k141_603</td><td align="center">0.99992</td></tr><tr >
<td>k141_604</td><td align="center">0.99987</td></tr><tr >
<td>k141_605</td><td align="center">0.99989</td></tr><tr >
<td>k141_608</td><td align="center">0.99991</td></tr><tr >
<td>k141_609</td><td align="center">0.99988</td></tr><tr >
<td>k141_622</td><td align="center">NA</td></tr><tr >
<td>k141_623</td><td align="center">0.99994</td></tr><tr >
<td>k141_624</td><td align="center">0.99988</td></tr><tr >
<td>k141_625</td><td align="center">0.99994</td></tr><tr >
<td>k141_626</td><td align="center">0.99995</td></tr><tr >
<td>k141_631</td><td align="center">0.99993</td></tr><tr >
<td>k141_634</td><td align="center">0.99993</td></tr><tr >
<td>k141_636</td><td align="center">0.99989</td></tr></table><p><b>pr(no read start)</b> is the probability that there will not be an aligned read whose first base matches a given position on a given strand.
<h3>Final Junction Predictions</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">value</th></tr><tr><td>Coverage evenness (position-hash) score of predicted junctions must be</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Minimum probablilty assigned that no mapped read will start at a given position and strand for junction prediction</td><td align="center">0.1</td></tr><tr><td>Junction allow suboptimal matches</td><td align="center">FALSE</td></tr><tr><td>Skew score of predicted junction (&minus;log10 probability of unusual coverage evenness) must be</td><td align="center">&le;&nbsp;0</td></tr><tr><td>Number of bases that at least one read must overlap each uniquely aligned side of a predicted junction</td><td align="center">&ge;&nbsp;6</td></tr></table>
<h2>Read Alignment Evidence</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>option</th><th>value</th></tr><tr><td>Mode</td><td>Full Polymorphism</td></tr><tr><td>Ploidy</td><td>1 (haploid)</td></tr><tr><td>Consensus mutation E-value cutoff</td><td>10</td></tr><tr><td>Consensus frequency cutoff</td><td>OFF</td></tr><tr><td>Consensus minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum variant coverage</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism E-value cutoff</td><td>2</td></tr><tr><td>Polymorphism frequency cutoff</td><td>0.05</td></tr><tr><td>Polymorphism minimum variant coverage each strand</td><td>2</td></tr><tr><td>Polymorphism minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum variant coverage</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism bias cutoff</td><td>OFF</td></tr><tr><td>Predict indel polymorphisms</td><td>YES</td></tr><tr><td>Skip indel polymorphisms in homopolymers runs of</td><td> &ge;3 bases</td></tr><tr><td>Skip base substitutions when they create a homopolymer flanked on each side by</td><td> &ge;5 bases</td></tr></table><p>
<h2>Software Versions</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>program</th><th>version</th></tr>
<tr><td>bowtie2</td><td>2.2.7</td></tr>
<tr><td>R</td><td>3.3.2</td></tr>
</table><p>
<h2>Execution Times</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>step</th><th>start</th><th>end</th><th>elapsed</th></tr>
<tr><td>Read and reference sequence file input</td><td>16:28:14&nbsp;28&nbsp;Apr&nbsp;2020</td><td>16:28:17&nbsp;28&nbsp;Apr&nbsp;2020</td><td>3&nbsp;seconds</td></tr>
<tr><td>Read alignment to reference genome</td><td>16:47:26&nbsp;28&nbsp;Apr&nbsp;2020</td><td>16:47:36&nbsp;28&nbsp;Apr&nbsp;2020</td><td>10&nbsp;seconds</td></tr>
<tr><td>Preprocessing alignments for candidate junction identification</td><td>16:47:36&nbsp;28&nbsp;Apr&nbsp;2020</td><td>16:47:36&nbsp;28&nbsp;Apr&nbsp;2020</td><td>0&nbsp;seconds</td></tr>
<tr><td>Preliminary analysis of coverage distribution</td><td>16:47:36&nbsp;28&nbsp;Apr&nbsp;2020</td><td>16:48:11&nbsp;28&nbsp;Apr&nbsp;2020</td><td>35&nbsp;seconds</td></tr>
<tr><td>Identifying junction candidates</td><td>16:48:11&nbsp;28&nbsp;Apr&nbsp;2020</td><td>16:48:11&nbsp;28&nbsp;Apr&nbsp;2020</td><td>0&nbsp;seconds</td></tr>
<tr><td>Re-alignment to junction candidates</td><td>16:48:11&nbsp;28&nbsp;Apr&nbsp;2020</td><td>16:48:11&nbsp;28&nbsp;Apr&nbsp;2020</td><td>0&nbsp;seconds</td></tr>
<tr><td>Resolving best read alignments</td><td>16:48:11&nbsp;28&nbsp;Apr&nbsp;2020</td><td>16:48:12&nbsp;28&nbsp;Apr&nbsp;2020</td><td>1&nbsp;second</td></tr>
<tr><td>Creating BAM files</td><td>16:48:12&nbsp;28&nbsp;Apr&nbsp;2020</td><td>16:48:12&nbsp;28&nbsp;Apr&nbsp;2020</td><td>0&nbsp;seconds</td></tr>
<tr><td>Tabulating error counts</td><td>16:48:12&nbsp;28&nbsp;Apr&nbsp;2020</td><td>16:48:13&nbsp;28&nbsp;Apr&nbsp;2020</td><td>1&nbsp;second</td></tr>
<tr><td>Re-calibrating base error rates</td><td>16:48:13&nbsp;28&nbsp;Apr&nbsp;2020</td><td>16:48:45&nbsp;28&nbsp;Apr&nbsp;2020</td><td>32&nbsp;seconds</td></tr>
<tr><td>Examining read alignment evidence</td><td>16:48:45&nbsp;28&nbsp;Apr&nbsp;2020</td><td>16:48:55&nbsp;28&nbsp;Apr&nbsp;2020</td><td>10&nbsp;seconds</td></tr>
<tr><td>Polymorphism statistics</td><td>16:48:55&nbsp;28&nbsp;Apr&nbsp;2020</td><td>16:48:55&nbsp;28&nbsp;Apr&nbsp;2020</td><td>0&nbsp;seconds</td></tr>
<tr><td>Output</td><td>16:48:55&nbsp;28&nbsp;Apr&nbsp;2020</td><td>16:50:53&nbsp;28&nbsp;Apr&nbsp;2020</td><td>1&nbsp;minute&nbsp;58&nbsp;seconds</td></tr>
<tr class="highlight_table_row">
<td colspan="3" ><b>Total</b></td>
<td><b>3&nbsp;minutes&nbsp;30&nbsp;seconds</b></td>
</tr>
</table></body></html>